{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "import pandas as pd\n",
    "import re\n",
    "\n",
    "# 指定Entrez使用的邮件地址，这是必需的\n",
    "Entrez.email = \"3575318018@qq.com\"\n",
    "\n",
    "# 保存表格的名称开头，便于分类\n",
    "# key_name = \"An\"\n",
    "# name = \"Aspergillus niger\"\n",
    "# key_name = \"Tr\"\n",
    "# name = \"Trichoderma reesei\"\n",
    "key_name = \"Nc\"\n",
    "name = \"Neurospora crassa\"\n",
    "\n",
    "# 指定搜索的关键词和数据库\n",
    "search_term = name + \" AND GSE[Entry Type]\"\n",
    "database = \"gds\"\n",
    "\n",
    "# 使用Entrez进行搜索并获取ID列表\n",
    "handle1 = Entrez.esearch(db=database, term=search_term, retstart=0, retmax=200)\n",
    "record = Entrez.read(handle1)\n",
    "id_list = record[\"IdList\"]\n",
    "\n",
    "# 获取所有ID的详细信息\n",
    "handle2 = Entrez.esummary(db=database, id=\",\".join(id_list))\n",
    "records = Entrez.read(handle2)\n",
    "records = list(records)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_gse_info(records):\n",
    "    \"\"\"根据records,获取相关菌种的gse数据集关键信息.\n",
    "    \"\"\"\n",
    "    gse_list = []\n",
    "    for record in records:\n",
    "        title = record[\"title\"]\n",
    "        summary = record[\"summary\"]\n",
    "        Gse_id = record['Accession']\n",
    "        samples_num = record['n_samples']\n",
    "        \n",
    "        if \"PubMedIds\" in record and len(record[\"PubMedIds\"]) > 0:\n",
    "            pmid = str(record[\"PubMedIds\"][0])\n",
    "            match = re.search(r'\\d+', pmid)\n",
    "            if match:\n",
    "                pmid = match.group(0)\n",
    "            else:\n",
    "                pmid = \"N/A\"\n",
    "        else:\n",
    "            pmid = \"N/A\"\n",
    "\n",
    "        gse_list.append({'GSE Series':Gse_id,'Title':title,'Summary':summary,'Samples':samples_num,'PMID':pmid})\n",
    "\n",
    "    df_gse = pd.DataFrame(gse_list)\n",
    "    df_gse.to_csv(key_name + '_' + 'GSE_Info_All.csv', index=False, columns=['GSE Series','Title','Summary','Samples','PMID'])  # All代表是NCBI上记录的此菌种的所有GSE数据集\n",
    "    \n",
    "    # 将df_gse中GSE Series的GSE号保存在一个txt文件中，行尾序列为LF格式\n",
    "    with open(key_name + '_' + 'All_GSE.txt', 'w') as f:\n",
    "        for i in df_gse['GSE Series']:\n",
    "            f.write(i + '\\n')\n",
    "\n",
    "    return df_gse\n",
    "\n",
    "\n",
    "def get_gsm_and_sample_name(records):\n",
    "    \"\"\"根据records,获取 GSM Sample ID 与 name 及样本对应的GSE Series ID.\n",
    "    \"\"\"\n",
    "    sample_list = []\n",
    "    for record in records:\n",
    "        Gse_id = record['Accession']\n",
    "        samples = record['Samples']\n",
    "        for key in samples:\n",
    "            accession = key['Accession']\n",
    "            title = key['Title']\n",
    "            sample_list.append({'GEO Series': Gse_id,'Accession': accession,'Title': title})\n",
    "            \n",
    "    df = pd.DataFrame(sample_list)\n",
    "    df.to_csv(key_name + '_' + 'GSM_ID.csv', index=False, columns=['GEO Series','Accession','Title'])\n",
    "    \n",
    "    return df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>GSE Series</th>\n",
       "      <th>Title</th>\n",
       "      <th>Summary</th>\n",
       "      <th>Samples</th>\n",
       "      <th>PMID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GSE237909</td>\n",
       "      <td>Cellular communication and fusion regulates ce...</td>\n",
       "      <td>In this study, we characterized Aoadv-1, Aoso,...</td>\n",
       "      <td>18</td>\n",
       "      <td>N/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GSE232935</td>\n",
       "      <td>Histone deacetylation and cytosine methylation...</td>\n",
       "      <td>This SuperSeries is composed of the SubSeries ...</td>\n",
       "      <td>31</td>\n",
       "      <td>N/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GSE232934</td>\n",
       "      <td>Histone deacetylation and cytosine methylation...</td>\n",
       "      <td>Chromosomes must correctly fold in eukaryotic ...</td>\n",
       "      <td>12</td>\n",
       "      <td>N/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GSE232933</td>\n",
       "      <td>Histone deacetylation and cytosine methylation...</td>\n",
       "      <td>Chromosomes must correctly fold in eukaryotic ...</td>\n",
       "      <td>19</td>\n",
       "      <td>N/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GSE220169</td>\n",
       "      <td>The nutrient-sensing GCN2 signaling pathway is...</td>\n",
       "      <td>Circadian clocks are evolved to adapt to the d...</td>\n",
       "      <td>12</td>\n",
       "      <td>37083494</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>GSE14909</td>\n",
       "      <td>Molecular characterization of a cryptochrome i...</td>\n",
       "      <td>Cryptochromes were identified in plants and an...</td>\n",
       "      <td>12</td>\n",
       "      <td>20305004</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>GSE13977</td>\n",
       "      <td>Expression of lignin-degrading enzymes in soil...</td>\n",
       "      <td>Microarrays have become established tools for ...</td>\n",
       "      <td>68</td>\n",
       "      <td>N/A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>GSE12893</td>\n",
       "      <td>Characterization of the Ku70 homologue HdfA de...</td>\n",
       "      <td>Targeting an engineered DNA fragment to a spec...</td>\n",
       "      <td>11</td>\n",
       "      <td>19269344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>GSE8932</td>\n",
       "      <td>Genome-wide characterization of light-inducibl...</td>\n",
       "      <td>To better understand the roles of WC-1, WC-2 a...</td>\n",
       "      <td>135</td>\n",
       "      <td>19262566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>GSE12690</td>\n",
       "      <td>Relics of repeat-induced point mutation direct...</td>\n",
       "      <td>Both RNAi-dependent and -independent mechanism...</td>\n",
       "      <td>13</td>\n",
       "      <td>19092133</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>158 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    GSE Series                                              Title   \n",
       "0    GSE237909  Cellular communication and fusion regulates ce...  \\\n",
       "1    GSE232935  Histone deacetylation and cytosine methylation...   \n",
       "2    GSE232934  Histone deacetylation and cytosine methylation...   \n",
       "3    GSE232933  Histone deacetylation and cytosine methylation...   \n",
       "4    GSE220169  The nutrient-sensing GCN2 signaling pathway is...   \n",
       "..         ...                                                ...   \n",
       "153   GSE14909  Molecular characterization of a cryptochrome i...   \n",
       "154   GSE13977  Expression of lignin-degrading enzymes in soil...   \n",
       "155   GSE12893  Characterization of the Ku70 homologue HdfA de...   \n",
       "156    GSE8932  Genome-wide characterization of light-inducibl...   \n",
       "157   GSE12690  Relics of repeat-induced point mutation direct...   \n",
       "\n",
       "                                               Summary  Samples      PMID  \n",
       "0    In this study, we characterized Aoadv-1, Aoso,...       18       N/A  \n",
       "1    This SuperSeries is composed of the SubSeries ...       31       N/A  \n",
       "2    Chromosomes must correctly fold in eukaryotic ...       12       N/A  \n",
       "3    Chromosomes must correctly fold in eukaryotic ...       19       N/A  \n",
       "4    Circadian clocks are evolved to adapt to the d...       12  37083494  \n",
       "..                                                 ...      ...       ...  \n",
       "153  Cryptochromes were identified in plants and an...       12  20305004  \n",
       "154  Microarrays have become established tools for ...       68       N/A  \n",
       "155  Targeting an engineered DNA fragment to a spec...       11  19269344  \n",
       "156  To better understand the roles of WC-1, WC-2 a...      135  19262566  \n",
       "157  Both RNAi-dependent and -independent mechanism...       13  19092133  \n",
       "\n",
       "[158 rows x 5 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "get_gse_info(records)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>GEO Series</th>\n",
       "      <th>Accession</th>\n",
       "      <th>Title</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GSE237909</td>\n",
       "      <td>GSM7656285</td>\n",
       "      <td>C_36_1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GSE237909</td>\n",
       "      <td>GSM7656271</td>\n",
       "      <td>A_0_2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GSE237909</td>\n",
       "      <td>GSM7656279</td>\n",
       "      <td>A_36_1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GSE237909</td>\n",
       "      <td>GSM7656282</td>\n",
       "      <td>B_36_1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GSE237909</td>\n",
       "      <td>GSM7656283</td>\n",
       "      <td>B_36_2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5848</th>\n",
       "      <td>GSE12690</td>\n",
       "      <td>GSM318587</td>\n",
       "      <td>wt strain N150 MeDIP rep 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5849</th>\n",
       "      <td>GSE12690</td>\n",
       "      <td>GSM318604</td>\n",
       "      <td>double argonaute mutant strain N3406 MeDIP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5850</th>\n",
       "      <td>GSE12690</td>\n",
       "      <td>GSM318578</td>\n",
       "      <td>wt H3K9me3 rep 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5851</th>\n",
       "      <td>GSE12690</td>\n",
       "      <td>GSM318582</td>\n",
       "      <td>HP1-FLAG ChIP-chip</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5852</th>\n",
       "      <td>GSE12690</td>\n",
       "      <td>GSM318602</td>\n",
       "      <td>dim-2 mutant strain N1850 H3K9me2 ChIP-chip</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5853 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     GEO Series   Accession                                        Title\n",
       "0     GSE237909  GSM7656285                                       C_36_1\n",
       "1     GSE237909  GSM7656271                                        A_0_2\n",
       "2     GSE237909  GSM7656279                                       A_36_1\n",
       "3     GSE237909  GSM7656282                                       B_36_1\n",
       "4     GSE237909  GSM7656283                                       B_36_2\n",
       "...         ...         ...                                          ...\n",
       "5848   GSE12690   GSM318587                   wt strain N150 MeDIP rep 1\n",
       "5849   GSE12690   GSM318604   double argonaute mutant strain N3406 MeDIP\n",
       "5850   GSE12690   GSM318578                             wt H3K9me3 rep 1\n",
       "5851   GSE12690   GSM318582                           HP1-FLAG ChIP-chip\n",
       "5852   GSE12690   GSM318602  dim-2 mutant strain N1850 H3K9me2 ChIP-chip\n",
       "\n",
       "[5853 rows x 3 columns]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "get_gsm_and_sample_name(records)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# for record in records:\n",
    "#     Samples_name = record['Samples']\n",
    "#    #  for i in Samples_name:\n",
    "#       #  Samples_name_singel = Samples_name['Accession']\n",
    "#       #  Samples_name_title = Samples_name['Title']\n",
    "\n",
    "#     print(\"Samples_name: {}\".format(Samples_name))\n",
    "\n",
    "# print(\"Title: {}\".format(title))\n",
    "# print(\"Summary: {}\".format(summary))\n",
    "# print(\"Pmid: {}\".format(pmid))\n",
    "# print(\"GSE: {}\".format(Gse_id))\n",
    "# print(\"Samples: {}\".format(samples_num))\n",
    "\n",
    "# for record in records:\n",
    "#     samples = record['Samples']\n",
    "#     for sample in samples:\n",
    "#         accession = sample['Accession']\n",
    "#         title = sample['Title']\n",
    "#         print(\"Accession: {}, Title: {}\".format(accession, title))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "code",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
