{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "from Bio import Medline\n",
    "from Bio import Entrez\n",
    "import string \n",
    "\n",
    "Entrez.email = \"1219131962@qq.com\"\n",
    "Entrez.api_key = \"6fcb01c2989c35aa1da3d9d0118abedfc409\"\n",
    "PMID_info = []\n",
    "CGM_result_dict = {}\n",
    "\n",
    "def get_keywords(PMID_list):\n",
    "    \"\"\"Thank to Dr. Mao Zhitao for his help in this function.\n",
    "    \"\"\"\n",
    "\n",
    "    for i in PMID_list:\n",
    "        handle = Entrez.efetch(db=\"pubmed\", id=i, rettype=\"medline\", retmode=\"text\")\n",
    "        records = Medline.parse(handle)\n",
    "        records = list(records)\n",
    "\n",
    "        for record in records:\n",
    "            title = record.get(\"TI\",\"?\")\n",
    "            abstract = record.get(\"AB\",\"?\")\n",
    "            keywords = record.get(\"OT\", \"?\")\n",
    "            Journal = record.get(\"JT\", \"?\")\n",
    "            country = record.get(\"AD\", \"?\")[0].split(', ')[-1].strip(string.punctuation)\n",
    "            year = record.get(\"DP\",\"?\")\n",
    "            pmid = record.get('PMID','?')\n",
    "\n",
    "            PMID_info.append({'PMID': pmid,\n",
    "                            'Title':title,\n",
    "                            'Abstract': abstract,\n",
    "                            'Keywords': keywords,\n",
    "                            'Journal': Journal,\n",
    "                            'Country': country,\n",
    "                            'Year': year})\n",
    "            \n",
    "    for key, value in CGM_result_dict.items():\n",
    "        PMID_info.append([key,value])\n",
    "\n",
    "    return PMID_info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# key_name = 'Tr'\n",
    "key_name = 'Nc'\n",
    "# key_name = 'An'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "读取所有数据集信息，然后只保留有对应PMID的数据集"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>GSE Series</th>\n",
       "      <th>Title</th>\n",
       "      <th>Summary</th>\n",
       "      <th>Samples</th>\n",
       "      <th>PMID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GSE220169</td>\n",
       "      <td>The nutrient-sensing GCN2 signaling pathway is...</td>\n",
       "      <td>Circadian clocks are evolved to adapt to the d...</td>\n",
       "      <td>12</td>\n",
       "      <td>37083494</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GSE201901</td>\n",
       "      <td>A role for gene expression and mRNA stability ...</td>\n",
       "      <td>To identify the role of the cleavage and polya...</td>\n",
       "      <td>4</td>\n",
       "      <td>36603054</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GSE181566</td>\n",
       "      <td>Circadian Clock-Controlled Translation of Spec...</td>\n",
       "      <td>This SuperSeries is composed of the SubSeries ...</td>\n",
       "      <td>160</td>\n",
       "      <td>36577368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GSE181565</td>\n",
       "      <td>Circadian Clock-Controlled Translation of Spec...</td>\n",
       "      <td>We report the genome-wide impact of rhythmic P...</td>\n",
       "      <td>80</td>\n",
       "      <td>36577368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GSE181564</td>\n",
       "      <td>Circadian Clock-Controlled Translation of Spec...</td>\n",
       "      <td>We report the genome-wide impact of rhythmic P...</td>\n",
       "      <td>80</td>\n",
       "      <td>36577368</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>GSE19824</td>\n",
       "      <td>Genome-wide evolutionary analysis of eukaryoti...</td>\n",
       "      <td>Eukaryotic cytosine methylation represses tran...</td>\n",
       "      <td>33</td>\n",
       "      <td>20395474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>GSE14909</td>\n",
       "      <td>Molecular characterization of a cryptochrome i...</td>\n",
       "      <td>Cryptochromes were identified in plants and an...</td>\n",
       "      <td>12</td>\n",
       "      <td>20305004</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>GSE12893</td>\n",
       "      <td>Characterization of the Ku70 homologue HdfA de...</td>\n",
       "      <td>Targeting an engineered DNA fragment to a spec...</td>\n",
       "      <td>11</td>\n",
       "      <td>19269344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>GSE8932</td>\n",
       "      <td>Genome-wide characterization of light-inducibl...</td>\n",
       "      <td>To better understand the roles of WC-1, WC-2 a...</td>\n",
       "      <td>135</td>\n",
       "      <td>19262566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>GSE12690</td>\n",
       "      <td>Relics of repeat-induced point mutation direct...</td>\n",
       "      <td>Both RNAi-dependent and -independent mechanism...</td>\n",
       "      <td>13</td>\n",
       "      <td>19092133</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>129 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    GSE Series                                              Title   \n",
       "0    GSE220169  The nutrient-sensing GCN2 signaling pathway is...  \\\n",
       "1    GSE201901  A role for gene expression and mRNA stability ...   \n",
       "2    GSE181566  Circadian Clock-Controlled Translation of Spec...   \n",
       "3    GSE181565  Circadian Clock-Controlled Translation of Spec...   \n",
       "4    GSE181564  Circadian Clock-Controlled Translation of Spec...   \n",
       "..         ...                                                ...   \n",
       "124   GSE19824  Genome-wide evolutionary analysis of eukaryoti...   \n",
       "125   GSE14909  Molecular characterization of a cryptochrome i...   \n",
       "126   GSE12893  Characterization of the Ku70 homologue HdfA de...   \n",
       "127    GSE8932  Genome-wide characterization of light-inducibl...   \n",
       "128   GSE12690  Relics of repeat-induced point mutation direct...   \n",
       "\n",
       "                                               Summary  Samples      PMID  \n",
       "0    Circadian clocks are evolved to adapt to the d...       12  37083494  \n",
       "1    To identify the role of the cleavage and polya...        4  36603054  \n",
       "2    This SuperSeries is composed of the SubSeries ...      160  36577368  \n",
       "3    We report the genome-wide impact of rhythmic P...       80  36577368  \n",
       "4    We report the genome-wide impact of rhythmic P...       80  36577368  \n",
       "..                                                 ...      ...       ...  \n",
       "124  Eukaryotic cytosine methylation represses tran...       33  20395474  \n",
       "125  Cryptochromes were identified in plants and an...       12  20305004  \n",
       "126  Targeting an engineered DNA fragment to a spec...       11  19269344  \n",
       "127  To better understand the roles of WC-1, WC-2 a...      135  19262566  \n",
       "128  Both RNAi-dependent and -independent mechanism...       13  19092133  \n",
       "\n",
       "[129 rows x 5 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# 将表中PMID值为空的行删除\n",
    "df = pd.read_csv(key_name + '_GSE_Info_All.csv')\n",
    "df = df.dropna(subset=['PMID'])\n",
    "df = df.reset_index(drop=True)\n",
    "df['PMID'] = df['PMID'].astype(int)\n",
    "\n",
    "df.to_csv(key_name + '_GSE_Info_Pub.csv', index=False)\n",
    "df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "根据PMID号码，解析此文献的基本信息"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'PMID': '37083494',\n",
       "  'Title': 'The nutrient-sensing GCN2 signaling pathway is essential for circadian clock function by regulating histone acetylation under amino acid starvation.',\n",
       "  'Abstract': 'Circadian clocks are evolved to adapt to the daily environmental changes under different conditions. The ability to maintain circadian clock functions in response to various stresses and perturbations is important for organismal fitness. Here, we show that the nutrient-sensing GCN2 signaling pathway is required for robust circadian clock function under amino acid starvation in Neurospora. The deletion of GCN2 pathway components disrupts rhythmic transcription of clock gene frq by suppressing WC complex binding at the frq promoter due to its reduced histone H3 acetylation levels. Under amino acid starvation, the activation of GCN2 kinase and its downstream transcription factor CPC-1 establish a proper chromatin state at the frq promoter by recruiting the histone acetyltransferase GCN-5. The arrhythmic phenotype of the GCN2 kinase mutants under amino acid starvation can be rescued by inhibiting histone deacetylation. Finally, genome-wide transcriptional analysis indicates that the GCN2 signaling pathway maintains robust rhythmic expression of metabolic genes under amino acid starvation. Together, these results uncover an essential role of the GCN2 signaling pathway in maintaining the robust circadian clock function in response to amino acid starvation, and demonstrate the importance of histone acetylation at the frq locus in rhythmic gene expression.',\n",
       "  'Keywords': ['CPC-3/CPC-1 pathway',\n",
       "   'N. crassa',\n",
       "   'Neurospora crassa',\n",
       "   'amino acid starvation',\n",
       "   'cell biology',\n",
       "   'circadian rhythm',\n",
       "   'genetics',\n",
       "   'genomics',\n",
       "   'histone acetylation'],\n",
       "  'Journal': 'eLife',\n",
       "  'Country': 'China',\n",
       "  'Year': '2023 Apr 21'},\n",
       " {'PMID': '36603054',\n",
       "  'Title': 'Nutritional compensation of the circadian clock is a conserved process influenced by gene expression regulation and mRNA stability.',\n",
       "  'Abstract': \"Compensation is a defining principle of a true circadian clock, where its approximately 24-hour period length is relatively unchanged across environmental conditions. Known compensation effectors directly regulate core clock factors to buffer the oscillator's period length from variables in the environment. Temperature Compensation mechanisms have been experimentally addressed across circadian model systems, but much less is known about the related process of Nutritional Compensation, where circadian period length is maintained across physiologically relevant nutrient levels. Using the filamentous fungus Neurospora crassa, we performed a genetic screen under glucose and amino acid starvation conditions to identify new regulators of Nutritional Compensation. Our screen uncovered 16 novel mutants, and together with 4 mutants characterized in prior work, a model emerges where Nutritional Compensation of the fungal clock is achieved at the levels of transcription, chromatin regulation, and mRNA stability. However, eukaryotic circadian Nutritional Compensation is completely unstudied outside of Neurospora. To test for conservation in cultured human cells, we selected top hits from our fungal genetic screen, performed siRNA knockdown experiments of the mammalian orthologs, and characterized the cell lines with respect to compensation. We find that the wild-type mammalian clock is also compensated across a large range of external glucose concentrations, as observed in Neurospora, and that knocking down the mammalian orthologs of the Neurospora compensation-associated genes CPSF6 or SETD2 in human cells also results in nutrient-dependent period length changes. We conclude that, like Temperature Compensation, Nutritional Compensation is a conserved circadian process in fungal and mammalian clocks and that it may share common molecular determinants.\",\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS biology',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2023 Jan'},\n",
       " {'PMID': '36577368',\n",
       "  'Title': 'A circadian clock translational control mechanism targets specific mRNAs to cytoplasmic messenger ribonucleoprotein granules.',\n",
       "  'Abstract': \"Phosphorylation of Neurospora crassa eukaryotic initiation factor 2 alpha (eIF2alpha), a conserved translation initiation factor, is clock controlled. To determine the impact of rhythmic eIF2alpha phosphorylation on translation, we performed temporal ribosome profiling and RNA sequencing (RNA-seq) in wild-type (WT), clock mutant Deltafrq, eIF2alpha kinase mutant Deltacpc-3, and constitutively active cpc-3(c) cells. About 14% of mRNAs are rhythmically translated in WT cells, and translation rhythms for  approximately 30% of these mRNAs, which we named circadian translation-initiation-controlled genes (cTICs), are dependent on the clock and CPC-3. Most cTICs are expressed from arrhythmic mRNAs and contain a P-body (PB) localization motif in their 5' leader sequence. Deletion of SNR-1, a component of cytoplasmic messenger ribonucleoprotein granules (cmRNPgs) that include PBs and stress granules (SGs), and the PB motif on one of the cTIC mRNAs, zip-1, significantly alters zip-1 rhythmic translation. These results reveal that the clock regulates rhythmic translation of specific mRNAs through rhythmic eIF2alpha activity and cmRNPg metabolism.\",\n",
       "  'Keywords': ['CP: Molecular biology',\n",
       "   'CPC-3',\n",
       "   'GCN2',\n",
       "   'Neurospora crassa',\n",
       "   'P-body',\n",
       "   'circadian clock',\n",
       "   'cytoplasmic messenger ribonucleoprotein granules',\n",
       "   'eIF2alpha',\n",
       "   'ribosome profiling',\n",
       "   'translation control'],\n",
       "  'Journal': 'Cell reports',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2022 Dec 27'},\n",
       " {'PMID': '36577368',\n",
       "  'Title': 'A circadian clock translational control mechanism targets specific mRNAs to cytoplasmic messenger ribonucleoprotein granules.',\n",
       "  'Abstract': \"Phosphorylation of Neurospora crassa eukaryotic initiation factor 2 alpha (eIF2alpha), a conserved translation initiation factor, is clock controlled. To determine the impact of rhythmic eIF2alpha phosphorylation on translation, we performed temporal ribosome profiling and RNA sequencing (RNA-seq) in wild-type (WT), clock mutant Deltafrq, eIF2alpha kinase mutant Deltacpc-3, and constitutively active cpc-3(c) cells. About 14% of mRNAs are rhythmically translated in WT cells, and translation rhythms for  approximately 30% of these mRNAs, which we named circadian translation-initiation-controlled genes (cTICs), are dependent on the clock and CPC-3. Most cTICs are expressed from arrhythmic mRNAs and contain a P-body (PB) localization motif in their 5' leader sequence. Deletion of SNR-1, a component of cytoplasmic messenger ribonucleoprotein granules (cmRNPgs) that include PBs and stress granules (SGs), and the PB motif on one of the cTIC mRNAs, zip-1, significantly alters zip-1 rhythmic translation. These results reveal that the clock regulates rhythmic translation of specific mRNAs through rhythmic eIF2alpha activity and cmRNPg metabolism.\",\n",
       "  'Keywords': ['CP: Molecular biology',\n",
       "   'CPC-3',\n",
       "   'GCN2',\n",
       "   'Neurospora crassa',\n",
       "   'P-body',\n",
       "   'circadian clock',\n",
       "   'cytoplasmic messenger ribonucleoprotein granules',\n",
       "   'eIF2alpha',\n",
       "   'ribosome profiling',\n",
       "   'translation control'],\n",
       "  'Journal': 'Cell reports',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2022 Dec 27'},\n",
       " {'PMID': '36577368',\n",
       "  'Title': 'A circadian clock translational control mechanism targets specific mRNAs to cytoplasmic messenger ribonucleoprotein granules.',\n",
       "  'Abstract': \"Phosphorylation of Neurospora crassa eukaryotic initiation factor 2 alpha (eIF2alpha), a conserved translation initiation factor, is clock controlled. To determine the impact of rhythmic eIF2alpha phosphorylation on translation, we performed temporal ribosome profiling and RNA sequencing (RNA-seq) in wild-type (WT), clock mutant Deltafrq, eIF2alpha kinase mutant Deltacpc-3, and constitutively active cpc-3(c) cells. About 14% of mRNAs are rhythmically translated in WT cells, and translation rhythms for  approximately 30% of these mRNAs, which we named circadian translation-initiation-controlled genes (cTICs), are dependent on the clock and CPC-3. Most cTICs are expressed from arrhythmic mRNAs and contain a P-body (PB) localization motif in their 5' leader sequence. Deletion of SNR-1, a component of cytoplasmic messenger ribonucleoprotein granules (cmRNPgs) that include PBs and stress granules (SGs), and the PB motif on one of the cTIC mRNAs, zip-1, significantly alters zip-1 rhythmic translation. These results reveal that the clock regulates rhythmic translation of specific mRNAs through rhythmic eIF2alpha activity and cmRNPg metabolism.\",\n",
       "  'Keywords': ['CP: Molecular biology',\n",
       "   'CPC-3',\n",
       "   'GCN2',\n",
       "   'Neurospora crassa',\n",
       "   'P-body',\n",
       "   'circadian clock',\n",
       "   'cytoplasmic messenger ribonucleoprotein granules',\n",
       "   'eIF2alpha',\n",
       "   'ribosome profiling',\n",
       "   'translation control'],\n",
       "  'Journal': 'Cell reports',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2022 Dec 27'},\n",
       " {'PMID': '35913159',\n",
       "  'Title': 'Transcriptional Regulation by the Velvet Protein VE-1 during Asexual Development in the Fungus Neurospora crassa.',\n",
       "  'Abstract': 'Asexual reproduction in fungi facilitates the dispersal and colonization of new substrates and, in pathogenic fungi, allows infection of plants and animals. The velvet complex is a fungus-specific protein complex that participates in the regulation of gene expression in response to environmental signals like light, as well as developmental processes, pathogenesis, and secondary metabolism. The velvet complex in the fungus Neurospora crassa is composed of three proteins, VE-1, VE-2, and LAE-1. Mutations in ve-1 or ve-2, but not in lae-1, led to shorter heights of aerial tissue, a mixture of aerial hyphae and developing macroconidia, and increased microconidiation when they were combined with mutations in the transcription factor gene fl. VE-2 and LAE-1 were detected during vegetative growth and conidiation, unlike VE-1, which was mostly observed in samples obtained from submerged vegetative hyphae. We propose that VE-1 is the limiting component of the velvet complex during conidiation and has a major role in the transcriptional regulation of conidiation. Characterization of the role of VE-1 during mycelial growth and asexual development (conidiation) by transcriptome sequencing (RNA-seq) experiments allowed the identification of a set of genes regulated by VE-1 that participate in the regulation of conidiation, most notably the transcription factor genes vib-1 and fl. We propose that VE-1 and VE-2 regulate the development of aerial tissue and the balance between macro- and microconidiation in coordination with FL and VIB-1. IMPORTANCE Most fungi disperse in nature and infect new hosts by producing vegetative spores or conidia during asexual development. This is a process that is regulated by environmental signals like light and the availability of nutrients. A protein complex, the velvet complex, participates in the integration of environmental signals to regulate conidiation. We have found that a key component of this complex in the fungus Neurospora crassa, VE-1, has a major role in the regulation of transcription during conidiation. VE-1 regulates a large number of genes, including the genes for the transcription factors FL and VIB-1. Our results will help to understand how environmental signals are integrated in the fungal cell to regulate development.',\n",
       "  'Keywords': ['RNA-seq',\n",
       "   'conidiation',\n",
       "   'light',\n",
       "   'transcriptional regulation',\n",
       "   'velvet complex'],\n",
       "  'Journal': 'mBio',\n",
       "  'Country': 'Spain',\n",
       "  'Year': '2022 Aug 30'},\n",
       " {'PMID': '35638725',\n",
       "  'Title': 'Secondary Metabolism Gene Clusters Exhibit Increasingly Dynamic and Differential Expression during Asexual Growth, Conidiation, and Sexual Development in Neurospora crassa.',\n",
       "  'Abstract': 'Secondary metabolite clusters (SMCs) encode the machinery for fungal toxin production. However, understanding their function and analyzing their products requires investigation of the developmental and environmental conditions in which they are expressed. Gene expression is often restricted to specific and unexamined stages of the life cycle. Therefore, we applied comparative genomics analyses to identify SMCs in Neurospora crassa and analyzed extensive transcriptomic data spanning nine independent experiments from diverse developmental and environmental conditions to reveal their life cycle-specific gene expression patterns. We reported 20 SMCs comprising 177 genes-a manageable set for investigation of the roles of SMCs across the life cycle of the fungal model N. crassa-as well as gene sets coordinately expressed in 18 predicted SMCs during asexual and sexual growth under three nutritional and two temperature conditions. Divergent activity of SMCs between asexual and sexual development was reported. Of 126 SMC genes that we examined for knockout phenotypes, al-2 and al-3 exhibited phenotypes in asexual growth and conidiation, whereas os-5, poi-2, and pmd-1 exhibited phenotypes in sexual development. SMCs with annotated function in mating and crossing were actively regulated during the switch between asexual and sexual growth. Our discoveries call for attention to roles that SMCs may play in the regulatory switches controlling mode of development, as well as the ecological associations of those developmental stages that may influence expression of SMCs. IMPORTANCE Secondary metabolites (SMs) are low-molecular-weight compounds that often mediate interactions between fungi and their environments. Fungi enriched with SMs are of significant research interest to agriculture and medicine, especially from the aspects of pathogen ecology and environmental epidemiology. However, SM clusters (SMCs) that have been predicted by comparative genomics alone have typically been poorly defined and insufficiently functionally annotated. Therefore, we have investigated coordinate expression in SMCs in the model system N. crassa, and our results suggest that SMCs respond to environmental signals and to stress that are associated with development. This study examined SMC regulation at the level of RNA to integrate observations and knowledge of these genes in various growth and development conditions, supporting combining comparative genomics and inclusive transcriptomics to improve computational annotation of SMCs. Our findings call for detailed study of the function of SMCs during the asexual-sexual switch, a key, often-overlooked developmental stage.',\n",
       "  'Keywords': ['Neurospora crassa',\n",
       "   'asexual development',\n",
       "   'environmental microbiology',\n",
       "   'filamentous fungi',\n",
       "   'gene cluster',\n",
       "   'secondary metabolism',\n",
       "   'sexual development',\n",
       "   'transcriptomics'],\n",
       "  'Journal': 'mSystems',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2022 Jun 28'},\n",
       " {'PMID': '35638725',\n",
       "  'Title': 'Secondary Metabolism Gene Clusters Exhibit Increasingly Dynamic and Differential Expression during Asexual Growth, Conidiation, and Sexual Development in Neurospora crassa.',\n",
       "  'Abstract': 'Secondary metabolite clusters (SMCs) encode the machinery for fungal toxin production. However, understanding their function and analyzing their products requires investigation of the developmental and environmental conditions in which they are expressed. Gene expression is often restricted to specific and unexamined stages of the life cycle. Therefore, we applied comparative genomics analyses to identify SMCs in Neurospora crassa and analyzed extensive transcriptomic data spanning nine independent experiments from diverse developmental and environmental conditions to reveal their life cycle-specific gene expression patterns. We reported 20 SMCs comprising 177 genes-a manageable set for investigation of the roles of SMCs across the life cycle of the fungal model N. crassa-as well as gene sets coordinately expressed in 18 predicted SMCs during asexual and sexual growth under three nutritional and two temperature conditions. Divergent activity of SMCs between asexual and sexual development was reported. Of 126 SMC genes that we examined for knockout phenotypes, al-2 and al-3 exhibited phenotypes in asexual growth and conidiation, whereas os-5, poi-2, and pmd-1 exhibited phenotypes in sexual development. SMCs with annotated function in mating and crossing were actively regulated during the switch between asexual and sexual growth. Our discoveries call for attention to roles that SMCs may play in the regulatory switches controlling mode of development, as well as the ecological associations of those developmental stages that may influence expression of SMCs. IMPORTANCE Secondary metabolites (SMs) are low-molecular-weight compounds that often mediate interactions between fungi and their environments. Fungi enriched with SMs are of significant research interest to agriculture and medicine, especially from the aspects of pathogen ecology and environmental epidemiology. However, SM clusters (SMCs) that have been predicted by comparative genomics alone have typically been poorly defined and insufficiently functionally annotated. Therefore, we have investigated coordinate expression in SMCs in the model system N. crassa, and our results suggest that SMCs respond to environmental signals and to stress that are associated with development. This study examined SMC regulation at the level of RNA to integrate observations and knowledge of these genes in various growth and development conditions, supporting combining comparative genomics and inclusive transcriptomics to improve computational annotation of SMCs. Our findings call for detailed study of the function of SMCs during the asexual-sexual switch, a key, often-overlooked developmental stage.',\n",
       "  'Keywords': ['Neurospora crassa',\n",
       "   'asexual development',\n",
       "   'environmental microbiology',\n",
       "   'filamentous fungi',\n",
       "   'gene cluster',\n",
       "   'secondary metabolism',\n",
       "   'sexual development',\n",
       "   'transcriptomics'],\n",
       "  'Journal': 'mSystems',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2022 Jun 28'},\n",
       " {'PMID': '35638725',\n",
       "  'Title': 'Secondary Metabolism Gene Clusters Exhibit Increasingly Dynamic and Differential Expression during Asexual Growth, Conidiation, and Sexual Development in Neurospora crassa.',\n",
       "  'Abstract': 'Secondary metabolite clusters (SMCs) encode the machinery for fungal toxin production. However, understanding their function and analyzing their products requires investigation of the developmental and environmental conditions in which they are expressed. Gene expression is often restricted to specific and unexamined stages of the life cycle. Therefore, we applied comparative genomics analyses to identify SMCs in Neurospora crassa and analyzed extensive transcriptomic data spanning nine independent experiments from diverse developmental and environmental conditions to reveal their life cycle-specific gene expression patterns. We reported 20 SMCs comprising 177 genes-a manageable set for investigation of the roles of SMCs across the life cycle of the fungal model N. crassa-as well as gene sets coordinately expressed in 18 predicted SMCs during asexual and sexual growth under three nutritional and two temperature conditions. Divergent activity of SMCs between asexual and sexual development was reported. Of 126 SMC genes that we examined for knockout phenotypes, al-2 and al-3 exhibited phenotypes in asexual growth and conidiation, whereas os-5, poi-2, and pmd-1 exhibited phenotypes in sexual development. SMCs with annotated function in mating and crossing were actively regulated during the switch between asexual and sexual growth. Our discoveries call for attention to roles that SMCs may play in the regulatory switches controlling mode of development, as well as the ecological associations of those developmental stages that may influence expression of SMCs. IMPORTANCE Secondary metabolites (SMs) are low-molecular-weight compounds that often mediate interactions between fungi and their environments. Fungi enriched with SMs are of significant research interest to agriculture and medicine, especially from the aspects of pathogen ecology and environmental epidemiology. However, SM clusters (SMCs) that have been predicted by comparative genomics alone have typically been poorly defined and insufficiently functionally annotated. Therefore, we have investigated coordinate expression in SMCs in the model system N. crassa, and our results suggest that SMCs respond to environmental signals and to stress that are associated with development. This study examined SMC regulation at the level of RNA to integrate observations and knowledge of these genes in various growth and development conditions, supporting combining comparative genomics and inclusive transcriptomics to improve computational annotation of SMCs. Our findings call for detailed study of the function of SMCs during the asexual-sexual switch, a key, often-overlooked developmental stage.',\n",
       "  'Keywords': ['Neurospora crassa',\n",
       "   'asexual development',\n",
       "   'environmental microbiology',\n",
       "   'filamentous fungi',\n",
       "   'gene cluster',\n",
       "   'secondary metabolism',\n",
       "   'sexual development',\n",
       "   'transcriptomics'],\n",
       "  'Journal': 'mSystems',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2022 Jun 28'},\n",
       " {'PMID': '35257662',\n",
       "  'Title': 'The ACF chromatin-remodeling complex is essential for Polycomb repression.',\n",
       "  'Abstract': 'Establishing and maintaining appropriate gene repression is critical for the health and development of multicellular organisms. Histone H3 lysine 27 (H3K27) methylation is a chromatin modification associated with repressed facultative heterochromatin, but the mechanism of this repression remains unclear. We used a forward genetic approach to identify genes involved in transcriptional silencing of H3K27-methylated chromatin in the filamentous fungus Neurospora crassa. We found that the N. crassa homologs of ISWI (NCU03875) and ACF1 (NCU00164) are required for repression of a subset of H3K27-methylated genes and that they form an ACF chromatin-remodeling complex. This ACF complex interacts with chromatin throughout the genome, yet association with facultative heterochromatin is specifically promoted by the H3K27 methyltransferase, SET-7. H3K27-methylated genes that are upregulated when iswi or acf1 are deleted show a downstream shift of the +1 nucleosome, suggesting that proper nucleosome positioning is critical for repression of facultative heterochromatin. Our findings support a direct role of the ACF complex in Polycomb repression.',\n",
       "  'Keywords': ['H3K27 methylation',\n",
       "   'ISWI',\n",
       "   'N. crassa',\n",
       "   'PRC2',\n",
       "   'chromatin remodeler',\n",
       "   'chromosomes',\n",
       "   'facultative heterochromatin',\n",
       "   'gene expression',\n",
       "   'genetics',\n",
       "   'genomics',\n",
       "   'nucleosome positioning'],\n",
       "  'Journal': 'eLife',\n",
       "  'Country': 'United States',\n",
       "  'Year': '2022 Mar 8'},\n",
       " {'PMID': '35244156',\n",
       "  'Title': 'The genome organization of Neurospora crassa at high resolution uncovers principles of fungal chromosome topology.',\n",
       "  'Abstract': 'The eukaryotic genome must be precisely organized for its proper function, as genome topology impacts transcriptional regulation, cell division, replication, and repair, among other essential processes. Disruptions to human genome topology can lead to diseases, including cancer. The advent of chromosome conformation capture with high-throughput sequencing (Hi-C) to assess genome organization has revolutionized the study of nuclear genome topology; Hi-C has elucidated numerous genomic structures, including chromosomal territories, active/silent chromatin compartments, Topologically Associated Domains, and chromatin loops. While low-resolution heatmaps can provide important insights into chromosomal level contacts, high-resolution Hi-C datasets are required to reveal folding principles of individual genes. Of particular interest are high-resolution chromosome conformation datasets of organisms modeling the human genome. Here, we report the genome topology of the fungal model organism Neurospora crassa at a high resolution. Our composite Hi-C dataset, which merges 2 independent datasets generated with restriction enzymes that monitor euchromatin (DpnII) and heterochromatin (MseI), along with our DpnII/MseI double digest dataset, provide exquisite detail for both the conformation of entire chromosomes and the folding of chromatin at the resolution of individual genes. Within constitutive heterochromatin, we observe strong yet stochastic internal contacts, while euchromatin enriched with either activating or repressive histone post-translational modifications associates with constitutive heterochromatic regions, suggesting intercompartment contacts form to regulate transcription. Consistent with this, a strain with compromised heterochromatin experiences numerous changes in gene expression. Our high-resolution Neurospora Hi-C datasets are outstanding resources to the fungal community and provide valuable insights into higher organism genome topology.',\n",
       "  'Keywords': ['Neurospora',\n",
       "   'chromatin',\n",
       "   'genome organization',\n",
       "   'genome topology',\n",
       "   'in situ Hi-C'],\n",
       "  'Journal': 'G3 (Bethesda, Md.)',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2022 May 6'},\n",
       " {'PMID': '35330254',\n",
       "  'Title': 'Evaluating the Mode of Antifungal Action of Heat-Stable Antifungal Factor (HSAF) in Neurospora crassa.',\n",
       "  'Abstract': 'Heat-stable antifungal factor (HSAF) isolated from Lysobacter enzymogenes has shown a broad-spectrum of antifungal activities. However, little is known about its mode of action. In this study, we used the model filamentous fungus Neurospora crassa to investigate the antifungal mechanism of HSAF. We first used HSAF to treat the N. crassa strain at different time points. Spore germination, growth phenotype and differential gene expression analysis were conducted by utilizing global transcriptional profiling combined with genetic and physiological analyses. Our data showed that HSAF could significantly inhibit the germination and aerial hyphae growth of N. crassa. RNA-seq analysis showed that a group of genes, associated with cell wall formation and remodeling, were highly activated. Screening of N. crassa gene deletion mutants combined with scanning electron microscopic observation revealed that three fungal cell wall integrity-related genes played an important role in the interaction between N. crassa and L. enzymogens. In addition, Weighted Gene Co-Expression Network Analysis (WGCNA), accompanied by confocal microscopy observation revealed that HSAF could trigger autophagy-mediated degradation and eventually result in cell death in N. crassa. The findings of this work provided new insights into the interactions between the predatory Lysobacter and its fungal prey.',\n",
       "  'Keywords': ['HSAF',\n",
       "   'Neurospora crassa',\n",
       "   'antifungal',\n",
       "   'autophagy',\n",
       "   'cell wall',\n",
       "   'transcriptome'],\n",
       "  'Journal': 'Journal of fungi (Basel, Switzerland)',\n",
       "  'Country': 'China',\n",
       "  'Year': '2022 Mar 1'},\n",
       " {'PMID': '35333354',\n",
       "  'Title': 'H3K56 deacetylation and H2A.Z deposition are required for aberrant heterochromatin spreading.',\n",
       "  'Abstract': 'Crucial mechanisms are required to restrict self-propagating heterochromatin spreading within defined boundaries and prevent euchromatic gene silencing. In the filamentous fungus Neurospora crassa, the JmjC domain protein DNA METHYLATION MODULATOR-1 (DMM-1) prevents aberrant spreading of heterochromatin, but the molecular details remain unknown. Here, we revealed that DMM-1 is highly enriched in a well-defined 5-kb heterochromatin domain upstream of the cat-3 gene, hereby called 5H-cat-3 domain, to constrain aberrant heterochromatin spreading. Interestingly, aberrant spreading of the 5H-cat-3 domain observed in the dmm-1KO strain is accompanied by robust deposition of histone variant H2A.Z, and deletion of H2A.Z abolishes aberrant spreading of the 5H-cat-3 domain into adjacent euchromatin. Furthermore, lysine 56 of histone H3 is deacetylated at the expanded heterochromatin regions, and mimicking H3K56 acetylation with an H3K56Q mutation effectively blocks H2A.Z-mediated aberrant spreading of the 5H-cat-3 domain. Importantly, genome-wide analyses demonstrated the general roles of H3K56 deacetylation and H2A.Z deposition in aberrant spreading of heterochromatin. Together, our results illustrate a previously unappreciated regulatory process that mediates aberrant heterochromatin spreading.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Nucleic acids research',\n",
       "  'Country': 'China',\n",
       "  'Year': '2022 Apr 22'},\n",
       " {'PMID': '35196787',\n",
       "  'Title': 'Coordinated Regulation of Membrane Homeostasis and Drug Accumulation by Novel Kinase STK-17 in Response to Antifungal Azole Treatment.',\n",
       "  'Abstract': 'The emergence of antifungal resistance, especially to the most widely used azole class of ergosterol biosynthesis inhibitors, makes fungal infections difficult to treat in clinics and agriculture. When exposed to azoles, fungi can make adaptive responses to alleviate azole toxicity and produce azole tolerance. However, except for azole efflux pumps and ergosterol biosynthesis genes, the role of most azole responsive genes in azole resistance is unknown. In this study, STK-17, whose transcription is upregulated by azoles, was characterized as a novel kinase that is required for azole resistance. Deletion or dysfunction of STK-17 led to azole hypersensitivity in Neurospora crassa and to other ergosterol biosynthesis inhibitors such as amorolfine, terbinafine, and amphotericin B, but not fatty acid and ceramide biosynthesis inhibitors. STK-17 was also required for oxidative stress resistance, but this was not connected to azole resistance. RNA-seq results showed that stk-17 deletion affected the basal expression and the response to ketoconazole of some membrane protein genes, indicating functional association of STK-17 with the membrane. Notably, deletion of stk-17 affected the normal response to azoles of erg genes, including the azole target-encoding gene erg11, and erg2, erg6, and erg24, and led to abnormal accumulation of sterols in the presence of azoles. HPLC-MS/MS analysis revealed increased intracellular azole accumulation in the stk-17 mutant, possibly due to enhanced azole influx and reduced azole efflux that was independent of the major efflux pump CDR4. Importantly, STK-17 was widely distributed and functionally conserved among fungi, thus providing a potential antifungal target. IMPORTANCE Antifungal resistance is increasing worldwide, especially to the most widely used azole class of ergosterol biosynthesis inhibitors, making control of fungal infections more challenging. A lot of effort has been expended in elucidating the mechanism of azole resistance and revealing potential antifungal targets. In this study, by analyzing azole-responsive genes in Neurospora crassa, we discovered STK-17, a novel kinase, that is required for azole resistance in several types of fungi. It has a role in regulating membrane homeostasis, responses to azole by ergosterol biosynthesis genes and azole accumulation, thus, deepening our understanding on the mechanism of azole stress response. Additionally, STK-17 is conserved among fungi and plays important roles in fungal development and stress resistance. Kinase inhibitors are broadly used for treating diseases, and our study pinpoints a potential drug target for antifungal development.',\n",
       "  'Keywords': ['Fusarium verticilioides',\n",
       "   'Neurospora crassa',\n",
       "   'STK-17',\n",
       "   'azole accumulation',\n",
       "   'azole resistance',\n",
       "   'kinase',\n",
       "   'sterol biosynthesis'],\n",
       "  'Journal': 'Microbiology spectrum',\n",
       "  'Country': 'China',\n",
       "  'Year': '2022 Feb 23'},\n",
       " {'PMID': '34899653',\n",
       "  'Title': 'Coordinated Regulation of Protoperithecium Development by MAP Kinases MAK-1 and MAK-2 in Neurospora crassa.',\n",
       "  'Abstract': 'Mitogen-activated protein (MAP) kinase pathways function as signaling hubs that are integral for many essential cellular processes, including sexual development. The molecular mechanisms and cross-talk between PR and CWI MAP kinase pathways have been extensively studied during asexual development. However, if these can be extended to sexual development remains elusive. By analyzing genome-wide transcriptional responses to deletion of each of two MAP kinase coding genes mak-2 (PR-MAP kinase pathway) and mak-1 (CWI-MAP kinase pathway) in Neurospora crassa during protoperithecium formation, 430 genes co-regulated by the MAK-1 and MAK-2 proteins were found, functionally enriched at integral components of membrane and oxidoreductase. These genes include 13 functionally known genes participating in sexual development (app, poi-2, stk-17, fsd-1, vsd-8, and NCU03863) and melanin synthesis (per-1, pkh-1, pkh-2, mld-1, scy-1, trn-2, and trn-1), as well as a set of functionally unknown genes. Phenotypic analysis of deletion mutants for the functionally unknown genes revealed that 12 genes were essential for female fertility. Among them, single-gene deletion mutants for NCU07743 (named as pfd-1), NCU02250 (oli), and NCU05948 (named as pfd-2) displayed similar protoperithecium development defects as the Deltamak-1 and Deltamak-2 mutants, failing to form protoperithecium. Western blotting analysis showed that both phosphorylated and total MAK-1 proteins were virtually abolished in the Deltanrc-1, Deltamek-2, and Deltamak-2 mutants, suggesting that the posttranscriptional regulation of MAK-1 is dependent on the PR-MAP kinase pathway during the protoperithecium development. Taken together, this study revealed the regulatory roles and cross-talk between PR and CWI-MAP kinase pathways during protoperithecium development.',\n",
       "  'Keywords': ['MAK-1',\n",
       "   'MAK-2',\n",
       "   'MAP kinase',\n",
       "   'Neurospora crassa',\n",
       "   'protoperithecium',\n",
       "   'sexual development'],\n",
       "  'Journal': 'Frontiers in microbiology',\n",
       "  'Country': 'China',\n",
       "  'Year': '2021'},\n",
       " {'PMID': '34417614',\n",
       "  'Title': 'Codon usage and protein length-dependent feedback from translation elongation regulates translation initiation and elongation speed.',\n",
       "  'Abstract': 'Essential cellular functions require efficient production of many large proteins but synthesis of large proteins encounters many obstacles in cells. Translational control is mostly known to be regulated at the initiation step. Whether translation elongation process can feedback to regulate initiation efficiency is unclear. Codon usage bias, a universal feature of all genomes, plays an important role in determining gene expression levels. Here, we discovered that there is a conserved but codon usage-dependent genome-wide negative correlation between protein abundance and CDS length. The codon usage effects on protein expression and ribosome flux on mRNAs are influenced by CDS length; optimal codon usage preferentially promotes production of large proteins. Translation of mRNAs with long CDS and non-optimal codon usage preferentially induces phosphorylation of initiation factor eIF2alpha, which inhibits translation initiation efficiency. Deletion of the eIF2alpha kinase CPC-3 (GCN2 homolog) in Neurospora preferentially up-regulates large proteins encoded by non-optimal codons. Surprisingly, CPC-3 also inhibits translation elongation rate in a codon usage and CDS length-dependent manner, resulting in slow elongation rates for long CDS mRNAs. Together, these results revealed a codon usage and CDS length-dependent feedback mechanism from translation elongation to regulate both translation initiation and elongation kinetics.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Nucleic acids research',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2021 Sep 20'},\n",
       " {'PMID': '34523983',\n",
       "  'Title': 'Aspects of the Neurospora crassa Sulfur Starvation Response Are Revealed by Transcriptional Profiling and DNA Affinity Purification Sequencing.',\n",
       "  'Abstract': 'Accurate nutrient sensing is important for rapid fungal growth and exploitation of available resources. Sulfur is an important nutrient source found in a number of biological macromolecules, including proteins and lipids. The model filamentous fungus Neurospora crassa is capable of utilizing sulfur found in a variety of sources from amino acids to sulfate. During sulfur starvation, the transcription factor CYS-3 is responsible for upregulation of genes involved in sulfur uptake and assimilation. Using a combination of RNA sequencing and DNA affinity purification sequencing, we performed a global survey of the N. crassa sulfur starvation response and the role of CYS-3 in regulating sulfur-responsive genes. The CYS-3 transcription factor bound the promoters and regulated genes involved in sulfur metabolism. Additionally, CYS-3 directly activated the expression of a number of uncharacterized transporter genes, suggesting that regulation of sulfur import is an important aspect of regulation by CYS-3. CYS-3 also directly regulated the expression of genes involved in mitochondrial electron transfer. During sulfur starvation, genes involved in nitrogen metabolism, such as amino acid and nucleic acid metabolic pathways, along with genes encoding proteases and nucleases that are necessary for scavenging nitrogen, were activated. Sulfur starvation also caused changes in the expression of genes involved in carbohydrate metabolism, such as those encoding glycosyl hydrolases. Thus, our data suggest a connection between sulfur metabolism and other aspects of cellular metabolism. IMPORTANCE Identification of nutrients present in the environment is a challenge common to all organisms. Sulfur is an important nutrient source found in proteins, lipids, and electron carriers that are required for the survival of filamentous fungi such as Neurospora crassa. Here, we transcriptionally profiled the response of N. crassa to characterize the global response to sulfur starvation. We also used DNA affinity purification sequencing to identify the direct downstream targets of the transcription factor responsible for regulating genes involved in sulfur uptake and assimilation. Along with genes involved in sulfur metabolism, this transcription factor regulated a number of uncharacterized transporter genes and genes involved in mitochondrial electron transfer. Our data also suggest a connection between sulfur, nitrogen, and carbon metabolism, indicating that the regulation of a number of metabolic pathways is intertwined.',\n",
       "  'Keywords': ['DAP-seq',\n",
       "   'nutrient sensing',\n",
       "   'sulfur starvation response',\n",
       "   'transcriptional profiling'],\n",
       "  'Journal': 'mSphere',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2021 Oct 27'},\n",
       " {'PMID': '30474279',\n",
       "  'Title': 'CLR-4, a novel conserved transcription factor for cellulase gene expression in ascomycete fungi.',\n",
       "  'Abstract': 'Fungal degradation of lignocellulosic biomass requires various (hemi-)cellulases and is an important part of the natural carbon cycle. Although induction of cellulases has been described for some saprobic filamentous fungi, the regulation of cellulase transcription is complex and many aspects are still poorly understood. Here, we identified and characterized the novel cellulase regulation factor NcCLR-4 in Neurospora crassa and its ortholog MtCLR-4 in Myceliophthora thermophila. Deletion of CLR-4 resulted in similarly defective cellulolytic enzyme production and activities. Transcriptome analyses of DeltaNcclr-4/DeltaMtclr-4 revealed the down-regulation of genes encoding (hemi-)cellulases and pivotal regulators (clr-1, clr-2 and xyr-1) and key genes in the cAMP signaling pathway such as adenylate cyclase Nccr-1. Intracellular cAMP levels were markedly lower in DeltaNcclr-4/DeltaMtclr-4 than in wild-type during cellulose utilization. In electrophoretic mobility shift (EMSA) and DNase I footprinting assays, NcCLR-4/MtCLR-4 can directly bound to the promoters of Nccr-1/Mtcr-1 (encoding adenylyl cyclase). EMSAs also demonstrated that NcCLR-4/MtCLR-4 could directly bound to clr-1 (encoding a key cellulase regulator), Mtclr-2 and Mtxyr-1 (encoding biomass deconstruction regulators). These findings about the novel cellulase expression regulators NcCLR-4 and MtCLR-4 enrich our understanding of how cellulose degradation is regulated and provide new targets for engineering fungi to deconstruct plant biomass in biorefineries.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Molecular microbiology',\n",
       "  'Country': 'China',\n",
       "  'Year': '2019 Feb'},\n",
       " {'PMID': '33753477',\n",
       "  'Title': 'DNA affinity purification sequencing and transcriptional profiling reveal new aspects of nitrogen regulation in a filamentous fungus.',\n",
       "  'Abstract': 'Sensing available nutrients and efficiently utilizing them is a challenge common to all organisms. The model filamentous fungus Neurospora crassa is capable of utilizing a variety of inorganic and organic nitrogen sources. Nitrogen utilization in N. crassa is regulated by a network of pathway-specific transcription factors that activate genes necessary to utilize specific nitrogen sources in combination with nitrogen catabolite repression regulatory proteins. We identified an uncharacterized pathway-specific transcription factor, amn-1, that is required for utilization of the nonpreferred nitrogen sources proline, branched-chain amino acids, and aromatic amino acids. AMN-1 also plays a role in regulating genes involved in responding to the simple sugar mannose, suggesting an integration of nitrogen and carbon metabolism. The utilization of nonpreferred nitrogen sources, which require metabolic processing before being used as a nitrogen source, is also regulated by the nitrogen catabolite regulator NIT-2. Using RNA sequencing combined with DNA affinity purification sequencing, we performed a survey of the role of NIT-2 and the pathway-specific transcription factors NIT-4 and AMN-1 in directly regulating genes involved in nitrogen utilization. Although previous studies suggested promoter binding by both a pathway-specific transcription factor and NIT-2 may be necessary for activation of nitrogen-responsive genes, our data show that pathway-specific transcription factors regulate genes involved in the catabolism of specific nitrogen sources, while NIT-2 regulates genes involved in utilization of all nonpreferred nitrogen sources, such as nitrogen transporters. Together, these transcription factors form a nutrient sensing network that allows N. crassa cells to regulate nitrogen utilization.',\n",
       "  'Keywords': ['DAPseq',\n",
       "   'RNAseq',\n",
       "   'nitrogen utilization',\n",
       "   'nutrient sensing',\n",
       "   'transcriptional networks'],\n",
       "  'Journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'Country': 'CA 94720; huberman@cornell.edu Lglass@berkeley.edu',\n",
       "  'Year': '2021 Mar 30'},\n",
       " {'PMID': '33574802',\n",
       "  'Title': 'Impact of the White Collar Photoreceptor WcoA on the Fusarium fujikuroi Transcriptome.',\n",
       "  'Abstract': 'The proteins of the White Collar 1 family (WC) constitute a major class of flavin photoreceptors, widely distributed in fungi, that work in cooperation with a WC 2 protein forming a regulatory complex. The WC complex was investigated in great detail in Neurospora crassa, a model fungus in photobiology studies, where it controls all its major photoresponses. The fungus Fusarium fujikuroi, a model system in the production of secondary metabolites, contains a single WC-1 gene called wcoA. The best-known light response in this fungus is the photoinduction of the synthesis of carotenoids, terpenoid pigments with antioxidant properties. Loss of WcoA in F. fujikuroi results in a drastic reduction in the mRNA levels of the carotenoid genes, and a diversity of morphological and metabolic changes, including alterations in the synthesis of several secondary metabolites, suggesting a complex regulatory role. To investigate the function of WcoA, the transcriptome of F. fujikuroi was analyzed in the dark and after 15-, 60- or 240-min illumination in a wild strain and in a formerly investigated wcoA insertional mutant. Using a threshold of four-fold change in transcript levels, 298 genes were activated and 160 were repressed in the wild strain under at least one of the light exposures. Different response patterns were observed among them, with genes exhibiting either fast, intermediate, and slow photoinduction, or intermediate or slow repression. All the fast and intermediate photoresponses, and most of the slow ones, were lost in the wcoA mutant. However, the wcoA mutation altered the expression of a much larger number of genes irrespective of illumination, reaching at least 16% of the annotated genes in this fungus. Such genes include many related to secondary metabolism, as well as others related to photobiology and other cellular functions, including the production of hydrophobins. As judged by the massive transcriptomic changes exhibited by the wcoA mutant in the dark, the results point to WcoA as a master regulatory protein in F. fujikuroi, in addition to a central function as the photoreceptor responsible for most of the transcriptional responses to light in this fungus.',\n",
       "  'Keywords': ['Fusarium',\n",
       "   'carotenoids',\n",
       "   'photoreceptor',\n",
       "   'photoregulation',\n",
       "   'secondary metabolism',\n",
       "   'white collar protein'],\n",
       "  'Journal': 'Frontiers in microbiology',\n",
       "  'Country': 'Spain',\n",
       "  'Year': '2020'},\n",
       " {'PMID': '33468665',\n",
       "  'Title': 'IMITATION SWITCH is required for normal chromatin structure and gene repression in PRC2 target domains.',\n",
       "  'Abstract': 'Polycomb Group (PcG) proteins are part of an epigenetic cell memory system that plays essential roles in multicellular development, stem cell biology, X chromosome inactivation, and cancer. In animals, plants, and many fungi, Polycomb Repressive Complex 2 (PRC2) catalyzes trimethylation of histone H3 lysine 27 (H3K27me3) to assemble transcriptionally repressed facultative heterochromatin. PRC2 is structurally and functionally conserved in the model fungus Neurospora crassa, and recent work in this organism has generated insights into PRC2 control and function. To identify components of the facultative heterochromatin pathway, we performed a targeted screen of Neurospora deletion strains lacking individual ATP-dependent chromatin remodeling enzymes. We found the Neurospora homolog of IMITATION SWITCH (ISW) is critical for normal transcriptional repression, nucleosome organization, and establishment of typical histone methylation patterns in facultative heterochromatin domains. We also found that stable interaction between PRC2 and chromatin depends on ISW. A functional ISW ATPase domain is required for gene repression and normal H3K27 methylation. ISW homologs interact with accessory proteins to form multiple complexes with distinct functions. Using proteomics and molecular approaches, we identified three distinct Neurospora ISW-containing complexes. A triple mutant lacking three ISW accessory factors and disrupting multiple ISW complexes led to widespread up-regulation of PRC2 target genes and altered H3K27 methylation patterns, similar to an ISW-deficient strain. Taken together, our data show that ISW is a key component of the facultative heterochromatin pathway in Neurospora, and that distinct ISW complexes perform an apparently overlapping role to regulate chromatin structure and gene repression at PRC2 target domains.',\n",
       "  'Keywords': ['H3K27me3',\n",
       "   'IMITATION SWITCH (ISWI)',\n",
       "   'Polycomb Repressive Complex 2 (PRC2)',\n",
       "   'chromatin remodeling',\n",
       "   'facultative heterochromatin'],\n",
       "  'Journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'Country': 'GA 30602',\n",
       "  'Year': '2021 Jan 26'},\n",
       " {'PMID': '32357961',\n",
       "  'Title': 'A Light-Inducible Strain for Genome-Wide Histone Turnover Profiling in Neurospora crassa.',\n",
       "  'Abstract': 'In chromatin, nucleosomes are composed of  approximately 146 bp of DNA wrapped around a histone octamer, and are highly dynamic structures subject to remodeling and exchange. Histone turnover has previously been implicated in various processes including the regulation of chromatin accessibility, segregation of chromatin domains, and dilution of histone marks. Histones in different chromatin environments may turnover at different rates, possibly with functional consequences. Neurospora crassa sports a chromatin environment that is more similar to that of higher eukaryotes than yeasts, which have been utilized in the past to explore histone exchange. We constructed a simple light-inducible system to profile histone exchange in N. crassa on a 3xFLAG-tagged histone H3 under the control of the rapidly inducible vvd promoter. After induction with blue light, incorporation of tagged H3 into chromatin occurred within 20 min. Previous studies of histone turnover involved considerably longer incubation periods and relied on a potentially disruptive change of medium for induction. We used this reporter to explore replication-independent histone turnover at genes and examine changes in histone turnover at heterochromatin domains in different heterochromatin mutant strains. In euchromatin, H3-3xFLAG patterns were almost indistinguishable from that observed in wild-type in all mutant backgrounds tested, suggesting that loss of heterochromatin machinery has little effect on histone turnover in euchromatin. However, turnover at heterochromatin domains increased with loss of trimethylation of lysine 9 of histone H3 or HP1, but did not depend on DNA methylation. Our reporter strain provides a simple yet powerful tool to assess histone exchange across multiple chromatin contexts.',\n",
       "  'Keywords': ['Neurospora crassa heterochromatin',\n",
       "   'chromatin profiling',\n",
       "   'histone exchange',\n",
       "   'light induction'],\n",
       "  'Journal': 'Genetics',\n",
       "  'Country': 'Oregon 97403',\n",
       "  'Year': '2020 Jul'},\n",
       " {'PMID': '33295874',\n",
       "  'Title': 'PRD-2 directly regulates casein kinase I and counteracts nonsense-mediated decay in the Neurospora circadian clock.',\n",
       "  'Abstract': \"Circadian clocks in fungi and animals are driven by a functionally conserved transcription-translation feedback loop. In Neurospora crassa, negative feedback is executed by a complex of Frequency (FRQ), FRQ-interacting RNA helicase (FRH), and casein kinase I (CKI), which inhibits the activity of the clock's positive arm, the White Collar Complex (WCC). Here, we show that the prd-2 (period-2) gene, whose mutation is characterized by recessive inheritance of a long 26 hr period phenotype, encodes an RNA-binding protein that stabilizes the ck-1a transcript, resulting in CKI protein levels sufficient for normal rhythmicity. Moreover, by examining the molecular basis for the short circadian period of upf-1(prd-6) mutants, we uncovered a strong influence of the Nonsense Mediated Decay pathway on CKI levels. The finding that circadian period defects in two classically derived Neurospora clock mutants each arise from disruption of ck-1a regulation is consistent with circadian period being exquisitely sensitive to levels of casein kinase I.\",\n",
       "  'Keywords': ['Casein Kinase I',\n",
       "   'N. crassa',\n",
       "   'RNA-binding protein',\n",
       "   'SUZ domain',\n",
       "   'circadian clock',\n",
       "   'genetics',\n",
       "   'genomics',\n",
       "   'neurospora',\n",
       "   'nonsense mediated decay',\n",
       "   'prd-2, upf1 / prd-6'],\n",
       "  'Journal': 'eLife',\n",
       "  'Country': 'United States',\n",
       "  'Year': '2020 Dec 9'},\n",
       " {'PMID': '34147068',\n",
       "  'Title': 'Chromatin accessibility profiling in Neurospora crassa reveals molecular features associated with accessible and inaccessible chromatin.',\n",
       "  'Abstract': 'BACKGROUND: Regulation of chromatin accessibility and transcription are tightly coordinated processes. Studies in yeast and higher eukaryotes have described accessible chromatin regions, but little work has been done in filamentous fungi. RESULTS: Here we present a genome-scale characterization of accessible chromatin regions in Neurospora crassa, which revealed characteristic molecular features of accessible and inaccessible chromatin. We present experimental evidence of inaccessibility within heterochromatin regions in Neurospora, and we examine features of both accessible and inaccessible chromatin, including the presence of histone modifications, types of transcription, transcription factor binding, and relative nucleosome turnover rates. Chromatin accessibility is not strictly correlated with expression level. Accessible chromatin regions in the model filamentous fungus Neurospora are characterized the presence of H3K27 acetylation and commonly associated with pervasive non-coding transcription. Conversely, methylation of H3 lysine-36 catalyzed by ASH1 is correlated with inaccessible chromatin within promoter regions. CONCLUSIONS: In N. crassa, H3K27 acetylation is the most predictive histone modification for open chromatin. Conversely, our data show that H3K36 methylation is a key marker of inaccessible chromatin in gene-rich regions of the genome. Our data are consistent with an expanded role for H3K36 methylation in intergenic regions of filamentous fungi compared to the model yeasts, S. cerevisiae and S. pombe, which lack homologs of the ASH1 methyltransferase.',\n",
       "  'Keywords': ['ATAC-seq', 'Chromatin accessibility', 'Fungi', 'H3K36me3'],\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2021 Jun 19'},\n",
       " {'PMID': '32873602',\n",
       "  'Title': 'Selection and Characterization of Mutants Defective in DNA Methylation in Neurospora crassa.',\n",
       "  'Abstract': 'DNA methylation, a prototypical epigenetic modification implicated in gene silencing, occurs in many eukaryotes and plays a significant role in the etiology of diseases such as cancer. The filamentous fungus Neurospora crassa places DNA methylation at regions of constitutive heterochromatin such as in centromeres and in other A:T-rich regions of the genome, but this modification is dispensable for normal growth and development. This and other features render N. crassa an excellent model to genetically dissect elements of the DNA methylation pathway. We implemented a forward genetic selection on a massive scale, utilizing two engineered antibiotic-resistance genes silenced by DNA methylation, to isolate mutants d efective i n m ethylation (dim). Hundreds of potential mutants were characterized, yielding a rich collection of informative alleles of 11 genes important for DNA methylation, most of which were already reported. In parallel, we characterized the pairwise interactions in nuclei of the DCDC, the only histone H3 lysine 9 methyltransferase complex in Neurospora, including those between the DIM-5 catalytic subunit and other complex members. We also dissected the N- and C-termini of the key protein DIM-7, required for DIM-5 histone methyltransferase localization and activation. Lastly, we identified two alleles of a novel gene, dim-10 - a homolog of Clr5 in Schizosaccharomyces pombe - that is not essential for DNA methylation, but is necessary for repression of the antibiotic-resistance genes used in the selection, which suggests that both DIM-10 and DNA methylation promote silencing of constitutive heterochromatin.',\n",
       "  'Keywords': ['H3K9me3',\n",
       "   'Neurospora',\n",
       "   'cytosine methylation',\n",
       "   'epigenetics',\n",
       "   'heterochromatin'],\n",
       "  'Journal': 'Genetics',\n",
       "  'Country': 'Oregon 97403',\n",
       "  'Year': '2020 Nov'},\n",
       " {'PMID': '33509260',\n",
       "  'Title': 'RCO-3 and COL-26 form an external-to-internal module that regulates the dual-affinity glucose transport system in Neurospora crassa.',\n",
       "  'Abstract': 'BACKGROUND: Low- and high-affinity glucose transport system is a conserved strategy of microorganism to cope with environmental glucose fluctuation for their growth and competitiveness. In Neurospora crassa, the dual-affinity glucose transport system consists of a low-affinity glucose transporter GLT-1 and two high-affinity glucose transporters HGT-1/HGT-2, which play diverse roles in glucose transport, carbon metabolism, and cellulase expression regulation. However, the regulation of this dual-transporter system in response to environmental glucose fluctuation is not yet clear. RESULTS: In this study, we report that a regulation module consisting of a downstream transcription factor COL-26 and an upstream non-transporting glucose sensor RCO-3 regulates the dual-affinity glucose transport system in N. crassa. COL-26 directly binds to the promoter regions of glt-1, hgt-1, and hgt-2, whereas RCO-3 is an upstream factor of the module whose deletion mutant resembles the Deltacol-26 mutant phenotypically. Transcriptional profiling analysis revealed that Deltacol-26 and Deltarco-3 mutants had similar transcriptional profiles, and both mutants had impaired response to a glucose gradient. We also showed that the AMP-activated protein kinase (AMPK) complex is involved in regulation of the glucose transporters. AMPK is required for repression of glt-1 expression in starvation conditions by inhibiting the activity of RCO-3. CONCLUSIONS: RCO-3 and COL-26 form an external-to-internal module that regulates the glucose dual-affinity transport system. Transcription factor COL-26 was identified as the key regulator. AMPK was also involved in the regulation of the dual-transporter system. Our findings provide novel insight into the molecular basis of glucose uptake and signaling in filamentous fungi, which may aid in the rational design of fungal strains for industrial purposes.',\n",
       "  'Keywords': ['Gene regulation',\n",
       "   'Glucose transport',\n",
       "   'Neurospora crassa',\n",
       "   'Phosphoproteome',\n",
       "   'RNA-seq',\n",
       "   'Transcription factor'],\n",
       "  'Journal': 'Biotechnology for biofuels',\n",
       "  'Country': 'China',\n",
       "  'Year': '2021 Jan 28'},\n",
       " {'PMID': '32946564',\n",
       "  'Title': 'LSD1 prevents aberrant heterochromatin formation in Neurospora crassa.',\n",
       "  'Abstract': 'Heterochromatin is a specialized form of chromatin that restricts access to DNA and inhibits genetic processes, including transcription and recombination. In Neurospora crassa, constitutive heterochromatin is characterized by trimethylation of lysine 9 on histone H3, hypoacetylation of histones, and DNA methylation. We explored whether the conserved histone demethylase, lysine-specific demethylase 1 (LSD1), regulates heterochromatin in Neurospora, and if so, how. Though LSD1 is implicated in heterochromatin regulation, its function is inconsistent across different systems; orthologs of LSD1 have been shown to either promote or antagonize heterochromatin expansion by removing H3K4me or H3K9me respectively. We identify three members of the Neurospora LSD complex (LSDC): LSD1, PHF1, and BDP-1. Strains deficient for any of these proteins exhibit variable spreading of heterochromatin and establishment of new heterochromatin domains throughout the genome. Although establishment of H3K9me3 is typically independent of DNA methylation in Neurospora, instances of DNA methylation-dependent H3K9me3 have been found outside regions of canonical heterochromatin. Consistent with this, the hyper-H3K9me3 phenotype of Deltalsd1 strains is dependent on the presence of DNA methylation, as well as HCHC-mediated histone deacetylation, suggesting that spreading is dependent on some feedback mechanism. Altogether, our results suggest LSD1 works in opposition to HCHC to maintain proper heterochromatin boundaries.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Nucleic acids research',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2020 Oct 9'},\n",
       " {'PMID': '33051373',\n",
       "  'Title': 'Nonoptimal Codon Usage Is Critical for Protein Structure and Function of the Master General Amino Acid Control Regulator CPC-1.',\n",
       "  'Abstract': 'Under amino acid starvation conditions, eukaryotic organisms activate a general amino acid control response. In Neurospora crassa, Cross Pathway Control Protein 1 (CPC-1), the ortholog of the Saccharomyces cerevisiae bZIP transcription factor GCN4, functions as the master regulator of the general amino acid control response. Codon usage biases are a universal feature of eukaryotic genomes and are critical for regulation of gene expression. Although codon usage has also been implicated in the regulation of protein structure and function, genetic evidence supporting this conclusion is very limited. Here, we show that Neurospora cpc-1 has a nonoptimal NNU-rich codon usage profile that contrasts with the strong NNC codon preference in the genome. Although substitution of the cpc-1 NNU codons with synonymous NNC codons elevated CPC-1 expression in Neurospora, it altered the CPC-1 degradation rate and abolished its amino acid starvation-induced protein stabilization. The codon-manipulated CPC-1 protein also exhibited different sensitivity to limited protease digestion. Furthermore, CPC-1 functions in rescuing the cell growth of the cpc-1 deletion mutant and activation of the expression of its target genes were impaired by the synonymous codon changes. Together, these results reveal the critical role of codon usage in regulation of CPC-1 expression and function and establish a genetic example of the importance of codon usage in protein folding.IMPORTANCE The general amino acid control response is critical for adaptation of organisms to amino acid starvation conditions. The preference to use certain synonymous codons is a universal feature of all genomes. Synonymous codon changes were previously thought to be silent mutations. In this study, we showed that the Neurospora cpc-1 gene has an unusual codon usage profile compared to other genes in the genome. We found that codon optimization of the cpc-1 gene without changing its amino acid sequence resulted in elevated CPC-1 expression, an altered protein degradation rate, and impaired protein functions due to changes in protein structure. Together, these results reveal the critical role of synonymous codon usage in regulation of CPC-1 expression and function and establish a genetic example of the importance of codon usage in protein structure.',\n",
       "  'Keywords': ['CPC-1',\n",
       "   'GCN4',\n",
       "   'Neurospora',\n",
       "   'codon usage',\n",
       "   'cotranslational protein folding',\n",
       "   'cross-pathway control',\n",
       "   'translation elongation'],\n",
       "  'Journal': 'mBio',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2020 Oct 13'},\n",
       " {'PMID': '32479508',\n",
       "  'Title': 'Adaptation of codon usage to tRNA I34 modification controls translation kinetics and proteome landscape.',\n",
       "  'Abstract': 'Codon usage bias is a universal feature of all genomes and plays an important role in regulating protein expression levels. Modification of adenosine to inosine at the tRNA anticodon wobble position (I34) by adenosine deaminases (ADATs) is observed in all eukaryotes and has been proposed to explain the correlation between codon usage and tRNA pool. However, how the tRNA pool is affected by I34 modification to influence codon usage-dependent gene expression is unclear. Using Neurospora crassa as a model system, by combining molecular, biochemical and bioinformatics analyses, we show that silencing of adat2 expression severely impaired the I34 modification levels for the ADAT-related tRNAs, resulting in major ADAT-related tRNA profile changes and reprogramming of translation elongation kinetics on ADAT-related codons. adat2 silencing also caused genome-wide codon usage-biased ribosome pausing on mRNAs and proteome landscape changes, leading to selective translational repression or induction of different mRNAs. The induced expression of CPC-1, the Neurospora ortholog of yeast GCN4p, mediates the transcriptional response after adat2 silencing and amino acid starvation. Together, our results demonstrate that the tRNA I34 modification by ADAT plays a major role in driving codon usage-biased translation to shape proteome landscape.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS genetics',\n",
       "  'Country': 'China',\n",
       "  'Year': '2020 Jun'},\n",
       " {'PMID': '32393638',\n",
       "  'Title': 'Evolutionarily ancient BAH-PHD protein mediates Polycomb silencing.',\n",
       "  'Abstract': 'Methylation of histone H3 lysine 27 (H3K27) is widely recognized as a transcriptionally repressive chromatin modification but the mechanism of repression remains unclear. We devised and implemented a forward genetic scheme to identify factors required for H3K27 methylation-mediated silencing in the filamentous fungus Neurospora crassa and identified a bromo-adjacent homology (BAH)-plant homeodomain (PHD)-containing protein, EPR-1 (effector of polycomb repression 1; NCU07505). EPR-1 associates with H3K27-methylated chromatin, and loss of EPR-1 de-represses H3K27-methylated genes without loss of H3K27 methylation. EPR-1 is not fungal-specific; orthologs of EPR-1 are present in a diverse array of eukaryotic lineages, suggesting an ancestral EPR-1 was a component of a primitive Polycomb repression pathway.',\n",
       "  'Keywords': ['H3K27 methylation',\n",
       "   'Polycomb repressive complex',\n",
       "   'epigenetics',\n",
       "   'facultative heterochromatin',\n",
       "   'histone reader'],\n",
       "  'Journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'Country': 'OR 97403',\n",
       "  'Year': '2020 May 26'},\n",
       " {'PMID': '32179551',\n",
       "  'Title': 'Identification of a PRC2 Accessory Subunit Required for Subtelomeric H3K27 Methylation in Neurospora crassa.',\n",
       "  'Abstract': 'Polycomb repressive complex 2 (PRC2) catalyzes methylation of histone H3 at lysine 27 (H3K27) in genomic regions of most eukaryotes and is critical for maintenance of the associated transcriptional repression. However, the mechanisms that shape the distribution of H3K27 methylation, such as recruitment of PRC2 to chromatin and/or stimulation of PRC2 activity, are unclear. Here, using a forward genetic approach in the model organism Neurospora crassa, we identified two alleles of a gene, NCU04278, encoding an unknown PRC2 accessory subunit (PAS). Loss of PAS resulted in losses of H3K27 methylation concentrated near the chromosome ends and derepression of a subset of associated subtelomeric genes. Immunoprecipitation followed by mass spectrometry confirmed reciprocal interactions between PAS and known PRC2 subunits, and sequence similarity searches demonstrated that PAS is not unique to N. crassa PAS homologs likely influence the distribution of H3K27 methylation and underlying gene repression in a variety of fungal lineages.',\n",
       "  'Keywords': ['H3K27me3',\n",
       "   'Polycomb repressive complex 2',\n",
       "   'facultative heterochromatin',\n",
       "   'non-core PRC2',\n",
       "   'telomeric silencing'],\n",
       "  'Journal': 'Molecular and cellular biology',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2020 May 14'},\n",
       " {'PMID': '31822585',\n",
       "  'Title': 'Integrative Activity of Mating Loci, Environmentally Responsive Genes, and Secondary Metabolism Pathways during Sexual Development of Chaetomium globosum.',\n",
       "  'Abstract': 'The origins and maintenance of the rich fungal diversity have been longstanding issues in evolutionary biology. To investigate how differences in expression regulation contribute to divergences in development and ecology among closely related species, transcriptomes were compared between Chaetomium globosum, a homothallic pathogenic fungus thriving in highly humid ecologies, and Neurospora crassa, a heterothallic postfire saprotroph. Gene expression was quantified in perithecia at nine distinct morphological stages during nearly synchronous sexual development. Unlike N. crassa, expression of all mating loci in C. globosum was highly correlated. Key regulators of the initiation of sexual development in response to light stimuli-including orthologs of N. crassasub-1, sub-1-dependent gene NCU00309, and asl-1-showed regulatory dynamics matching between C. globosum and N. crassa Among 24 secondary metabolism gene clusters in C. globosum, 11-including the cochliodones biosynthesis cluster-exhibited highly coordinated expression across perithecial development. C. globosum exhibited coordinately upregulated expression of histidine kinases in hyperosmotic response pathways-consistent with gene expression responses to high humidity we identified in fellow pathogen Fusarium graminearum Bayesian networks indicated that gene interactions during sexual development have diverged in concert with the capacities both to reproduce asexually and to live a self-compatible versus self-incompatible life cycle, shifting the hierarchical roles of genes associated with conidiation and heterokaryon incompatibility in N. crassa and C. globosum This divergence supports an evolutionary history of loss of conidiation due to unfavorable combinations of heterokaryon incompatibility in homothallic species.IMPORTANCE Fungal diversity has amazed evolutionary biologists for decades. One societally important aspect of this diversity manifests in traits that enable pathogenicity. The opportunistic pathogen Chaetomium globosum is well adapted to a high-humidity environment and produces numerous secondary metabolites that defend it from predation. Many of these chemicals can threaten human health. Understanding the phases of the C. globosum life cycle in which these products are made enables better control and even utilization of this fungus. Among its intriguing traits is that it both is self-fertile and lacks any means of propagule-based asexual reproduction. By profiling genome-wide gene expression across the process of sexual reproduction in C. globosum and comparing it to genome-wide gene expression in the model filamentous fungus N. crassa and other closely related fungi, we revealed associations among mating-type genes, sexual developmental genes, sexual incompatibility regulators, environmentally responsive genes, and secondary metabolic pathways.',\n",
       "  'Keywords': ['Bayesian network',\n",
       "   'environmental response',\n",
       "   'fungal pathogen',\n",
       "   'homothallism',\n",
       "   'secondary metabolism',\n",
       "   'sexual development',\n",
       "   'transcriptomics'],\n",
       "  'Journal': 'mBio',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2019 Dec 10'},\n",
       " {'PMID': '34041500',\n",
       "  'Title': 'Rapid and inexpensive preparation of genome-wide nucleosome footprints from model and non-model organisms.',\n",
       "  'Abstract': 'MNase-seq (micrococcal nuclease sequencing) is used to map nucleosome positions in eukaryotic genomes to study the relationship between chromatin structure and DNA-dependent processes. Current protocols require at least two days to isolate nucleosome-protected DNA fragments. We have developed a streamlined protocol for S. cerevisiae and other fungi which takes only three hours. Modified protocols were developed for wild fungi and mammalian cells. This method for rapidly producing sequencing-ready nucleosome footprints from several organisms makes MNase-seq faster and easier, with less chemical waste.',\n",
       "  'Keywords': ['Genomics',\n",
       "   'Model Organisms',\n",
       "   'Molecular Biology',\n",
       "   'Sequencing'],\n",
       "  'Journal': 'STAR protocols',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2021 Jun 18'},\n",
       " {'PMID': '31649059',\n",
       "  'Title': 'Identification and dynamic quantification of regulatory elements using total RNA.',\n",
       "  'Abstract': 'The spatial and temporal regulation of transcription initiation is pivotal for controlling gene expression. Here, we introduce capped-small RNA-seq (csRNA-seq), which uses total RNA as starting material to detect transcription start sites (TSSs) of both stable and unstable RNAs at single-nucleotide resolution. csRNA-seq is highly sensitive to acute changes in transcription and identifies an order of magnitude more regulated transcripts than does RNA-seq. Interrogating tissues from species across the eukaryotic kingdoms identified unstable transcripts resembling enhancer RNAs, pri-miRNAs, antisense transcripts, and promoter upstream transcripts in multicellular animals, plants, and fungi spanning 1.6 billion years of evolution. Integration of epigenomic data from these organisms revealed that histone H3 trimethylation (H3K4me3) was largely confined to TSSs of stable transcripts, whereas H3K27ac marked nucleosomes downstream from all active TSSs, suggesting an ancient role for posttranslational histone modifications in transcription. Our findings show that total RNA is sufficient to identify transcribed regulatory elements and capture the dynamics of initiated stable and unstable transcripts at single-nucleotide resolution in eukaryotes.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Genome research',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2019 Nov'},\n",
       " {'PMID': '31551996',\n",
       "  'Title': 'The pH Signaling Transcription Factor PAC-3 Regulates Metabolic and Developmental Processes in Pathogenic Fungi.',\n",
       "  'Abstract': 'The zinc finger transcription factor PAC-3/RIM101/PacC has a defined role in the secretion of enzymes and proteins in response to ambient pH, and also contributes to the virulence of species. Herein we evaluated the role of PAC-3 in the regulation of Neurospora crassa genes, in a model that examined the plant-fungi interactions. N. crassa is a model fungal species capable of exhibiting dynamic responses to its environment by employing endophytic or phytopathogenic behavior according to a given circumstance. Since plant growth and productivity are highly affected by pH and phosphorus (P) acquisition, we sought to verify the impact that induction of a Deltapac-3 mutation would have under limited and sufficient Pi availability, while ensuring that the targeted physiological adjustments mimicked ambient pH and nutritional conditions required for efficient fungal growth and development. Our results suggest direct regulatory functions for PAC-3 in cell wall biosynthesis, homeostasis, oxidation-reduction processes, hydrolase activity, transmembrane transport, and modulation of genes associated with fungal virulence. Pi-dependent modulation was observed mainly in genes encoding for transporter proteins or related to cell wall development, thereby advancing the current understanding regarding colonization and adaptation processes in response to challenging environments. We have also provided comprehensive evidence that suggests a role for PAC-3 as a global regulator in plant pathogenic fungi, thus presenting results that have the potential to be applied to various types of microbes, with diverse survival mechanisms.',\n",
       "  'Keywords': ['Neurospora crassa',\n",
       "   'RNA-sequencing',\n",
       "   'inorganic orthophosphate',\n",
       "   'mycorrhizal association',\n",
       "   'phytopathogen'],\n",
       "  'Journal': 'Frontiers in microbiology',\n",
       "  'Country': 'Brazil',\n",
       "  'Year': '2019'},\n",
       " {'PMID': '30979837',\n",
       "  'Title': 'The Blue-Light Photoreceptor Sfwc-1 Gene Regulates the Phototropic Response and Fruiting-Body Development in the Homothallic Ascomycete Sordaria fimicola.',\n",
       "  'Abstract': 'Sordaria fimicola, a coprophilous ascomycete, is a homothallic fungus that can undergo sexual differentiation with cellular and morphological changes followed by multicellular tissue development to complete its sexual cycle. In this study, we identified and characterized the blue-light photoreceptor gene in S. fimicola The S. fimicola white collar-1 photoreceptor (SfWC-1) contains light-oxygen-voltage-sensing (LOV), Per-Arnt-Sim (PAS), and other conserved domains and is homologous to the WC-1 blue-light photoreceptor of Neurospora crassa The LOV domain of Sfwc-1 was deleted by homologous recombination using Agrobacterium-mediated protoplast transformation. The Sfwc-1(()(Deltalov)()) mutant showed normal vegetative growth but produced less carotenoid pigment under illumination. The mutant showed delayed and less-pronounced fruiting-body formation, was defective in phototropism of the perithecial beaks, and lacked the fruiting-body zonation pattern compared with the wild type under the illumination condition. Gene expression analyses supported the light-induced functions of the Sfwc-1 gene in the physiology and developmental process of perithecial formation in S. fimicola Moreover, green fluorescent protein (GFP)-tagged SfWC-1 fluorescence signals were transiently strong upon light induction and prominently located inside the nuclei of living hyphae. Our studies focused on the putative blue-light photoreceptor in a model ascomycete and contribute to a better understanding of the photoregulatory functions and networks mediated by the evolutionarily conserved blue-light photoreceptors across diverse fungal phyla.IMPORTANCESordaria sp. has been a model for study of fruiting-body differentiation in fungi. Several environmental factors, including light, affect cellular and morphological changes during multicellular tissue development. Here, we created a light-oxygen-voltage-sensing (LOV) domain-deleted Sfwc-1 mutant to study blue-light photoresponses in Sordaria fimicola Phototropism and rhythmic zonation of perithecia were defective in the Sfwc-1(()(Deltalov)()) mutant. Moreover, fruiting-body development in the mutant was reduced and also significantly delayed. Gene expression analysis and subcellular localization study further revealed the light-induced differential gene expression and cellular responses upon light stimulation in S. fimicola.',\n",
       "  'Keywords': ['SfWC-1 photoreceptor',\n",
       "   'Sordaria fimicola',\n",
       "   'blue-light photoreceptor',\n",
       "   'perithecial beak'],\n",
       "  'Journal': 'Applied and environmental microbiology',\n",
       "  'Country': 'Taiwan',\n",
       "  'Year': '2019 Jun 15'},\n",
       " {'PMID': '31133749',\n",
       "  'Title': 'Similarity regression predicts evolution of transcription factor sequence specificity.',\n",
       "  'Abstract': 'Transcription factor (TF) binding specificities (motifs) are essential for the analysis of gene regulation. Accurate prediction of TF motifs is critical, because it is infeasible to assay all TFs in all sequenced eukaryotic genomes. There is ongoing controversy regarding the degree of motif diversification among related species that is, in part, because of uncertainty in motif prediction methods. Here we describe similarity regression, a significantly improved method for predicting motifs, which we use to update and expand the Cis-BP database. Similarity regression inherently quantifies TF motif evolution, and shows that previous claims of near-complete conservation of motifs between human and Drosophila are inflated, with nearly half of the motifs in each species absent from the other, largely due to extensive divergence in C2H2 zinc finger proteins. We conclude that diversification in DNA-binding motifs is pervasive, and present a new tool and updated resource to study TF diversity and gene regulation across eukaryotes.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Nature genetics',\n",
       "  'Country': 'Canada',\n",
       "  'Year': '2019 Jun'},\n",
       " {'PMID': '30778188',\n",
       "  'Title': 'Diversity of cytosine methylation across the fungal tree of life.',\n",
       "  'Abstract': 'The generation of thousands of fungal genomes is leading to a better understanding of genes and genomic organization within the kingdom. However, the epigenome, which includes DNA and chromatin modifications, remains poorly investigated in fungi. Large comparative studies in animals and plants have deepened our understanding of epigenomic variation, particularly of the modified base 5-methylcytosine (5mC), but taxonomic sampling of disparate groups is needed to develop unifying explanations for 5mC variation. Here, we utilize the largest phylogenetic resolution of 5mC methyltransferases (5mC MTases) and genome evolution to better understand levels and patterns of 5mC across fungi. We show that extant 5mC MTase genotypes are descendent from ancestral maintenance and de novo genotypes, whereas the 5mC MTases DIM-2 and RID are more recently derived, and that 5mC levels are correlated with 5mC MTase genotype and transposon content. Our survey also revealed that fungi lack canonical gene-body methylation, which distinguishes fungal epigenomes from certain insect and plant species. However, some fungal species possess independently derived clusters of contiguous 5mC encompassing many genes. In some cases, DNA repair pathways and the N(6)-methyladenine DNA modification negatively coevolved with 5mC pathways, which additionally contributed to interspecific epigenomic variation across fungi.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Nature ecology & evolution',\n",
       "  'Country': 'USA. bewickaj@uga.edu',\n",
       "  'Year': '2019 Mar'},\n",
       " {'PMID': '30554169',\n",
       "  'Title': 'Nucleosome Positioning by an Evolutionarily Conserved Chromatin Remodeler Prevents Aberrant DNA Methylation in Neurospora.',\n",
       "  'Abstract': 'In the filamentous fungus Neurospora crassa, constitutive heterochromatin is marked by tri-methylation of histone H3 lysine 9 (H3K9me3) and DNA methylation. We identified mutations in the Neurospora defective in methylation-1 (dim-1) gene that cause defects in cytosine methylation and implicate a putative AAA-ATPase chromatin remodeler. Although it was well-established that chromatin remodelers can affect transcription by influencing DNA accessibility with nucleosomes, little was known about the role of remodelers on chromatin that is normally not transcribed, including regions of constitutive heterochromatin. We found that dim-1 mutants display both reduced DNA methylation in heterochromatic regions as well as increased DNA methylation and H3K9me3 in some intergenic regions associated with highly expressed genes. Deletion of dim-1 leads to atypically spaced nucleosomes throughout the genome and numerous changes in gene expression. DIM-1 localizes to both heterochromatin and intergenic regions that become hyper-methylated in dim-1 strains. Our findings indicate that DIM-1 normally positions nucleosomes in both heterochromatin and euchromatin and that the standard arrangement and density of nucleosomes is required for the proper function of heterochromatin machinery.',\n",
       "  'Keywords': ['CATP',\n",
       "   'DIM-1',\n",
       "   'DNA methylation',\n",
       "   'Neurospora crassa',\n",
       "   'heterochromatin',\n",
       "   'nucleosome'],\n",
       "  'Journal': 'Genetics',\n",
       "  'Country': 'Oregon 97403 selker@uoregon.edu aklocko@uccs.edu',\n",
       "  'Year': '2019 Feb'},\n",
       " {'PMID': '31068130',\n",
       "  'Title': 'Histone H3 lysine 4 methyltransferase is required for facultative heterochromatin at specific loci.',\n",
       "  'Abstract': 'BACKGROUND: Histone H3 lysine 4 tri-methylation (H3K4me3) and histone H3 lysine 9 tri-methylation (H3K9me3) are widely perceived to be opposing and often mutually exclusive chromatin modifications. However, both are needed for certain light-activated genes in Neurospora crassa (Neurospora), including frequency (frq) and vivid (vvd). Except for these 2 loci, little is known about how H3K4me3 and H3K9me3 impact and contribute to light-regulated gene expression. RESULTS: In this report, we performed a multi-dimensional genomic analysis to understand the role of H3K4me3 and H3K9me3 using the Neurospora light response as the system. RNA-seq on strains lacking H3 lysine 4 methyltransferase (KMT2/SET-1) and histone H3 lysine 9 methyltransferase (KMT1/DIM-5) revealed some light-activated genes had altered expression, but the light response was largely intact. Comparing these 2 mutants to wild-type (WT), we found that roughly equal numbers of genes showed elevated and reduced expression in the dark and the light making the environmental stimulus somewhat ancillary to the genome-wide effects. ChIP-seq experiments revealed H3K4me3 and H3K9me3 had only minor changes in response to light in WT, but there were notable alterations in H3K4me3 in Deltakmt1/Deltadim-5 and H3K9me3 in Deltakmt2/Deltaset-1 indicating crosstalk and redistribution between the modifications. Integrated analysis of the RNA-seq and ChIP-seq highlighted context-dependent roles for KMT2/SET1 and KMT1/DIM-5 as either co-activators or co-repressors with some overlap as co-regulators. At a small subset of loci, H3K4 methylation is required for H3K9me3-mediated facultative heterochromatin including, the central clock gene frequency (frq). Finally, we used sequential ChIP (re-ChIP) experiment to confirm Neurospora contains K4/K9 bivalent domains. CONCLUSIONS: Collectively, these data indicate there are obfuscated regulatory roles for H3K4 methylation and H3K9 methylation depending on genome location with some minor overlap and co-dependency.',\n",
       "  'Keywords': ['Heterochromatin',\n",
       "   'Histone H3 lysine 4 methylation',\n",
       "   'Histone H3 lysine 9 methylation',\n",
       "   'Histone lysine methyltransferase',\n",
       "   'Light-activated gene expression',\n",
       "   'Long non-coding RNA'],\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2019 May 8'},\n",
       " {'PMID': '31068130',\n",
       "  'Title': 'Histone H3 lysine 4 methyltransferase is required for facultative heterochromatin at specific loci.',\n",
       "  'Abstract': 'BACKGROUND: Histone H3 lysine 4 tri-methylation (H3K4me3) and histone H3 lysine 9 tri-methylation (H3K9me3) are widely perceived to be opposing and often mutually exclusive chromatin modifications. However, both are needed for certain light-activated genes in Neurospora crassa (Neurospora), including frequency (frq) and vivid (vvd). Except for these 2 loci, little is known about how H3K4me3 and H3K9me3 impact and contribute to light-regulated gene expression. RESULTS: In this report, we performed a multi-dimensional genomic analysis to understand the role of H3K4me3 and H3K9me3 using the Neurospora light response as the system. RNA-seq on strains lacking H3 lysine 4 methyltransferase (KMT2/SET-1) and histone H3 lysine 9 methyltransferase (KMT1/DIM-5) revealed some light-activated genes had altered expression, but the light response was largely intact. Comparing these 2 mutants to wild-type (WT), we found that roughly equal numbers of genes showed elevated and reduced expression in the dark and the light making the environmental stimulus somewhat ancillary to the genome-wide effects. ChIP-seq experiments revealed H3K4me3 and H3K9me3 had only minor changes in response to light in WT, but there were notable alterations in H3K4me3 in Deltakmt1/Deltadim-5 and H3K9me3 in Deltakmt2/Deltaset-1 indicating crosstalk and redistribution between the modifications. Integrated analysis of the RNA-seq and ChIP-seq highlighted context-dependent roles for KMT2/SET1 and KMT1/DIM-5 as either co-activators or co-repressors with some overlap as co-regulators. At a small subset of loci, H3K4 methylation is required for H3K9me3-mediated facultative heterochromatin including, the central clock gene frequency (frq). Finally, we used sequential ChIP (re-ChIP) experiment to confirm Neurospora contains K4/K9 bivalent domains. CONCLUSIONS: Collectively, these data indicate there are obfuscated regulatory roles for H3K4 methylation and H3K9 methylation depending on genome location with some minor overlap and co-dependency.',\n",
       "  'Keywords': ['Heterochromatin',\n",
       "   'Histone H3 lysine 4 methylation',\n",
       "   'Histone H3 lysine 9 methylation',\n",
       "   'Histone lysine methyltransferase',\n",
       "   'Light-activated gene expression',\n",
       "   'Long non-coding RNA'],\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2019 May 8'},\n",
       " {'PMID': '31068130',\n",
       "  'Title': 'Histone H3 lysine 4 methyltransferase is required for facultative heterochromatin at specific loci.',\n",
       "  'Abstract': 'BACKGROUND: Histone H3 lysine 4 tri-methylation (H3K4me3) and histone H3 lysine 9 tri-methylation (H3K9me3) are widely perceived to be opposing and often mutually exclusive chromatin modifications. However, both are needed for certain light-activated genes in Neurospora crassa (Neurospora), including frequency (frq) and vivid (vvd). Except for these 2 loci, little is known about how H3K4me3 and H3K9me3 impact and contribute to light-regulated gene expression. RESULTS: In this report, we performed a multi-dimensional genomic analysis to understand the role of H3K4me3 and H3K9me3 using the Neurospora light response as the system. RNA-seq on strains lacking H3 lysine 4 methyltransferase (KMT2/SET-1) and histone H3 lysine 9 methyltransferase (KMT1/DIM-5) revealed some light-activated genes had altered expression, but the light response was largely intact. Comparing these 2 mutants to wild-type (WT), we found that roughly equal numbers of genes showed elevated and reduced expression in the dark and the light making the environmental stimulus somewhat ancillary to the genome-wide effects. ChIP-seq experiments revealed H3K4me3 and H3K9me3 had only minor changes in response to light in WT, but there were notable alterations in H3K4me3 in Deltakmt1/Deltadim-5 and H3K9me3 in Deltakmt2/Deltaset-1 indicating crosstalk and redistribution between the modifications. Integrated analysis of the RNA-seq and ChIP-seq highlighted context-dependent roles for KMT2/SET1 and KMT1/DIM-5 as either co-activators or co-repressors with some overlap as co-regulators. At a small subset of loci, H3K4 methylation is required for H3K9me3-mediated facultative heterochromatin including, the central clock gene frequency (frq). Finally, we used sequential ChIP (re-ChIP) experiment to confirm Neurospora contains K4/K9 bivalent domains. CONCLUSIONS: Collectively, these data indicate there are obfuscated regulatory roles for H3K4 methylation and H3K9 methylation depending on genome location with some minor overlap and co-dependency.',\n",
       "  'Keywords': ['Heterochromatin',\n",
       "   'Histone H3 lysine 4 methylation',\n",
       "   'Histone H3 lysine 9 methylation',\n",
       "   'Histone lysine methyltransferase',\n",
       "   'Light-activated gene expression',\n",
       "   'Long non-coding RNA'],\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2019 May 8'},\n",
       " {'PMID': '30468429',\n",
       "  'Title': 'ASH1-catalyzed H3K36 methylation drives gene repression and marks H3K27me2/3-competent chromatin.',\n",
       "  'Abstract': 'Methylation of histone H3 at lysine 36 (H3K36me), a widely-distributed chromatin mark, largely results from association of the lysine methyltransferase (KMT) SET-2 with RNA polymerase II (RNAPII), but most eukaryotes also have additional H3K36me KMTs that act independently of RNAPII. These include the orthologs of ASH1, which are conserved in animals, plants, and fungi but whose function and control are poorly understood. We found that Neurospora crassa has just two H3K36 KMTs, ASH1 and SET-2, and were able to explore the function and distribution of each enzyme independently. While H3K36me deposited by SET-2 marks active genes, inactive genes are modified by ASH1 and its activity is critical for their repression. ASH1-marked chromatin can be further modified by methylation of H3K27, and ASH1 catalytic activity modulates the accumulation of H3K27me2/3 both positively and negatively. These findings provide new insight into ASH1 function, H3K27me2/3 establishment, and repression in facultative heterochromatin.',\n",
       "  'Keywords': ['H3K27 methylation',\n",
       "   'H3K36 methylation',\n",
       "   'N. crassa',\n",
       "   'Set2',\n",
       "   'chromosomes',\n",
       "   'gene expression',\n",
       "   'gene silencing',\n",
       "   'genetics',\n",
       "   'genomics',\n",
       "   'heterochromatin',\n",
       "   'transcription'],\n",
       "  'Journal': 'eLife',\n",
       "  'Country': 'United States',\n",
       "  'Year': '2018 Nov 23'},\n",
       " {'PMID': '30472532',\n",
       "  'Title': 'Ultradian Rhythms in the Transcriptome of Neurospora crassa.',\n",
       "  'Abstract': 'In many organisms, the circadian clock drives rhythms in the transcription of clock-controlled genes that can be either circadian ( approximately 24-hr period) or ultradian (<24-hr period). Ultradian rhythms with periods that are a fraction of 24 hr are termed harmonics. Several harmonic transcripts were discovered in the mouse liver, but their functional significance remains unclear. Using a model-based analysis, we report for the first time  approximately 7-hr third harmonic transcripts in Neurospora crassa, a well-established fungal circadian model organism. Several third harmonic genes are regulated by female fertility 7 (FF-7), whose transcript itself is third harmonic. The knockout of circadian output regulator CSP1 superimposes circadian rhythms on the third harmonic genes, whereas the knockout of stress response regulator MSN1 converts third harmonic rhythms to second harmonic rhythms. The 460  approximately 7-hr genes are co-regulated in two anti-phasic groups in multiple genotypes and include kinases, chromatin remodelers, and homologs of harmonic genes in the mouse liver.',\n",
       "  'Keywords': ['Biological Sciences', 'Chronobiology', 'Transcriptomics'],\n",
       "  'Journal': 'iScience',\n",
       "  'Country': 'Germany. Electronic address: bharath.ananth@charite.de',\n",
       "  'Year': '2018 Nov 30'},\n",
       " {'PMID': '28972146',\n",
       "  'Title': 'Role for ribosome-associated complex and stress-seventy subfamily B (RAC-Ssb) in integral membrane protein translation.',\n",
       "  'Abstract': 'Targeting of most integral membrane proteins to the endoplasmic reticulum is controlled by the signal recognition particle, which recognizes a hydrophobic signal sequence near the protein N terminus. Proper folding of these proteins is monitored by the unfolded protein response and involves protein degradation pathways to ensure quality control. Here, we identify a new pathway for quality control of major facilitator superfamily transporters that occurs before the first transmembrane helix, the signal sequence recognized by the signal recognition particle, is made by the ribosome. Increased rates of translation elongation of the N-terminal sequence of these integral membrane proteins can divert the nascent protein chains to the ribosome-associated complex and stress-seventy subfamily B chaperones. We also show that quality control of integral membrane proteins by ribosome-associated complex-stress-seventy subfamily B couples translation rate to the unfolded protein response, which has implications for understanding mechanisms underlying human disease and protein production in biotechnology.',\n",
       "  'Keywords': ['chaperone',\n",
       "   'major facilitator superfamily (MFS)',\n",
       "   'membrane protein',\n",
       "   'protein synthesis',\n",
       "   'ribosome associated complex (RAC)',\n",
       "   'stress-seventy subfamily B (Ssb)',\n",
       "   'translation control',\n",
       "   'unfolded protein response (UPR)'],\n",
       "  'Journal': 'The Journal of biological chemistry',\n",
       "  'Country': 'From the Departments of Molecular and Cell Biology and',\n",
       "  'Year': '2017 Dec 1'},\n",
       " {'PMID': '29668735',\n",
       "  'Title': 'mus-52 disruption and metabolic regulation in Neurospora crassa: Transcriptional responses to extracellular phosphate availability.',\n",
       "  'Abstract': 'Advances in the understanding of molecular systems depend on specific tools like the disruption of genes to produce strains with the desired characteristics. The disruption of any mutagen sensitive (mus) genes in the model fungus Neurospora crassa, i.e. mus-51, mus-52, or mus-53, orthologous to the human genes KU70, KU80, and LIG4, respectively, provides efficient tools for gene targeting. Accordingly, we used RNA-sequencing and reverse transcription-quantitative polymerase chain reaction amplification techniques to evaluate the effects of mus-52 deletion in N. crassa gene transcriptional modulation, and thus, infer its influence regarding metabolic response to extracellular availability of inorganic phosphate (Pi). Notably, the absence of MUS-52 affected the transcription of a vast number of genes, highlighting the expression of those coding for transcription factors, kinases, circadian clocks, oxi-reduction balance, and membrane- and nucleolus-related proteins. These findings may provide insights toward the KU molecular mechanisms, which have been related to telomere maintenance, apoptosis, DNA replication, and gene transcription regulation, as well as associated human conditions including immune system disorders, cancer, and aging.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PloS one',\n",
       "  'Country': 'Brazil',\n",
       "  'Year': '2018'},\n",
       " {'PMID': '29908565',\n",
       "  'Title': 'Disruption of gul-1 decreased the culture viscosity and improved protein secretion in the filamentous fungus Neurospora crassa.',\n",
       "  'Abstract': 'BACKGROUND: The cellulolytic fungus Neurospora crassa is considered a potential host for enzyme and bioethanol production. However, large scale applications are hindered by its filamentous growth. Although previous investigations have shown that mycelial morphology in submerged culture can be controlled by altering physical factors, there is little knowledge available about the potential for morphology control by genetic modification. RESULTS: In this study, we screened morphological mutants in the filamentous fungus N. crassa. Of the 90 morphological mutants screened, 14 mutants exhibited considerably higher viscosity compared with that of the wild type strain, and only two mutants showed low-viscosity morphologies in submerged culture. We observed that disruption of gul-1 (NCU01197), which encodes an mRNA binding protein involved in cell wall remodeling, caused pellet formation as the fermentation progressed, and resulted in the most significant decrease in viscosity of culture broth. Moreover, over-expression of gul-1 caused dramatically increased viscosity, suggesting that the gul-1 had an important function in mycelial morphology during submerged cultivation. Transcriptional profiling showed that expression of genes encoding eight GPI-anchored cell wall proteins was lowered in Deltagul-1 while expression of genes associated with two non-anchored cell wall proteins was elevated. Meanwhile, the expression levels of two hydrophobin genes were also significantly altered. These results suggested that GUL-1 affected the transcription of cell wall-related genes, thereby influencing cell wall structure and mycelial morphology. Additionally, the deletion of gul-1 caused increased protein secretion, probably due to a defect in cell wall integrity, suggesting this as an alternative strategy of strain improvement for enzyme production. To confirm practical applications, deletion of gul-1 in the hyper-cellulase producing strain (∆ncw-1∆Ncap3m) significantly reduced the viscosity of culture broth. CONCLUSIONS: Using the model filamentous fungus N. crassa, genes that affect mycelial morphology in submerged culture were explored through systematic screening of morphological mutants. Disrupting several candidate genes altered viscosities in submerged culture. This work provides an example for controlling fungal morphology in submerged fermentation by genetic engineering, and will be beneficial for industrial fungal strain improvement.',\n",
       "  'Keywords': ['Mycelial morphology',\n",
       "   'Neurospora crassa',\n",
       "   'Pellet',\n",
       "   'Protein secretion',\n",
       "   'Viscosity'],\n",
       "  'Journal': 'Microbial cell factories',\n",
       "  'Country': 'China',\n",
       "  'Year': '2018 Jun 16'},\n",
       " {'PMID': '29258423',\n",
       "  'Title': 'The coding and noncoding transcriptome of Neurospora crassa.',\n",
       "  'Abstract': 'BACKGROUND: Long non protein coding RNAs (lncRNAs) have been identified in many different organisms and cell types. Emerging examples emphasize the biological importance of these RNA species but their regulation and functions remain poorly understood. In the filamentous fungus Neurospora crassa, the annotation and characterization of lncRNAs is incomplete. RESULTS: We have performed a comprehensive transcriptome analysis of Neurospora crassa by using ChIP-seq, RNA-seq and polysome fractionation datasets. We have annotated and characterized 1478 long intergenic noncoding RNAs (lincRNAs) and 1056 natural antisense transcripts, indicating that 20% of the RNA Polymerase II transcripts of Neurospora are not coding for protein. Both classes of lncRNAs accumulate at lower levels than protein-coding mRNAs and they are considerably shorter. Our analysis showed that the vast majority of lincRNAs and antisense transcripts do not contain introns and carry less H3K4me2 modifications than similarly expressed protein coding genes. In contrast, H3K27me3 modifications inversely correlate with transcription of protein coding and lincRNA genes. We show furthermore most lincRNA sequences evolve rapidly, even between phylogenetically close species. CONCLUSIONS: Our transcriptome analyses revealed distinct features of Neurospora lincRNAs and antisense transcripts in comparison to mRNAs and showed that the prevalence of noncoding transcripts in this organism is higher than previously anticipated. The study provides a broad repertoire and a resource for further studies of lncRNAs.',\n",
       "  'Keywords': ['Antisense',\n",
       "   'Neurospora',\n",
       "   'Noncoding RNA',\n",
       "   'Splicing',\n",
       "   'Transcriptome'],\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'Germany',\n",
       "  'Year': '2017 Dec 19'},\n",
       " {'PMID': '29297465',\n",
       "  'Title': 'Telomere repeats induce domains of H3K27 methylation in Neurospora.',\n",
       "  'Abstract': 'Development in higher organisms requires selective gene silencing, directed in part by di-/trimethylation of lysine 27 on histone H3 (H3K27me2/3). Knowledge of the cues that control formation of such repressive Polycomb domains is extremely limited. We exploited natural and engineered chromosomal rearrangements in the fungus Neurospora crassa to elucidate the control of H3K27me2/3. Analyses of H3K27me2/3 in strains bearing chromosomal rearrangements revealed both position-dependent and position-independent facultative heterochromatin. We found that proximity to chromosome ends is necessary to maintain, and sufficient to induce, transcriptionally repressive, subtelomeric H3K27me2/3. We ascertained that such telomere-proximal facultative heterochromatin requires native telomere repeats and found that a short array of ectopic telomere repeats, (TTAGGG)(17,) can induce a large domain (~225 kb) of H3K27me2/3. This provides an example of a cis-acting sequence that directs H3K27 methylation. Our findings provide new insight into the relationship between genome organization and control of heterochromatin formation.',\n",
       "  'Keywords': ['H3K27',\n",
       "   'N. crassa',\n",
       "   'chromosome rearrangements',\n",
       "   'chromosomes',\n",
       "   'genes',\n",
       "   'polycomb repressive complex 2',\n",
       "   'position effect',\n",
       "   'telomere repeats'],\n",
       "  'Journal': 'eLife',\n",
       "  'Country': 'United States',\n",
       "  'Year': '2018 Jan 3'},\n",
       " {'PMID': '28973881',\n",
       "  'Title': 'Network of nutrient-sensing pathways and a conserved kinase cascade integrate osmolarity and carbon sensing in Neurospora crassa.',\n",
       "  'Abstract': 'Identifying nutrients available in the environment and utilizing them in the most efficient manner is a challenge common to all organisms. The model filamentous fungus Neurospora crassa is capable of utilizing a variety of carbohydrates, from simple sugars to the complex carbohydrates found in plant cell walls. The zinc binuclear cluster transcription factor CLR-1 is necessary for utilization of cellulose, a major, recalcitrant component of the plant cell wall; however, expression of clr-1 in the absence of an inducer is not sufficient to induce cellulase gene expression. We performed a screen for unidentified actors in the cellulose-response pathway and identified a gene encoding a hypothetical protein (clr-3) that is required for repression of CLR-1 activity in the absence of an inducer. Using clr-3 mutants, we implicated the hyperosmotic-response pathway in the tunable regulation of glycosyl hydrolase production in response to changes in osmolarity. The role of the hyperosmotic-response pathway in nutrient sensing may indicate that cells use osmolarity as a proxy for the presence of free sugar in their environment. These signaling pathways form a nutrient-sensing network that allows Ncrassa cells to tightly regulate gene expression in response to environmental conditions.',\n",
       "  'Keywords': ['MAP kinase cascade',\n",
       "   'carbon catabolite repression',\n",
       "   'carbon metabolism',\n",
       "   'hyperosmotic response',\n",
       "   'plant biomass degradation'],\n",
       "  'Journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'Country': 'CA 94720',\n",
       "  'Year': '2017 Oct 10'},\n",
       " {'PMID': '28947916',\n",
       "  'Title': 'Network reconstruction and systems analysis of plant cell wall deconstruction by Neurospora crassa.',\n",
       "  'Abstract': 'BACKGROUND: Plant biomass degradation by fungal-derived enzymes is rapidly expanding in economic importance as a clean and efficient source for biofuels. The ability to rationally engineer filamentous fungi would facilitate biotechnological applications for degradation of plant cell wall polysaccharides. However, incomplete knowledge of biomolecular networks responsible for plant cell wall deconstruction impedes experimental efforts in this direction. RESULTS: To expand this knowledge base, a detailed network of reactions important for deconstruction of plant cell wall polysaccharides into simple sugars was constructed for the filamentous fungus Neurospora crassa. To reconstruct this network, information was integrated from five heterogeneous data types: functional genomics, transcriptomics, proteomics, genetics, and biochemical characterizations. The combined information was encapsulated into a feature matrix and the evidence weighted to assign annotation confidence scores for each gene within the network. Comparative analyses of RNA-seq and ChIP-seq data shed light on the regulation of the plant cell wall degradation network, leading to a novel hypothesis for degradation of the hemicellulose mannan. The transcription factor CLR-2 was subsequently experimentally shown to play a key role in the mannan degradation pathway of N. crassa. CONCLUSIONS: Here we built a network that serves as a scaffold for integration of diverse experimental datasets. This approach led to the elucidation of regulatory design principles for plant cell wall deconstruction by filamentous fungi and a novel function for the transcription factor CLR-2. This expanding network will aid in efforts to rationally engineer industrially relevant hyper-production strains.',\n",
       "  'Keywords': ['Biofuels',\n",
       "   'CLR-2',\n",
       "   'Mannan',\n",
       "   'Network reconstruction',\n",
       "   'Neurospora crassa',\n",
       "   'Plant cell wall degradation network',\n",
       "   'Systems biology',\n",
       "   'Transcriptional regulatory networks'],\n",
       "  'Journal': 'Biotechnology for biofuels',\n",
       "  'Country': 'WA 98109 USA. ISNI: 0000 0004 0463 2320. GRID: grid.64212.33',\n",
       "  'Year': '2017'},\n",
       " {'PMID': '29078403',\n",
       "  'Title': 'Induction of H3K9me3 and DNA methylation by tethered heterochromatin factors in Neurospora crassa.',\n",
       "  'Abstract': 'Functionally different chromatin domains display distinct chemical marks. Constitutive heterochromatin is commonly associated with trimethylation of lysine 9 on histone H3 (H3K9me3), hypoacetylated histones, and DNA methylation, but the contributions of and interplay among these features are not fully understood. To dissect the establishment of heterochromatin, we investigated the relationships among these features using an in vivo tethering system in Neurospora crassa Artificial recruitment of the H3K9 methyltransferase DIM-5 (defective in methylation-5) induced H3K9me3 and DNA methylation at a normally active, euchromatic locus but did not bypass the requirement of DIM-7, previously implicated in the localization of DIM-5, indicating additional DIM-7 functionality. Tethered heterochromatin protein 1 (HP1) induced H3K9me3, DNA methylation, and gene silencing. The induced heterochromatin required histone deacetylase 1 (HDA-1), with an intact catalytic domain, but HDA-1 was not essential for de novo heterochromatin formation at native heterochromatic regions. Silencing did not require H3K9me3 or DNA methylation. However, DNA methylation contributed to establishment of H3K9me3 induced by tethered HP1. Our analyses also revealed evidence of regulatory mechanisms, dependent on HDA-1 and DIM-5, to control the localization and catalytic activity of the DNA methyltransferase DIM-2. Our study clarifies the interrelationships among canonical aspects of heterochromatin and supports a central role of HDA-1-mediated histone deacetylation in heterochromatin spreading and gene silencing.',\n",
       "  'Keywords': ['DNA methylation',\n",
       "   'H3K9 methylation',\n",
       "   'HP1',\n",
       "   'heterochromatin',\n",
       "   'histone deacetylation'],\n",
       "  'Journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'Country': 'OR 97403',\n",
       "  'Year': '2017 Nov 7'},\n",
       " {'PMID': '30914504',\n",
       "  'Title': 'Metabolism and Development during Conidial Germination in Response to a Carbon-Nitrogen-Rich Synthetic or a Natural Source of Nutrition in Neurospora crassa.',\n",
       "  'Abstract': 'Fungal spores germinate and undergo vegetative growth, leading to either asexual or sexual reproductive dispersal. Previous research has indicated that among developmental regulatory genes, expression is conserved across nutritional environments, whereas pathways for carbon and nitrogen metabolism appear highly responsive-perhaps to accommodate differential nutritive processing. To comprehensively investigate conidial germination and the adaptive life history decision-making underlying these two modes of reproduction, we profiled transcription of Neurospora crassa germinating on two media: synthetic Bird medium, designed to promote asexual reproduction; and a natural maple sap medium, on which both asexual reproduction and sexual reproduction manifest. A later start to germination but faster development was observed on synthetic medium. Metabolic genes exhibited altered expression in response to nutrients-at least 34% of the genes in the genome were significantly downregulated during the first two stages of conidial germination on synthetic medium. Knockouts of genes exhibiting differential expression across development altered germination and growth rates, as well as in one case causing abnormal germination. A consensus Bayesian network of these genes indicated especially tight integration of environmental sensing, asexual and sexual development, and nitrogen metabolism on a natural medium, suggesting that in natural environments, a more dynamic and tentative balance of asexual and sexual development may be typical of N. crassa colonies.IMPORTANCE One of the most remarkable successes of life is its ability to flourish in response to temporally and spatially varying environments. Fungi occupy diverse ecosystems, and their sensitivity to these environmental changes often drives major fungal life history decisions, including the major switch from vegetative growth to asexual or sexual reproduction. Spore germination comprises the first and simplest stage of vegetative growth. We examined the dependence of this early life history on the nutritional environment using genome-wide transcriptomics. We demonstrated that for developmental regulatory genes, expression was generally conserved across nutritional environments, whereas metabolic gene expression was highly labile. The level of activation of developmental genes did depend on current nutrient conditions, as did the modularity of metabolic and developmental response network interactions. This knowledge is critical to the development of future technologies that could manipulate fungal growth for medical, agricultural, or industrial purposes.',\n",
       "  'Keywords': ['artificial medium',\n",
       "   'asexual development',\n",
       "   'asexual-sexual switch',\n",
       "   'conidiospore',\n",
       "   'filamentous fungi',\n",
       "   'germination',\n",
       "   'natural medium'],\n",
       "  'Journal': 'mBio',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2019 Mar 26'},\n",
       " {'PMID': '28600326',\n",
       "  'Title': 'Up-Frameshift Protein UPF1 Regulates Neurospora crassa Circadian and Diurnal Growth Rhythms.',\n",
       "  'Abstract': 'Nonsense-mediated RNA decay (NMD) is a crucial post-transcriptional regulatory mechanism that recognizes and eliminates aberrantly processed transcripts, and mediates the expression of normal gene transcripts. In this study, we report that in the filamentous fungus Neurospora crassa, the NMD factors play a conserved role in regulating the surveillance of NMD targets including premature termination codon (PTC)-containing transcripts and normal transcripts. The circadian rhythms in all of the knockout strains of upf1-3 genes, which encode the Up-frameshift proteins, were aberrant. The upf1 knockout strain displays a shortened circadian period, which can be restored by constantly expressing exogenous Up-frameshift protein 1 (UPF1). UPF1 regulates the circadian clock by modulating the splicing of the core clock gene frequency (frq) through spliceosome and spliceosome-related arginine/serine-rich splicing factors, which partly account for the short periods in the upf1 knockout strain. We also demonstrated that the clock genes including White Collar (WC)-1, WC-2, and FRQ are involved in controlling the diurnal growth rhythm, and UPF1 may affect the growth rhythms by mediating the FRQ protein levels in the daytime. These findings suggest that the NMD factors play important roles in regulating the circadian clock and diurnal growth rhythms in Neurospora.',\n",
       "  'Keywords': ['NMD',\n",
       "   'Neurospora crassa',\n",
       "   'alternative splicing',\n",
       "   'circadian clock',\n",
       "   'diurnal rhythm',\n",
       "   'frq'],\n",
       "  'Journal': 'Genetics',\n",
       "  'Country': 'China',\n",
       "  'Year': '2017 Aug'},\n",
       " {'PMID': '28877740',\n",
       "  'Title': 'Non-base-contacting residues enable kaleidoscopic evolution of metazoan C2H2 zinc finger DNA binding.',\n",
       "  'Abstract': 'BACKGROUND: The C2H2 zinc finger (C2H2-ZF) is the most numerous protein domain in many metazoans, but is not as frequent or diverse in other eukaryotes. The biochemical and evolutionary mechanisms that underlie the diversity of this DNA-binding domain exclusively in metazoans are, however, mostly unknown. RESULTS: Here, we show that the C2H2-ZF expansion in metazoans is facilitated by contribution of non-base-contacting residues to DNA binding energy, allowing base-contacting specificity residues to mutate without catastrophic loss of DNA binding. In contrast, C2H2-ZF DNA binding in fungi, plants, and other lineages is constrained by reliance on base-contacting residues for DNA-binding functionality. Reconstructions indicate that virtually every DNA triplet was recognized by at least one C2H2-ZF domain in the common progenitor of placental mammals, but that extant C2H2-ZF domains typically bind different sequences from these ancestral domains, with changes facilitated by non-base-contacting residues. CONCLUSIONS: Our results suggest that the evolution of C2H2-ZFs in metazoans was expedited by the interaction of non-base-contacting residues with the DNA backbone. We term this phenomenon \"kaleidoscopic evolution,\" to reflect the diversity of both binding motifs and binding motif transitions and the facilitation of their diversification.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Genome biology',\n",
       "  'Country': 'Canada. hamed.najafabadi@mcgill.ca',\n",
       "  'Year': '2017 Sep 6'},\n",
       " {'PMID': '28322269',\n",
       "  'Title': 'A Ras GTPase associated protein is involved in the phototropic and circadian photobiology responses in fungi.',\n",
       "  'Abstract': 'Light is an environmental signal perceived by most eukaryotic organisms and that can have major impacts on their growth and development. The MadC protein in the fungus Phycomyces blakesleeanus (Mucoromycotina) has been postulated to form part of the photosensory input for phototropism of the fruiting body sporangiophores, but the madC gene has remained unidentified since the 1960s when madC mutants were first isolated. In this study the madC gene was identified by positional cloning. All madC mutant strains contain loss-of-function point mutations within a gene predicted to encode a GTPase activating protein (GAP) for Ras. The madC gene complements the Saccharomyces cerevisiae Ras-GAP ira1 mutant and the encoded MadC protein interacts with P. blakesleeanus Ras homologs in yeast two-hybrid assays, indicating that MadC is a regulator of Ras signaling. Deletion of the homolog in the filamentous ascomycete Neurospora crassa affects the circadian clock output, yielding a pattern of asexual conidiation similar to a ras-1 mutant that is used in circadian studies in N. crassa. Thus, MadC is unlikely to be a photosensor, yet is a fundamental link in the photoresponses from blue light perceived by the conserved White Collar complex with Ras signaling in two distantly-related filamentous fungal species.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Scientific reports',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2017 Mar 21'},\n",
       " {'PMID': '28616073',\n",
       "  'Title': 'The transcription factor PDR-1 is a multi-functional regulator and key component of pectin deconstruction and catabolism in Neurospora crassa.',\n",
       "  'Abstract': 'BACKGROUND: Pectin is an abundant component in many fruit and vegetable wastes and could therefore be an excellent resource for biorefinery, but is currently underutilized. Fungal pectinases already play a crucial role for industrial purposes, such as for foodstuff processing. However, the regulation of pectinase gene expression is still poorly understood. For an optimal utilization of plant biomass for biorefinery and biofuel production, a detailed analysis of the underlying regulatory mechanisms is warranted. In this study, we applied the genetic resources of the filamentous ascomycete species Neurospora crassa to screen for transcription factors that play a major role in pectinase induction. RESULTS: The pectin degradation regulator-1 (PDR-1) was identified through a transcription factor mutant screen in N. crassa. The Deltapdr-1 mutant exhibited a severe growth defect on pectin and all tested pectin-related poly- and monosaccharides. Biochemical as well as transcriptional analyses of WT and the Deltapdr-1 mutant revealed that while PDR-1-mediated gene induction was dependent on the presence of l-rhamnose, it also strongly affected the degradation of the homogalacturonan backbone. The expression of the endo-polygalacturonase gh28-1 was greatly reduced in the Deltapdr-1 mutant, while the expression levels of all pectate lyase genes increased. Moreover, a pdr-1 overexpression strain displayed substantially increased pectinase production. Promoter analysis of the PDR-1 regulon allowed refinement of the putative PDR-1 DNA-binding motif. CONCLUSIONS: PDR-1 is highly conserved in filamentous ascomycete fungi and is present in many pathogenic and industrially important fungi. Our data demonstrate that the function of PDR-1 in N. crassa combines features of two recently described transcription factors in Aspergillus niger (RhaR) and Botrytis cinerea (GaaR). The results presented in this study contribute to a broader understanding of how pectin degradation is orchestrated in filamentous fungi and how it could be manipulated for optimized pectinase production.',\n",
       "  'Keywords': ['Biorefinery',\n",
       "   'Gene regulation',\n",
       "   'Neurospora crassa',\n",
       "   'Pectin degradation',\n",
       "   'Zn2Cys6 transcription factor',\n",
       "   'd-galacturonic acid catabolism',\n",
       "   'l-rhamnose catabolism'],\n",
       "  'Journal': 'Biotechnology for biofuels',\n",
       "  'Country': 'Germany. ISNI: 0000000123222966. GRID: grid.6936.a',\n",
       "  'Year': '2017'},\n",
       " {'PMID': '28471414',\n",
       "  'Title': 'Melanisation of Aspergillus terreus-Is Butyrolactone I Involved in the Regulation of Both DOPA and DHN Types of Pigments in Submerged Culture?',\n",
       "  'Abstract': 'Pigments and melanins of fungal spores have been investigated for decades, revealing important roles in the survival of the fungus in hostile environments. The key genes and the encoded enzymes for pigment and melanin biosynthesis have recently been found in Ascomycota, including Aspergillus spp. In Aspergillus terreus, the pigmentation has remained mysterious with only one class of melanin biogenesis being found. In this study, we examined an intriguing, partially annotated gene cluster of A. terreus strain NIH2624, utilizing previously sequenced transcriptome and improved gene expression data of strain MUCL 38669, under the influence of a suggested quorum sensing inducing metabolite, butyrolactone I. The core polyketide synthase (PKS) gene of the cluster was predicted to be significantly longer on the basis of the obtained transcriptional data, and the surrounding cluster was positively regulated by butyrolactone I at the late growth phase of submerged culture, presumably during sporulation. Phylogenetic analysis of the extended PKS revealed remarkable similarity with a group of known pigments of Fusarium spp., indicating a similar function for this PKS. We present a hypothesis of this PKS cluster to biosynthesise a 1,8-dihydroxynaphthalene (DHN)-type of pigment during sporulation with the influence of butyrolactone I under submerged culture.',\n",
       "  'Keywords': ['Aspergillus terreus',\n",
       "   'NR-PKS cluster',\n",
       "   'butyrolactone I',\n",
       "   'filamentous fungi',\n",
       "   'gene expression',\n",
       "   'melanin',\n",
       "   'pigment',\n",
       "   'transcriptome sequencing'],\n",
       "  'Journal': 'Microorganisms',\n",
       "  'Country': 'Finland. elina.palonen@abo.fi',\n",
       "  'Year': '2017 May 4'},\n",
       " {'PMID': '28655822',\n",
       "  'Title': 'Translation Initiation from Conserved Non-AUG Codons Provides Additional Layers of Regulation and Coding Capacity.',\n",
       "  'Abstract': \"Neurospora crassa cpc-1 and Saccharomyces cerevisiae GCN4 are homologs specifying transcription activators that drive the transcriptional response to amino acid limitation. The cpc-1 mRNA contains two upstream open reading frames (uORFs) in its >700-nucleotide (nt) 5' leader, and its expression is controlled at the level of translation in response to amino acid starvation. We used N. crassa cell extracts and obtained data indicating that cpc-1 uORF1 and uORF2 are functionally analogous to GCN4 uORF1 and uORF4, respectively, in controlling translation. We also found that the 5' region upstream of the main coding sequence of the cpc-1 mRNA extends for more than 700 nucleotides without any in-frame stop codon. For 100 cpc-1 homologs from Pezizomycotina and from selected Basidiomycota, 5' conserved extensions of the CPC1 reading frame are also observed. Multiple non-AUG near-cognate codons (NCCs) in the CPC1 reading frame upstream of uORF2, some deeply conserved, could potentially initiate translation. At least four NCCs initiated translation in vitroIn vivo data were consistent with initiation at NCCs to produce N-terminally extended N. crassa CPC1 isoforms. The pivotal role played by CPC1, combined with its translational regulation by uORFs and NCC utilization, underscores the emerging significance of noncanonical initiation events in controlling gene expression.IMPORTANCE There is a deepening and widening appreciation of the diverse roles of translation in controlling gene expression. A central fungal transcription factor, the best-studied example of which is Saccharomyces cerevisiae GCN4, is crucial for the response to amino acid limitation. Two upstream open reading frames (uORFs) in the GCN4 mRNA are critical for controlling GCN4 synthesis. We observed that two uORFs in the corresponding Neurospora crassa cpc-1 mRNA appear functionally analogous to the GCN4 uORFs. We also discovered that, surprisingly, unlike GCN4, the CPC1 coding sequence extends far upstream from the presumed AUG start codon with no other in-frame AUG codons. Similar extensions were seen in homologs from many filamentous fungi. We observed that multiple non-AUG near-cognate codons (NCCs) in this extended reading frame, some conserved, initiated translation to produce longer forms of CPC1, underscoring the significance of noncanonical initiation in controlling gene expression.\",\n",
       "  'Keywords': ['Neurospora',\n",
       "   'filamentous fungi',\n",
       "   'gene regulation',\n",
       "   'molecular genetics',\n",
       "   'translational control'],\n",
       "  'Journal': 'mBio',\n",
       "  'Country': 'Ireland',\n",
       "  'Year': '2017 Jun 27'},\n",
       " {'PMID': '28420736',\n",
       "  'Title': 'Deciphering the Regulatory Network between the SREBP Pathway and Protein Secretion in Neurospora crassa.',\n",
       "  'Abstract': 'Sterol regulatory element binding proteins (SREBPs) are conserved from yeast to mammalian cells and function in the regulation of sterol homeostasis. In fungi, the SREBP pathway has been implicated in the adaptation to hypoxia and in virulence. In Neurospora crassa and Trichoderma reesei, the SREBP pathway also negatively regulates protein secretion under lignocellulolytic conditions. Here we utilized global transcriptional profiling combined with genetic and physiological analyses to address the regulatory link between the SREBP pathway and protein secretion in N. crassa Our results demonstrated that the function of the SREBP pathway in ergosterol biosynthesis and adaptation to hypoxia was conserved in N. crassa Under lignocellulolytic conditions, the SREBP pathway was highly activated, resulting in the reduced expression of lytic polysaccharide monooxygenases, which require molecular oxygen for catalytic activity. Additionally, activation of the SREBP pathway under lignocellulolytic conditions repressed a set of genes predicted to be involved in the endoplasmic reticulum stress response. Here we show that the inability of a hac-1 mutant, which bears a deletion of the major regulator of the unfolded protein response (UPR), to efficiently produce cellulases and utilize cellulose was suppressed by mutations in the SREBP pathway. The analyses presented here demonstrated new SREBP pathway functions, including linkages to the UPR, and provide new clues for genetic engineering of filamentous fungi to improve their production of extracellular proteins.IMPORTANCE The role of SREBP transcription factors in the regulation of sterol biosynthesis is conserved from humans to yeast. In filamentous fungi, this pathway regulates the secretion of lignocellulolytic enzymes during plant biomass deconstruction. Here we show that the SREBP pathway in Neurospora crassa regulates the production of specific cellulases, lytic polysaccharide monooxygenases that utilize molecular oxygen. Via global transcriptional profile and genetic analyses, a relationship between the SREBP pathway and the unfolded protein response (UPR) pathway was revealed, suggesting a regulatory interplay of these two pathways in the trafficking of plant biomass-degrading enzymes. These findings have implications for our understanding of the cross talk of the SREBP and UPR pathways in other organisms and will guide the rational engineering of fungal strains to improve cellulolytic enzyme production.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'mBio',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2017 Apr 18'},\n",
       " {'PMID': '28467421',\n",
       "  'Title': 'A fungal transcription factor essential for starch degradation affects integration of carbon and nitrogen metabolism.',\n",
       "  'Abstract': 'In Neurospora crassa, the transcription factor COL-26 functions as a regulator of glucose signaling and metabolism. Its loss leads to resistance to carbon catabolite repression. Here, we report that COL-26 is necessary for the expression of amylolytic genes in N. crassa and is required for the utilization of maltose and starch. Additionally, the Deltacol-26 mutant shows growth defects on preferred carbon sources, such as glucose, an effect that was alleviated if glutamine replaced ammonium as the primary nitrogen source. This rescue did not occur when maltose was used as a sole carbon source. Transcriptome and metabolic analyses of the Deltacol-26 mutant relative to its wild type parental strain revealed that amino acid and nitrogen metabolism, the TCA cycle and GABA shunt were adversely affected. Phylogenetic analysis showed a single col-26 homolog in Sordariales, Ophilostomatales, and the Magnaporthales, but an expanded number of col-26 homologs in other filamentous fungal species. Deletion of the closest homolog of col-26 in Trichoderma reesei, bglR, resulted in a mutant with similar preferred carbon source growth deficiency, and which was alleviated if glutamine was the sole nitrogen source, suggesting conservation of COL-26 and BglR function. Our finding provides novel insight into the role of COL-26 for utilization of starch and in integrating carbon and nitrogen metabolism for balanced metabolic activities for optimal carbon and nitrogen distribution.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS genetics',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2017 May'},\n",
       " {'PMID': '28467421',\n",
       "  'Title': 'A fungal transcription factor essential for starch degradation affects integration of carbon and nitrogen metabolism.',\n",
       "  'Abstract': 'In Neurospora crassa, the transcription factor COL-26 functions as a regulator of glucose signaling and metabolism. Its loss leads to resistance to carbon catabolite repression. Here, we report that COL-26 is necessary for the expression of amylolytic genes in N. crassa and is required for the utilization of maltose and starch. Additionally, the Deltacol-26 mutant shows growth defects on preferred carbon sources, such as glucose, an effect that was alleviated if glutamine replaced ammonium as the primary nitrogen source. This rescue did not occur when maltose was used as a sole carbon source. Transcriptome and metabolic analyses of the Deltacol-26 mutant relative to its wild type parental strain revealed that amino acid and nitrogen metabolism, the TCA cycle and GABA shunt were adversely affected. Phylogenetic analysis showed a single col-26 homolog in Sordariales, Ophilostomatales, and the Magnaporthales, but an expanded number of col-26 homologs in other filamentous fungal species. Deletion of the closest homolog of col-26 in Trichoderma reesei, bglR, resulted in a mutant with similar preferred carbon source growth deficiency, and which was alleviated if glutamine was the sole nitrogen source, suggesting conservation of COL-26 and BglR function. Our finding provides novel insight into the role of COL-26 for utilization of starch and in integrating carbon and nitrogen metabolism for balanced metabolic activities for optimal carbon and nitrogen distribution.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS genetics',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2017 May'},\n",
       " {'PMID': '28115989',\n",
       "  'Title': 'Identification and characterization of the glucose dual-affinity transport system in Neurospora crassa: pleiotropic roles in nutrient transport, signaling, and carbon catabolite repression.',\n",
       "  'Abstract': 'BACKGROUND: The glucose dual-affinity transport system (low- and high-affinity) is a conserved strategy used by microorganisms to cope with natural fluctuations in nutrient availability in the environment. The glucose-sensing and uptake processes are believed to be tightly associated with cellulase expression regulation in cellulolytic fungi. However, both the identities and functions of the major molecular components of this evolutionarily conserved system in filamentous fungi remain elusive. Here, we systematically identified and characterized the components of the glucose dual-affinity transport system in the model fungus Neurospora crassa. RESULTS: Using RNA sequencing coupled with functional transport analyses, we assigned GLT-1 (K(m) = 18.42 +/- 3.38 mM) and HGT-1/-2 (K(m) = 16.13 +/- 0.95 and 98.97 +/- 22.02 microM) to the low- and high-affinity glucose transport systems, respectively. The high-affinity transporters hgt-1/-2 complemented a moderate growth defect under high glucose when glt-1 was deleted. Simultaneous deletion of hgt-1/-2 led to extensive derepression of genes for plant cell wall deconstruction in cells grown on cellulose. The suppression by HGT-1/-2 was connected to both carbon catabolite repression (CCR) and the cyclic adenosine monophosphate-protein kinase A pathway. Alteration of a residue conserved across taxa in hexose transporters resulted in a loss of glucose-transporting function, whereas CCR signal transduction was retained, indicating dual functions for HGT-1/-2 as \"transceptors.\" CONCLUSIONS: In this study, GLT-1 and HGT-1/-2 were identified as the key components of the glucose dual-affinity transport system, which plays diverse roles in glucose transport and carbon metabolism. Given the wide conservation of the glucose dual-affinity transport system across fungal species, the identification of its components and their pleiotropic roles in this study shed important new light on the molecular basis of nutrient transport, signaling, and plant cell wall degradation in fungi.',\n",
       "  'Keywords': ['Cellulase',\n",
       "   'Dual-affinity',\n",
       "   'GLT-1',\n",
       "   'Glucose transporter',\n",
       "   'HGT-1/-2',\n",
       "   'Signaling'],\n",
       "  'Journal': 'Biotechnology for biofuels',\n",
       "  'Country': '300308 China. ISNI: 0000000119573309. GRID: grid.9227.e',\n",
       "  'Year': '2017'},\n",
       " {'PMID': '27681634',\n",
       "  'Title': 'Dual chromatin recognition by the histone deacetylase complex HCHC is required for proper DNA methylation in Neurospora crassa.',\n",
       "  'Abstract': 'DNA methylation, heterochromatin protein 1 (HP1), histone H3 lysine 9 (H3K9) methylation, histone deacetylation, and highly repeated sequences are prototypical heterochromatic features, but their interrelationships are not fully understood. Prior work showed that H3K9 methylation directs DNA methylation and histone deacetylation via HP1 in Neurospora crassa and that the histone deacetylase complex HCHC is required for proper DNA methylation. The complex consists of the chromodomain proteins HP1 and chromodomain protein 2 (CDP-2), the histone deacetylase HDA-1, and the AT-hook motif protein CDP-2/HDA-1-associated protein (CHAP). We show that the complex is required for proper chromosome segregation, dissect its function, and characterize interactions among its components. Our analyses revealed the existence of an HP1-based DNA methylation pathway independent of its chromodomain. The pathway partially depends on CHAP but not on the CDP-2 chromodomain. CDP-2 serves as a bridge between the recognition of H3K9 trimethylation (H3K9me3) by HP1 and the histone deacetylase activity of HDA-1. CHAP is also critical for HDA-1 localization to heterochromatin. Specifically, the CHAP zinc finger interacts directly with the HDA-1 argonaute-binding protein 2 (Arb2) domain, and the CHAP AT-hook motifs recognize heterochromatic regions by binding to AT-rich DNA. Our data shed light on the interrelationships among the prototypical heterochromatic features and support a model in which dual recognition by the HP1 chromodomain and the CHAP AT-hooks are required for proper heterochromatin formation.',\n",
       "  'Keywords': ['DNA methylation',\n",
       "   'H3K9 methylation',\n",
       "   'HP1',\n",
       "   'heterochromatin',\n",
       "   'histone deacetylation'],\n",
       "  'Journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'Country': 'Japan',\n",
       "  'Year': '2016 Oct 11'},\n",
       " {'PMID': '26694141',\n",
       "  'Title': 'Neurospora crassa transcriptomics reveals oxidative stress and plasma membrane homeostasis biology genes as key targets in response to chitosan.',\n",
       "  'Abstract': 'Chitosan is a natural polymer with antimicrobial activity. Chitosan causes plasma membrane permeabilization and induction of intracellular reactive oxygen species (ROS) in Neurospora crassa. We have determined the transcriptional profile of N. crassa to chitosan and identified the main gene targets involved in the cellular response to this compound. Global network analyses showed membrane, transport and oxidoreductase activity as key nodes affected by chitosan. Activation of oxidative metabolism indicates the importance of ROS and cell energy together with plasma membrane homeostasis in N. crassa response to chitosan. Deletion strain analysis of chitosan susceptibility pointed NCU03639 encoding a class 3 lipase, involved in plasma membrane repair by lipid replacement, and NCU04537 a MFS monosaccharide transporter related to assimilation of simple sugars, as main gene targets of chitosan. NCU10521, a glutathione S-transferase-4 involved in the generation of reducing power for scavenging intracellular ROS is also a determinant chitosan gene target. Ca(2+) increased tolerance to chitosan in N. crassa. Growth of NCU10610 (fig 1 domain) and SYT1 (a synaptotagmin) deletion strains was significantly increased by Ca(2+) in the presence of chitosan. Both genes play a determinant role in N. crassa membrane homeostasis. Our results are of paramount importance for developing chitosan as an antifungal.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Molecular bioSystems',\n",
       "  'Country': 'Spain. federico.lopez@ua.es lv.lopez@ua.es',\n",
       "  'Year': '2016 Feb'},\n",
       " {'PMID': '27870869',\n",
       "  'Title': 'Glucose Starvation Alters Heat Shock Response, Leading to Death of Wild Type Cells and Survival of MAP Kinase Signaling Mutant.',\n",
       "  'Abstract': 'A moderate heat shock induces Neurospora crassa to synthesize large quantities of heat shock proteins that are protective against higher, otherwise lethal temperatures. However, wild type cells do not survive when carbohydrate deprivation is added to heat shock. In contrast, a mutant strain defective in a stress-activated protein kinase does survive the combined stresses. In order to understand the basis for this difference in survival, we have determined the relative levels of detected proteins in the mutant and wild type strain during dual stress, and we have identified gene transcripts in both strains whose quantities change in response to heat shock or dual stress. These data and supportive experimental evidence point to reasons for survival of the mutant strain. By using alternative respiratory mechanisms, these cells experience less of the oxidative stress that proves damaging to wild type cells. Of central importance, mutant cells recycle limited resources during dual stress by undergoing autophagy, a process that we find utilized by both wild type and mutant cells during heat shock. Evidence points to inappropriate activation of TORC1, the central metabolic regulator, in wild type cells during dual stress, based upon behavior of an additional signaling mutant and inhibitor studies.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PloS one',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2016'},\n",
       " {'PMID': '27856763',\n",
       "  'Title': 'Normal chromosome conformation depends on subtelomeric facultative heterochromatin in Neurospora crassa.',\n",
       "  'Abstract': 'High-throughput chromosome conformation capture (Hi-C) analyses revealed that the 3D structure of the Neurospora crassa genome is dominated by intra- and interchromosomal links between regions of heterochromatin, especially constitutive heterochromatin. Elimination of trimethylation of lysine 9 on histone H3 (H3K9me3) or its binding partner Heterochromatin Protein 1 (HP1)-both prominent features of constitutive heterochromatin-have little effect on the Hi-C pattern. It remained possible that di- or trimethylation of lysine 27 on histone H3 (H3K27me2/3), which becomes localized in regions of constitutive heterochromatin when H3K9me3 or HP1 are lost, plays a critical role in the 3D structure of the genome. We found that H3K27me2/3, catalyzed by the Polycomb Repressive Complex 2 (PRC2) member SET-7 (SET domain protein-7), does indeed play a prominent role in the Hi-C pattern of WT, but that its presence in regions normally occupied by H3K9me3 is not responsible for maintenance of the genome architecture when H3K9me3 is lost. The Hi-C pattern of a mutant defective in the PRC2 member N. crassa p55 (NPF), which is predominantly required for subtelomeric H3K27me2/3, was equivalent to that of the set-7 deletion strain, suggesting that subtelomeric facultative heterochromatin is paramount for normal chromosome conformation. Both PRC2 mutants showed decreased heterochromatin-heterochromatin contacts and increased euchromatin-heterochromatin contacts. Cytological observations suggested elimination of H3K27me2/3 leads to partial displacement of telomere clusters from the nuclear periphery. Transcriptional profiling of Deltadim-5, Deltaset-7, Deltaset-7; Deltadim-5, and Deltanpf strains detailed anticipated changes in gene expression but did not support the idea that global changes in genome architecture, per se, led to altered transcription.',\n",
       "  'Keywords': ['H3K27me2/3',\n",
       "   'Hi-C',\n",
       "   'Neurospora crassa',\n",
       "   'PRC2',\n",
       "   'facultative heterochromatin'],\n",
       "  'Journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'Country': 'OR 97403',\n",
       "  'Year': '2016 Dec 27'},\n",
       " {'PMID': '28039555',\n",
       "  'Title': 'Disruption of non-anchored cell wall protein NCW-1 promotes cellulase production by increasing cellobiose uptake in Neurospora crassa.',\n",
       "  'Abstract': 'OBJECTIVES: To elucidate the mechanism of cellulase signal transduction in filamentous fungi including the components of the cellulase induction pathway. RESULTS: Neurospora crassa ncw-1 encodes a non-anchored cell wall protein. The absence of ncw-1 increased cellulase gene expression and this is not due to relieving carbon catabolite repression mediated by the cre-1 pathway. A mutant lacking genes encoding both three major beta-glucosidase enzymes and NCW-1 (Delta3betaGDeltancw-1) was constructed. Transcriptome analysis of the quadruple mutant demonstrated enhanced expression of cellodextrin transporters after ncw-1 deletion, indicating that ncw-1 affects cellulase expression and production by inhibiting the uptake of the cellodextrin. CONCLUSIONS: NCW-1 is a novel component that plays a critical role in the cellulase induction signaling pathway.',\n",
       "  'Keywords': ['Cell wall protein',\n",
       "   'Cellobiose uptake',\n",
       "   'Cellodextrin transporter',\n",
       "   'Cellulase induction',\n",
       "   'Neurospora crassa'],\n",
       "  'Journal': 'Biotechnology letters',\n",
       "  'Country': 'China',\n",
       "  'Year': '2017 Apr'},\n",
       " {'PMID': '27260477',\n",
       "  'Title': 'Neurospora chromosomes are organized by blocks of importin alpha-dependent heterochromatin that are largely independent of H3K9me3.',\n",
       "  'Abstract': 'Eukaryotic genomes are organized into chromatin domains with three-dimensional arrangements that presumably result from interactions between the chromatin constituents-proteins, DNA, and RNA-within the physical constraints of the nucleus. We used chromosome conformation capture (3C) followed by high-throughput sequencing (Hi-C) with wild-type and mutant strains of Neurospora crassa to gain insight into the role of heterochromatin in the organization and function of the genome. We tested the role of three proteins thought to be important for establishment of heterochromatin, namely, the histone H3 lysine 9 methyltransferase DIM-5, Heterochromatin Protein 1 (HP1), which specifically binds to the product of DIM-5 (trimethylated H3 lysine 9 [H3K9me3]), and DIM-3 (importin alpha), which is involved in DIM-5 localization. The average genome configuration of the wild-type strain revealed strong intra- and inter-chromosomal associations between both constitutive and facultative heterochromatic domains, with the strongest interactions among the centromeres, subtelomeres, and interspersed heterochromatin. Surprisingly, loss of either H3K9me3 or HP1 had only mild effects on heterochromatin compaction, whereas dim-3 caused more drastic changes, specifically decreasing interactions between constitutive heterochromatic domains. Thus, associations between heterochromatic regions are a major component of the chromosome conformation in Neurospora, but two widely studied key heterochromatin proteins are not necessary, implying that undefined protein factors play key roles in maintaining overall chromosome organization.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Genome research',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2016 Aug'},\n",
       " {'PMID': '27172195',\n",
       "  'Title': 'Histone H1 Limits DNA Methylation in Neurospora crassa.',\n",
       "  'Abstract': 'Histone H1 variants, known as linker histones, are essential chromatin components in higher eukaryotes, yet compared to the core histones relatively little is known about their in vivo functions. The filamentous fungus Neurospora crassa encodes a single H1 protein that is not essential for viability. To investigate the role of N. crassa H1, we constructed a functional FLAG-tagged H1 fusion protein and performed genomic and molecular analyses. Cell fractionation experiments showed that H1-3XFLAG is a chromatin binding protein. Chromatin-immunoprecipitation combined with sequencing (ChIP-seq) revealed that H1-3XFLAG is globally enriched throughout the genome with a subtle preference for promoters of expressed genes. In mammals, the stoichiometry of H1 impacts nucleosome repeat length. To determine if H1 impacts nucleosome occupancy or nucleosome positioning in N. crassa, we performed micrococcal nuclease digestion in the wild-type and the [Formula: see text]hH1 strain followed by sequencing (MNase-seq). Deletion of hH1 did not significantly impact nucleosome positioning or nucleosome occupancy. Analysis of DNA methylation by whole-genome bisulfite sequencing (MethylC-seq) revealed a modest but global increase in DNA methylation in the [Formula: see text]hH1 mutant. Together, these data suggest that H1 acts as a nonspecific chromatin binding protein that can limit accessibility of the DNA methylation machinery in N. crassa.',\n",
       "  'Keywords': ['DNA methylation',\n",
       "   'chromatin',\n",
       "   'histone H1',\n",
       "   'nucleosome positioning'],\n",
       "  'Journal': 'G3 (Bethesda, Md.)',\n",
       "  'Country': 'Georgia 30602',\n",
       "  'Year': '2016 Jul 7'},\n",
       " {'PMID': '27172195',\n",
       "  'Title': 'Histone H1 Limits DNA Methylation in Neurospora crassa.',\n",
       "  'Abstract': 'Histone H1 variants, known as linker histones, are essential chromatin components in higher eukaryotes, yet compared to the core histones relatively little is known about their in vivo functions. The filamentous fungus Neurospora crassa encodes a single H1 protein that is not essential for viability. To investigate the role of N. crassa H1, we constructed a functional FLAG-tagged H1 fusion protein and performed genomic and molecular analyses. Cell fractionation experiments showed that H1-3XFLAG is a chromatin binding protein. Chromatin-immunoprecipitation combined with sequencing (ChIP-seq) revealed that H1-3XFLAG is globally enriched throughout the genome with a subtle preference for promoters of expressed genes. In mammals, the stoichiometry of H1 impacts nucleosome repeat length. To determine if H1 impacts nucleosome occupancy or nucleosome positioning in N. crassa, we performed micrococcal nuclease digestion in the wild-type and the [Formula: see text]hH1 strain followed by sequencing (MNase-seq). Deletion of hH1 did not significantly impact nucleosome positioning or nucleosome occupancy. Analysis of DNA methylation by whole-genome bisulfite sequencing (MethylC-seq) revealed a modest but global increase in DNA methylation in the [Formula: see text]hH1 mutant. Together, these data suggest that H1 acts as a nonspecific chromatin binding protein that can limit accessibility of the DNA methylation machinery in N. crassa.',\n",
       "  'Keywords': ['DNA methylation',\n",
       "   'chromatin',\n",
       "   'histone H1',\n",
       "   'nucleosome positioning'],\n",
       "  'Journal': 'G3 (Bethesda, Md.)',\n",
       "  'Country': 'Georgia 30602',\n",
       "  'Year': '2016 Jul 7'},\n",
       " {'PMID': '27072538',\n",
       "  'Title': 'Transcriptomic and metabolomic profiling of ionic liquid stimuli unveils enhanced secondary metabolism in Aspergillus nidulans.',\n",
       "  'Abstract': \"BACKGROUND: The inherent potential of filamentous fungi, especially of Ascomycota, for producing diverse bioactive metabolites remains largely silent under standard laboratory culture conditions. Innumerable strategies have been described to trigger their production, one of the simplest being manipulation of the growth media composition. Supplementing media with ionic liquids surprisingly enhanced the diversity of extracellular metabolites generated by penicillia. This finding led us to evaluate the impact of ionic liquids' stimuli on the fungal metabolism in Aspergillus nidulans and how it reflects on the biosynthesis of secondary metabolites (SMs). RESULTS: Whole transcriptional profiling showed that exposure to 0.7 M cholinium chloride or 1-ethyl-3-methylimidazolium chloride dramatically affected expression of genes encoding both primary and secondary metabolism. Both ionic liquids apparently induced stress responses and detoxification mechanisms but response profiles to each stimulus were unique. Primary metabolism was up-regulated by choline, but down-regulated by 1-ethyl-3-methylimidazolium chloride; both stimulated production of acetyl-CoA (key precursor to numerous SMs) and non proteinogenic amino acids (building blocks of bioactive classes of SMs). In total, twenty one of the sixty six described backbone genes underwent up-regulation. Accordingly, differential analysis of the fungal metabolome showed that supplementing growth media with ionic liquids resulted in ca. 40 differentially accumulated ion masses compared to control conditions. In particular, it stimulated production of monodictyphenone and orsellinic acid, otherwise cryptic. Expression levels of genes encoding corresponding polyketide biosynthetic enzymes (i.e. backbone genes) increased compared to control conditions. The corresponding metabolite extracts showed increased cell polarity modulation potential in an ex vivo whole tissue assay (The lial Live Targeted Epithelia; theLiTE). CONCLUSIONS: Ionic liquids, a diverse class of chemicals composed solely of ions, can provide an unexpected means to further resolve the diversity of natural compounds, guiding discovery of fungal metabolites with clinical potential.\",\n",
       "  'Keywords': ['Aspergillus nidulans',\n",
       "   'Ionic liquids',\n",
       "   'Metabolomics',\n",
       "   'Monodictyphenone',\n",
       "   'Orsellinic acid',\n",
       "   'Secondary metabolism',\n",
       "   'Transcriptomics'],\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'Portugal',\n",
       "  'Year': '2016 Apr 12'},\n",
       " {'PMID': '26771905',\n",
       "  'Title': 'The LSH/DDM1 Homolog MUS-30 Is Required for Genome Stability, but Not for DNA Methylation in Neurospora crassa.',\n",
       "  'Abstract': 'LSH/DDM1 enzymes are required for DNA methylation in higher eukaryotes and have poorly defined roles in genome maintenance in yeast, plants, and animals. The filamentous fungus Neurospora crassa is a tractable system that encodes a single LSH/DDM1 homolog (NCU06306). We report that the Neurospora LSH/DDM1 enzyme is encoded by mutagen sensitive-30 (mus-30), a locus identified in a genetic screen over 25 years ago. We show that MUS-30-deficient cells have normal DNA methylation, but are hypersensitive to DNA damaging agents. MUS-30 is a nuclear protein, consistent with its predicted role as a chromatin remodeling enzyme, and levels of MUS-30 are increased following DNA damage. MUS-30 co-purifies with Neurospora WDR76, a homolog of yeast Changed Mutation Rate-1 and mammalian WD40 repeat domain 76. Deletion of wdr76 rescued DNA damage-hypersensitivity of Deltamus-30 strains, demonstrating that the MUS-30-WDR76 interaction is functionally important. DNA damage-sensitivity of Deltamus-30 is partially suppressed by deletion of methyl adenine glycosylase-1, a component of the base excision repair machinery (BER); however, the rate of BER is not affected in Deltamus-30 strains. We found that MUS-30-deficient cells are not defective for DSB repair, and we observed a negative genetic interaction between Deltamus-30 and Deltamei-3, the Neurospora RAD51 homolog required for homologous recombination. Together, our findings suggest that MUS-30, an LSH/DDM1 homolog, is required to prevent DNA damage arising from toxic base excision repair intermediates. Overall, our study provides important new information about the functions of the LSH/DDM1 family of enzymes.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS genetics',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2016 Jan'},\n",
       " {'PMID': '26484172',\n",
       "  'Title': 'Transcription profiling of the Neurospora crassa response to a group of synthetic (thio)xanthones and a natural acetophenone.',\n",
       "  'Abstract': 'Xanthones are a class of heterocyclic compounds characterized by a dibenzo-gamma-pyrone nucleus. Analysis of their mode of action in cells, namely uncovering alterations in gene expression, is important because these compounds have potential therapeutic applications. Thus, we studied the transcriptional response of the filamentous fungus Neurospora crassa to a group of synthetic (thio)xanthone derivatives with antitumor activity using high throughput RNA sequencing. The induction of ABC transporters in N. crassa, particularly atrb and cdr4, is a common consequence of the treatment with xanthones. In addition, we found a group of genes repressed by all of the tested (thio)xanthone derivatives, that are evocative of genes downregulated during oxidative stress. The transcriptional response of N. crassa treated with an acetophenone isolated from the soil fungus Neosartorya siamensis shares some features with the (thio)xanthone-elicited gene expression profiles. Two of the (thio)xanthone derivatives and the N. siamensis-derived acetophenone inhibited the growth of N. crassa. Our work also provides framework datasets that may orientate future studies on the mechanisms of action of some groups of xanthones.',\n",
       "  'Keywords': ['ABC transporters',\n",
       "   'Acetophenone',\n",
       "   'CZT-1',\n",
       "   'Neurospora crassa',\n",
       "   'Transcriptional profiling',\n",
       "   'Xanthones'],\n",
       "  'Journal': 'Genomics data',\n",
       "  'Country': 'Portugal',\n",
       "  'Year': '2015 Jun'},\n",
       " {'PMID': '26537359',\n",
       "  'Title': 'Loss of HP1 causes depletion of H3K27me3 from facultative heterochromatin and gain of H3K27me2 at constitutive heterochromatin.',\n",
       "  'Abstract': 'Methylated lysine 27 on histone H3 (H3K27me) marks repressed \"facultative heterochromatin,\" including developmentally regulated genes in plants and animals. The mechanisms responsible for localization of H3K27me are largely unknown, perhaps in part because of the complexity of epigenetic regulatory networks. We used a relatively simple model organism bearing both facultative and constitutive heterochromatin, Neurospora crassa, to explore possible interactions between elements of heterochromatin. In higher eukaryotes, reductions of H3K9me3 and DNA methylation in constitutive heterochromatin have been variously reported to cause redistribution of H3K27me3. In Neurospora, we found that elimination of any member of the DCDC H3K9 methylation complex caused massive changes in the distribution of H3K27me; regions of facultative heterochromatin lost H3K27me3, while regions that are normally marked by H3K9me3 became methylated at H3K27. Elimination of DNA methylation had no obvious effect on the distribution of H3K27me. Elimination of HP1, which \"reads\" H3K9me3, also caused major changes in the distribution of H3K27me, indicating that HP1 is important for normal localization of facultative heterochromatin. Because loss of HP1 caused redistribution of H3K27me2/3, but not H3K9me3, these normally nonoverlapping marks became superimposed. Indeed, mass spectrometry revealed substantial cohabitation of H3K9me3 and H3K27me2 on H3 molecules from an hpo strain. Loss of H3K27me machinery (e.g., the methyltransferase SET-7) did not impact constitutive heterochromatin but partially rescued the slow growth of the DCDC mutants, suggesting that the poor growth of these mutants is partly attributable to ectopic H3K27me. Altogether, our findings with Neurospora clarify interactions of facultative and constitutive heterochromatin in eukaryotes.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Genome research',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2016 Jan'},\n",
       " {'PMID': '26463163',\n",
       "  'Title': 'Direct target network of the Neurospora crassa plant cell wall deconstruction regulators CLR-1, CLR-2, and XLR-1.',\n",
       "  'Abstract': 'Fungal deconstruction of the plant cell requires a complex orchestration of a wide array of intracellular and extracellular enzymes. In Neurospora crassa, CLR-1, CLR-2, and XLR-1 have been identified as key transcription factors regulating plant cell wall degradation in response to soluble sugars. The XLR-1 regulon was defined using a constitutively active mutant allele, resulting in hemicellulase gene expression and secretion under noninducing conditions. To define genes directly regulated by CLR-1, CLR-2, and XLR-1, we performed chromatin immunoprecipitation and next-generation sequencing (ChIPseq) on epitope-tagged constructs of these three transcription factors. When N. crassa is exposed to plant cell wall material, CLR-1, CLR-2, and XLR-1 individually bind to the promoters of the most strongly induced genes in their respective regulons. These include promoters of genes encoding cellulases for CLR-1 and CLR-2 (CLR-1/CLR-2) and promoters of genes encoding hemicellulases for XLR-1. CLR-1 bound to its regulon under noninducing conditions; however, this binding alone did not translate into gene expression and enzyme secretion. Motif analysis of the bound genes revealed conserved DNA binding motifs, with the CLR-2 motif matching that of its closest paralog in Saccharomyces cerevisiae, Gal4p. Coimmunoprecipitation studies showed that CLR-1 and CLR-2 act in a homocomplex but not as a CLR-1/CLR-2 heterocomplex. IMPORTANCE: Understanding fungal regulation of complex plant cell wall deconstruction pathways in response to multiple environmental signals via interconnected transcriptional circuits provides insight into fungus/plant interactions and eukaryotic nutrient sensing. Coordinated optimization of these regulatory networks is likely required for optimal microbial enzyme production.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'mBio',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2015 Oct 13'},\n",
       " {'PMID': '26463163',\n",
       "  'Title': 'Direct target network of the Neurospora crassa plant cell wall deconstruction regulators CLR-1, CLR-2, and XLR-1.',\n",
       "  'Abstract': 'Fungal deconstruction of the plant cell requires a complex orchestration of a wide array of intracellular and extracellular enzymes. In Neurospora crassa, CLR-1, CLR-2, and XLR-1 have been identified as key transcription factors regulating plant cell wall degradation in response to soluble sugars. The XLR-1 regulon was defined using a constitutively active mutant allele, resulting in hemicellulase gene expression and secretion under noninducing conditions. To define genes directly regulated by CLR-1, CLR-2, and XLR-1, we performed chromatin immunoprecipitation and next-generation sequencing (ChIPseq) on epitope-tagged constructs of these three transcription factors. When N. crassa is exposed to plant cell wall material, CLR-1, CLR-2, and XLR-1 individually bind to the promoters of the most strongly induced genes in their respective regulons. These include promoters of genes encoding cellulases for CLR-1 and CLR-2 (CLR-1/CLR-2) and promoters of genes encoding hemicellulases for XLR-1. CLR-1 bound to its regulon under noninducing conditions; however, this binding alone did not translate into gene expression and enzyme secretion. Motif analysis of the bound genes revealed conserved DNA binding motifs, with the CLR-2 motif matching that of its closest paralog in Saccharomyces cerevisiae, Gal4p. Coimmunoprecipitation studies showed that CLR-1 and CLR-2 act in a homocomplex but not as a CLR-1/CLR-2 heterocomplex. IMPORTANCE: Understanding fungal regulation of complex plant cell wall deconstruction pathways in response to multiple environmental signals via interconnected transcriptional circuits provides insight into fungus/plant interactions and eukaryotic nutrient sensing. Coordinated optimization of these regulatory networks is likely required for optimal microbial enzyme production.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'mBio',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2015 Oct 13'},\n",
       " {'PMID': '26463163',\n",
       "  'Title': 'Direct target network of the Neurospora crassa plant cell wall deconstruction regulators CLR-1, CLR-2, and XLR-1.',\n",
       "  'Abstract': 'Fungal deconstruction of the plant cell requires a complex orchestration of a wide array of intracellular and extracellular enzymes. In Neurospora crassa, CLR-1, CLR-2, and XLR-1 have been identified as key transcription factors regulating plant cell wall degradation in response to soluble sugars. The XLR-1 regulon was defined using a constitutively active mutant allele, resulting in hemicellulase gene expression and secretion under noninducing conditions. To define genes directly regulated by CLR-1, CLR-2, and XLR-1, we performed chromatin immunoprecipitation and next-generation sequencing (ChIPseq) on epitope-tagged constructs of these three transcription factors. When N. crassa is exposed to plant cell wall material, CLR-1, CLR-2, and XLR-1 individually bind to the promoters of the most strongly induced genes in their respective regulons. These include promoters of genes encoding cellulases for CLR-1 and CLR-2 (CLR-1/CLR-2) and promoters of genes encoding hemicellulases for XLR-1. CLR-1 bound to its regulon under noninducing conditions; however, this binding alone did not translate into gene expression and enzyme secretion. Motif analysis of the bound genes revealed conserved DNA binding motifs, with the CLR-2 motif matching that of its closest paralog in Saccharomyces cerevisiae, Gal4p. Coimmunoprecipitation studies showed that CLR-1 and CLR-2 act in a homocomplex but not as a CLR-1/CLR-2 heterocomplex. IMPORTANCE: Understanding fungal regulation of complex plant cell wall deconstruction pathways in response to multiple environmental signals via interconnected transcriptional circuits provides insight into fungus/plant interactions and eukaryotic nutrient sensing. Coordinated optimization of these regulatory networks is likely required for optimal microbial enzyme production.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'mBio',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2015 Oct 13'},\n",
       " {'PMID': '26321254',\n",
       "  'Title': 'Codon Usage Influences the Local Rate of Translation Elongation to Regulate Co-translational Protein Folding.',\n",
       "  'Abstract': 'Codon usage bias is a universal feature of eukaryotic and prokaryotic genomes and has been proposed to regulate translation efficiency, accuracy, and protein folding based on the assumption that codon usage affects translation dynamics. The roles of codon usage in translation, however, are not clear and have been challenged by recent ribosome profiling studies. Here we used a Neurospora cell-free translation system to directly monitor the velocity of mRNA translation. We demonstrated that the preferred codons enhance the rate of translation elongation, whereas non-optimal codons slow elongation. Codon usage also controls ribosome traffic on mRNA. These conclusions were supported by ribosome profiling results in vitro and in vivo with template mRNAs designed to increase the signal-to-noise ratio. Finally, we demonstrate that codon usage regulates protein function by affecting co-translational protein folding. These results resolve a long-standing fundamental question and suggest the existence of a codon usage code for protein folding.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Molecular cell',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2015 Sep 3'},\n",
       " {'PMID': '25435910',\n",
       "  'Title': 'Leveraging transcription factors to speed cellobiose fermentation by Saccharomyces cerevisiae.',\n",
       "  'Abstract': 'BACKGROUND: Saccharomyces cerevisiae, a key organism used for the manufacture of renewable fuels and chemicals, has been engineered to utilize non-native sugars derived from plant cell walls, such as cellobiose and xylose. However, the rates and efficiencies of these non-native sugar fermentations pale in comparison with those of glucose. Systems biology methods, used to understand biological networks, hold promise for rational microbial strain development in metabolic engineering. Here, we present a systematic strategy for optimizing non-native sugar fermentation by recombinant S. cerevisiae, using cellobiose as a model. RESULTS: Differences in gene expression between cellobiose and glucose metabolism revealed by RNA deep sequencing indicated that cellobiose metabolism induces mitochondrial activation and reduces amino acid biosynthesis under fermentation conditions. Furthermore, glucose-sensing and signaling pathways and their target genes, including the cAMP-dependent protein kinase A pathway controlling the majority of glucose-induced changes, the Snf3-Rgt2-Rgt1 pathway regulating hexose transport, and the Snf1-Mig1 glucose repression pathway, were at most only partially activated under cellobiose conditions. To separate correlations from causative effects, the expression levels of 19 transcription factors perturbed under cellobiose conditions were modulated, and the three strongest promoters under cellobiose conditions were applied to fine-tune expression of the heterologous cellobiose-utilizing pathway. Of the changes in these 19 transcription factors, only overexpression of SUT1 or deletion of HAP4 consistently improved cellobiose fermentation. SUT1 overexpression and HAP4 deletion were not synergistic, suggesting that SUT1 and HAP4 may regulate overlapping genes important for improved cellobiose fermentation. Transcription factor modulation coupled with rational tuning of the cellobiose consumption pathway significantly improved cellobiose fermentation. CONCLUSIONS: We used systems-level input to reveal the regulatory mechanisms underlying suboptimal metabolism of the non-glucose sugar cellobiose. By identifying key transcription factors that cause suboptimal cellobiose fermentation in engineered S. cerevisiae, and by fine-tuning the expression of a heterologous cellobiose consumption pathway, we were able to greatly improve cellobiose fermentation by engineered S. cerevisiae. Our results demonstrate a powerful strategy for applying systems biology methods to rapidly identify metabolic engineering targets and overcome bottlenecks in performance of engineered strains.',\n",
       "  'Keywords': ['Biofuels',\n",
       "   'Cellobiose',\n",
       "   'Glycolysis',\n",
       "   'Metabolic engineering',\n",
       "   'Systems biology',\n",
       "   'Transcription factor'],\n",
       "  'Journal': 'Biotechnology for biofuels',\n",
       "  'Country': 'CA 94720 USA',\n",
       "  'Year': '2014'},\n",
       " {'PMID': '25883682',\n",
       "  'Title': 'Genome-wide analysis of the endoplasmic reticulum stress response during lignocellulase production in Neurospora crassa.',\n",
       "  'Abstract': 'BACKGROUND: Lignocellulolytic fungal cells suffer endoplasmic reticulum (ER) stress during lignocellulase synthesis; however, an understanding of this integrated process on a genome-wide scale remains poor. Here, we undertook a systematic investigation of this process in Neurospora crassa (N. crassa) using transcriptomic analysis coupled with genetic screens. RESULTS: A set of 766 genes was identified as the ER stress response targets (ESRTs) in N. crassa under cellulose utilization conditions. Among these, the expression of 223 and 186 genes showed dependence on IRE-1 and HAC-1, respectively. A total of 527 available mutants for ESRT genes were screened, 249 of which exhibited ER stress susceptibility, including 100 genes with unknown function. Disruption of ire-1 or hac-1 in N. crassa did not affect transcriptional induction of lignocellulase genes by cellulose but severely affected secretion of the corresponding enzymes. A global investigation of transcription factors (TFs) discovered three novel regulators (RES-1, RES-2, RRG-2) involved in lignocellulase secretion. Production of lignocellulases in Deltares-1 increased by more than 30% in comparison to wild type (WT), while secretion decreased by nearly 30% in strains Deltares-2 and Deltarrg-2. Transcriptional profiling of the three TF mutants suggests they are deeply involved in lignocellulase secretion and ER stress response. CONCLUSIONS: Here, we determined the transcriptional scope of the ER stress response during lignocellulase synthesis in the model cellulolytic fungus N. crassa. Through genome-wide mutant screening and analysis, dozens of novel genes were discovered to be involved in the process. The findings of this work will be useful for strain improvement to facilitate lignocellulase and biomass-based chemical production.',\n",
       "  'Keywords': ['ER stress',\n",
       "   'Lignocellulase secretion',\n",
       "   'RESS',\n",
       "   'RNA-seq',\n",
       "   'Transcription factor',\n",
       "   'UPR'],\n",
       "  'Journal': 'Biotechnology for biofuels',\n",
       "  'Country': '100049 China',\n",
       "  'Year': '2015'},\n",
       " {'PMID': '25793375',\n",
       "  'Title': 'Neurospora importin alpha is required for normal heterochromatic formation and DNA methylation.',\n",
       "  'Abstract': 'Heterochromatin and associated gene silencing processes play roles in development, genome defense, and chromosome function. In many species, constitutive heterochromatin is decorated with histone H3 tri-methylated at lysine 9 (H3K9me3) and cytosine methylation. In Neurospora crassa, a five-protein complex, DCDC, catalyzes H3K9 methylation, which then directs DNA methylation. Here, we identify and characterize a gene important for DCDC function, dim-3 (defective in methylation-3), which encodes the nuclear import chaperone NUP-6 (Importin alpha). The critical mutation in dim-3 results in a substitution in an ARM repeat of NUP-6 and causes a substantial loss of H3K9me3 and DNA methylation. Surprisingly, nuclear transport of all known proteins involved in histone and DNA methylation, as well as a canonical transport substrate, appear normal in dim-3 strains. Interactions between DCDC members also appear normal, but the nup-6(dim-3) allele causes the DCDC members DIM-5 and DIM-7 to mislocalize from heterochromatin and NUP-6dim-3 itself is mislocalized from the nuclear envelope, at least in conidia. GCN-5, a member of the SAGA histone acetyltransferase complex, also shows altered localization in dim-3, raising the possibility that NUP-6 is necessary to localize multiple chromatin complexes following nucleocytoplasmic transport.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS genetics',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2015 Mar'},\n",
       " {'PMID': '25793375',\n",
       "  'Title': 'Neurospora importin alpha is required for normal heterochromatic formation and DNA methylation.',\n",
       "  'Abstract': 'Heterochromatin and associated gene silencing processes play roles in development, genome defense, and chromosome function. In many species, constitutive heterochromatin is decorated with histone H3 tri-methylated at lysine 9 (H3K9me3) and cytosine methylation. In Neurospora crassa, a five-protein complex, DCDC, catalyzes H3K9 methylation, which then directs DNA methylation. Here, we identify and characterize a gene important for DCDC function, dim-3 (defective in methylation-3), which encodes the nuclear import chaperone NUP-6 (Importin alpha). The critical mutation in dim-3 results in a substitution in an ARM repeat of NUP-6 and causes a substantial loss of H3K9me3 and DNA methylation. Surprisingly, nuclear transport of all known proteins involved in histone and DNA methylation, as well as a canonical transport substrate, appear normal in dim-3 strains. Interactions between DCDC members also appear normal, but the nup-6(dim-3) allele causes the DCDC members DIM-5 and DIM-7 to mislocalize from heterochromatin and NUP-6dim-3 itself is mislocalized from the nuclear envelope, at least in conidia. GCN-5, a member of the SAGA histone acetyltransferase complex, also shows altered localization in dim-3, raising the possibility that NUP-6 is necessary to localize multiple chromatin complexes following nucleocytoplasmic transport.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS genetics',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2015 Mar'},\n",
       " {'PMID': '25793375',\n",
       "  'Title': 'Neurospora importin alpha is required for normal heterochromatic formation and DNA methylation.',\n",
       "  'Abstract': 'Heterochromatin and associated gene silencing processes play roles in development, genome defense, and chromosome function. In many species, constitutive heterochromatin is decorated with histone H3 tri-methylated at lysine 9 (H3K9me3) and cytosine methylation. In Neurospora crassa, a five-protein complex, DCDC, catalyzes H3K9 methylation, which then directs DNA methylation. Here, we identify and characterize a gene important for DCDC function, dim-3 (defective in methylation-3), which encodes the nuclear import chaperone NUP-6 (Importin alpha). The critical mutation in dim-3 results in a substitution in an ARM repeat of NUP-6 and causes a substantial loss of H3K9me3 and DNA methylation. Surprisingly, nuclear transport of all known proteins involved in histone and DNA methylation, as well as a canonical transport substrate, appear normal in dim-3 strains. Interactions between DCDC members also appear normal, but the nup-6(dim-3) allele causes the DCDC members DIM-5 and DIM-7 to mislocalize from heterochromatin and NUP-6dim-3 itself is mislocalized from the nuclear envelope, at least in conidia. GCN-5, a member of the SAGA histone acetyltransferase complex, also shows altered localization in dim-3, raising the possibility that NUP-6 is necessary to localize multiple chromatin complexes following nucleocytoplasmic transport.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS genetics',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2015 Mar'},\n",
       " {'PMID': '26587879',\n",
       "  'Title': 'Evolutionary Conservation and Diversification of Puf RNA Binding Proteins and Their mRNA Targets.',\n",
       "  'Abstract': \"Reprogramming of a gene's expression pattern by acquisition and loss of sequences recognized by specific regulatory RNA binding proteins may be a major mechanism in the evolution of biological regulatory programs. We identified that RNA targets of Puf3 orthologs have been conserved over 100-500 million years of evolution in five eukaryotic lineages. Focusing on Puf proteins and their targets across 80 fungi, we constructed a parsimonious model for their evolutionary history. This model entails extensive and coordinated changes in the Puf targets as well as changes in the number of Puf genes and alterations of RNA binding specificity including that: 1) Binding of Puf3 to more than 200 RNAs whose protein products are predominantly involved in the production and organization of mitochondrial complexes predates the origin of budding yeasts and filamentous fungi and was maintained for 500 million years, throughout the evolution of budding yeast. 2) In filamentous fungi, remarkably, more than 150 of the ancestral Puf3 targets were gained by Puf4, with one lineage maintaining both Puf3 and Puf4 as regulators and a sister lineage losing Puf3 as a regulator of these RNAs. The decrease in gene expression of these mRNAs upon deletion of Puf4 in filamentous fungi (N. crassa) in contrast to the increase upon Puf3 deletion in budding yeast (S. cerevisiae) suggests that the output of the RNA regulatory network is different with Puf4 in filamentous fungi than with Puf3 in budding yeast. 3) The coregulated Puf4 target set in filamentous fungi expanded to include mitochondrial genes involved in the tricarboxylic acid (TCA) cycle and other nuclear-encoded RNAs with mitochondrial function not bound by Puf3 in budding yeast, observations that provide additional evidence for substantial rewiring of post-transcriptional regulation. 4) Puf3 also expanded and diversified its targets in filamentous fungi, gaining interactions with the mRNAs encoding the mitochondrial electron transport chain (ETC) complex I as well as hundreds of other mRNAs with nonmitochondrial functions. The many concerted and conserved changes in the RNA targets of Puf proteins strongly support an extensive role of RNA binding proteins in coordinating gene expression, as originally proposed by Keene. Rewiring of Puf-coordinated mRNA targets and transcriptional control of the same genes occurred at different points in evolution, suggesting that there have been distinct adaptations via RNA binding proteins and transcription factors. The changes in Puf targets and in the Puf proteins indicate an integral involvement of RNA binding proteins and their RNA targets in the adaptation, reprogramming, and function of gene expression.\",\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS biology',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2015'},\n",
       " {'PMID': '26587879',\n",
       "  'Title': 'Evolutionary Conservation and Diversification of Puf RNA Binding Proteins and Their mRNA Targets.',\n",
       "  'Abstract': \"Reprogramming of a gene's expression pattern by acquisition and loss of sequences recognized by specific regulatory RNA binding proteins may be a major mechanism in the evolution of biological regulatory programs. We identified that RNA targets of Puf3 orthologs have been conserved over 100-500 million years of evolution in five eukaryotic lineages. Focusing on Puf proteins and their targets across 80 fungi, we constructed a parsimonious model for their evolutionary history. This model entails extensive and coordinated changes in the Puf targets as well as changes in the number of Puf genes and alterations of RNA binding specificity including that: 1) Binding of Puf3 to more than 200 RNAs whose protein products are predominantly involved in the production and organization of mitochondrial complexes predates the origin of budding yeasts and filamentous fungi and was maintained for 500 million years, throughout the evolution of budding yeast. 2) In filamentous fungi, remarkably, more than 150 of the ancestral Puf3 targets were gained by Puf4, with one lineage maintaining both Puf3 and Puf4 as regulators and a sister lineage losing Puf3 as a regulator of these RNAs. The decrease in gene expression of these mRNAs upon deletion of Puf4 in filamentous fungi (N. crassa) in contrast to the increase upon Puf3 deletion in budding yeast (S. cerevisiae) suggests that the output of the RNA regulatory network is different with Puf4 in filamentous fungi than with Puf3 in budding yeast. 3) The coregulated Puf4 target set in filamentous fungi expanded to include mitochondrial genes involved in the tricarboxylic acid (TCA) cycle and other nuclear-encoded RNAs with mitochondrial function not bound by Puf3 in budding yeast, observations that provide additional evidence for substantial rewiring of post-transcriptional regulation. 4) Puf3 also expanded and diversified its targets in filamentous fungi, gaining interactions with the mRNAs encoding the mitochondrial electron transport chain (ETC) complex I as well as hundreds of other mRNAs with nonmitochondrial functions. The many concerted and conserved changes in the RNA targets of Puf proteins strongly support an extensive role of RNA binding proteins in coordinating gene expression, as originally proposed by Keene. Rewiring of Puf-coordinated mRNA targets and transcriptional control of the same genes occurred at different points in evolution, suggesting that there have been distinct adaptations via RNA binding proteins and transcription factors. The changes in Puf targets and in the Puf proteins indicate an integral involvement of RNA binding proteins and their RNA targets in the adaptation, reprogramming, and function of gene expression.\",\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS biology',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2015'},\n",
       " {'PMID': '25691917',\n",
       "  'Title': 'A transcriptomic analysis of Neurospora crassa using five major crop residues and the novel role of the sporulation regulator rca-1 in lignocellulase production.',\n",
       "  'Abstract': 'BACKGROUND: Crop residue is an abundant, low-cost plant biomass material available worldwide for use in the microbial production of enzymes, biofuels, and valuable chemicals. However, the diverse chemical composition and complex structure of crop residues are more challenging for efficient degradation by microbes than are homogeneous polysaccharides. In this study, the transcriptional responses of Neurospora crassa to various plant straws were analyzed using RNA-Seq, and novel beneficial factors for biomass-induced enzyme production were evaluated. RESULTS: Comparative transcriptional profiling of N. crassa grown on five major crop straws of China (barley, corn, rice, soybean, and wheat straws) revealed a highly overlapping group of 430 genes, the biomass commonly induced core set (BICS). A large proportion of induced carbohydrate-active enzyme (CAZy) genes (82 out of 113) were also conserved across the five plant straws. Excluding 178 genes within the BICS that were also upregulated under no-carbon conditions, the remaining 252 genes were defined as the biomass regulon (BR). Interestingly, 88 genes were only induced by plant biomass and not by three individual polysaccharides (Avicel, xylan, and pectin); these were denoted as the biomass unique set (BUS). Deletion of one BUS gene, the transcriptional regulator rca-1, significantly improved lignocellulase production using plant biomass as the sole carbon source, possibly functioning via de-repression of the regulator clr-2. Thus, this result suggests that rca-1 is a potential engineering target for biorefineries, especially for plant biomass direct microbial conversion processes. CONCLUSIONS: Transcriptional profiling revealed a large core response to different sources of plant biomass in N. crassa. The sporulation regulator rca-1 was identified as beneficial for biomass-based enzyme production.',\n",
       "  'Keywords': ['Biomass regulon',\n",
       "   'Crop residues',\n",
       "   'Neurospora crassa',\n",
       "   'Transcriptional profiling',\n",
       "   'rca-1'],\n",
       "  'Journal': 'Biotechnology for biofuels',\n",
       "  'Country': '100049 China',\n",
       "  'Year': '2015'},\n",
       " {'PMID': '25329823',\n",
       "  'Title': 'Gene expression differences among three Neurospora species reveal genes required for sexual reproduction in Neurospora crassa.',\n",
       "  'Abstract': 'Many fungi form complex three-dimensional fruiting bodies, within which the meiotic machinery for sexual spore production has been considered to be largely conserved over evolutionary time. Indeed, much of what we know about meiosis in plant and animal taxa has been deeply informed by studies of meiosis in Saccharomyces and Neurospora. Nevertheless, the genetic basis of fruiting body development and its regulation in relation to meiosis in fungi is barely known, even within the best studied multicellular fungal model Neurospora crassa. We characterized morphological development and genome-wide transcriptomics in the closely related species Neurospora crassa, Neurospora tetrasperma, and Neurospora discreta, across eight stages of sexual development. Despite diverse life histories within the genus, all three species produce vase-shaped perithecia. Transcriptome sequencing provided gene expression levels of orthologous genes among all three species. Expression of key meiosis genes and sporulation genes corresponded to known phenotypic and developmental differences among these Neurospora species during sexual development. We assembled a list of genes putatively relevant to the recent evolution of fruiting body development by sorting genes whose relative expression across developmental stages increased more in N. crassa relative to the other species. Then, in N. crassa, we characterized the phenotypes of fruiting bodies arising from crosses of homozygous knockout strains of the top genes. Eight N. crassa genes were found to be critical for the successful formation of perithecia. The absence of these genes in these crosses resulted in either no perithecium formation or in arrested development at an early stage. Our results provide insight into the genetic basis of Neurospora sexual reproduction, which is also of great importance with regard to other multicellular ascomycetes, including perithecium-forming pathogens, such as Claviceps purpurea, Ophiostoma ulmi, and Glomerella graminicola.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PloS one',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2014'},\n",
       " {'PMID': '25329823',\n",
       "  'Title': 'Gene expression differences among three Neurospora species reveal genes required for sexual reproduction in Neurospora crassa.',\n",
       "  'Abstract': 'Many fungi form complex three-dimensional fruiting bodies, within which the meiotic machinery for sexual spore production has been considered to be largely conserved over evolutionary time. Indeed, much of what we know about meiosis in plant and animal taxa has been deeply informed by studies of meiosis in Saccharomyces and Neurospora. Nevertheless, the genetic basis of fruiting body development and its regulation in relation to meiosis in fungi is barely known, even within the best studied multicellular fungal model Neurospora crassa. We characterized morphological development and genome-wide transcriptomics in the closely related species Neurospora crassa, Neurospora tetrasperma, and Neurospora discreta, across eight stages of sexual development. Despite diverse life histories within the genus, all three species produce vase-shaped perithecia. Transcriptome sequencing provided gene expression levels of orthologous genes among all three species. Expression of key meiosis genes and sporulation genes corresponded to known phenotypic and developmental differences among these Neurospora species during sexual development. We assembled a list of genes putatively relevant to the recent evolution of fruiting body development by sorting genes whose relative expression across developmental stages increased more in N. crassa relative to the other species. Then, in N. crassa, we characterized the phenotypes of fruiting bodies arising from crosses of homozygous knockout strains of the top genes. Eight N. crassa genes were found to be critical for the successful formation of perithecia. The absence of these genes in these crosses resulted in either no perithecium formation or in arrested development at an early stage. Our results provide insight into the genetic basis of Neurospora sexual reproduction, which is also of great importance with regard to other multicellular ascomycetes, including perithecium-forming pathogens, such as Claviceps purpurea, Ophiostoma ulmi, and Glomerella graminicola.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PloS one',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2014'},\n",
       " {'PMID': '25595444',\n",
       "  'Title': 'Identification and characterization of LFD-2, a predicted fringe protein required for membrane integrity during cell fusion in neurospora crassa.',\n",
       "  'Abstract': 'The molecular mechanisms of membrane merger during somatic cell fusion in eukaryotic species are poorly understood. In the filamentous fungus Neurospora crassa, somatic cell fusion occurs between genetically identical germinated asexual spores (germlings) and between hyphae to form the interconnected network characteristic of a filamentous fungal colony. In N. crassa, two proteins have been identified to function at the step of membrane fusion during somatic cell fusion: PRM1 and LFD-1. The absence of either one of these two proteins results in an increase of germling pairs arrested during cell fusion with tightly appressed plasma membranes and an increase in the frequency of cell lysis of adhered germlings. The level of cell lysis in DeltaPrm1 or Deltalfd-1 germlings is dependent on the extracellular calcium concentration. An available transcriptional profile data set was used to identify genes encoding predicted transmembrane proteins that showed reduced expression levels in germlings cultured in the absence of extracellular calcium. From these analyses, we identified a mutant (lfd-2, for late fusion defect-2) that showed a calcium-dependent cell lysis phenotype. lfd-2 encodes a protein with a Fringe domain and showed endoplasmic reticulum and Golgi membrane localization. The deletion of an additional gene predicted to encode a low-affinity calcium transporter, fig1, also resulted in a strain that showed a calcium-dependent cell lysis phenotype. Genetic analyses showed that LFD-2 and FIG1 likely function in separate pathways to regulate aspects of membrane merger and repair during cell fusion.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Eukaryotic cell',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2015 Mar'},\n",
       " {'PMID': '25398875',\n",
       "  'Title': 'The putative cellodextrin transporter-like protein CLP1 is involved in cellulase induction in Neurospora crassa.',\n",
       "  'Abstract': 'Neurospora crassa recently has become a novel system to investigate cellulase induction. Here, we discovered a novel membrane protein, cellodextrin transporter-like protein 1 (CLP1; NCU05853), a putative cellodextrin transporter-like protein that is a critical component of the cellulase induction pathway in N. crassa. Although CLP1 protein cannot transport cellodextrin, the suppression of cellulase induction by this protein was discovered on both cellobiose and Avicel. The co-disruption of the cellodextrin transporters cdt2 and clp1 in strain Delta3betaG formed strain CPL7. With induction by cellobiose, cellulase production was enhanced 6.9-fold in CPL7 compared with Delta3betaG. We also showed that the suppression of cellulase expression by CLP1 occurred by repressing the expression of cellodextrin transporters, particularly cdt1 expression. Transcriptome analysis of the hypercellulase-producing strain CPL7 showed that the cellulase expression machinery was dramatically stimulated, as were the cellulase enzyme genes including the inducer transporters and the major transcriptional regulators.',\n",
       "  'Keywords': ['Bioenergy',\n",
       "   'Biofuel',\n",
       "   'CLP1',\n",
       "   'Cellodextrin Transporter',\n",
       "   'Cellulase',\n",
       "   'Fungi',\n",
       "   'Neurospora crassa',\n",
       "   'Transporter'],\n",
       "  'Journal': 'The Journal of biological chemistry',\n",
       "  'Country': 'China',\n",
       "  'Year': '2015 Jan 9'},\n",
       " {'PMID': '25053707',\n",
       "  'Title': 'Genome-wide characterization of light-regulated genes in Neurospora crassa.',\n",
       "  'Abstract': 'The filamentous fungus Neurospora crassa responds to light in complex ways. To thoroughly study the transcriptional response of this organism to light, RNA-seq was used to analyze capped and polyadenylated mRNA prepared from mycelium grown for 24 hr in the dark and then exposed to light for 0 (control) 15, 60, 120, and 240 min. More than three-quarters of all defined protein coding genes (79%) were expressed in these cells. The increased sensitivity of RNA-seq compared with previous microarray studies revealed that the RNA levels for 31% of expressed genes were affected two-fold or more by exposure to light. Additionally, a large class of mRNAs, enriched for transcripts specifying products involved in rRNA metabolism, showed decreased expression in response to light, indicating a heretofore undocumented effect of light on this pathway. Based on measured changes in mRNA levels, light generally increases cellular metabolism and at the same time causes significant oxidative stress to the organism. To deal with this stress, protective photopigments are made, antioxidants are produced, and genes involved in ribosome biogenesis are transiently repressed.',\n",
       "  'Keywords': ['Neurospora', 'RNA-seq', 'light'],\n",
       "  'Journal': 'G3 (Bethesda, Md.)',\n",
       "  'Country': 'Texas 77843-3258',\n",
       "  'Year': '2014 Jul 21'},\n",
       " {'PMID': '25215497',\n",
       "  'Title': 'Determination and inference of eukaryotic transcription factor sequence specificity.',\n",
       "  'Abstract': 'Transcription factor (TF) DNA sequence preferences direct their regulatory activity, but are currently known for only  approximately 1% of eukaryotic TFs. Broadly sampling DNA-binding domain (DBD) types from multiple eukaryotic clades, we determined DNA sequence preferences for >1,000 TFs encompassing 54 different DBD classes from 131 diverse eukaryotes. We find that closely related DBDs almost always have very similar DNA sequence preferences, enabling inference of motifs for  approximately 34% of the  approximately 170,000 known or predicted eukaryotic TFs. Sequences matching both measured and inferred motifs are enriched in chromatin immunoprecipitation sequencing (ChIP-seq) peaks and upstream of transcription start sites in diverse eukaryotic lineages. SNPs defining expression quantitative trait loci in Arabidopsis promoters are also enriched for predicted TF binding sites. Importantly, our motif \"library\" can be used to identify specific TFs whose binding may be altered by human disease risk alleles. These data present a powerful resource for mapping transcriptional networks across eukaryotes.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Cell',\n",
       "  'Country': 'Canada',\n",
       "  'Year': '2014 Sep 11'},\n",
       " {'PMID': '25043916',\n",
       "  'Title': 'Elucidating how the saprophytic fungus Aspergillus nidulans uses the plant polyester suberin as carbon source.',\n",
       "  'Abstract': 'BACKGROUND: Lipid polymers in plant cell walls, such as cutin and suberin, build recalcitrant hydrophobic protective barriers. Their degradation is of foremost importance for both plant pathogenic and saprophytic fungi. Regardless of numerous reports on fungal degradation of emulsified fatty acids or cutin, and on fungi-plant interactions, the pathways involved in the degradation and utilisation of suberin remain largely overlooked. As a structural component of the plant cell wall, suberin isolation, in general, uses harsh depolymerisation methods that destroy its macromolecular structure. We recently overcame this limitation isolating suberin macromolecules in a near-native state. RESULTS: Suberin macromolecules were used here to analyse the pathways involved in suberin degradation and utilisation by Aspergillus nidulans. Whole-genome profiling data revealed the complex degrading enzymatic machinery used by this saprophytic fungus. Initial suberin modification involved ester hydrolysis and omega-hydroxy fatty acid oxidation that released long chain fatty acids. These fatty acids were processed through peroxisomal beta-oxidation, leading to up-regulation of genes encoding the major enzymes of these pathways (e.g. faaB and aoxA). The obtained transcriptome data was further complemented by secretome, microscopic and spectroscopic analyses. CONCLUSIONS: Data support that during fungal growth on suberin, cutinase 1 and some lipases (e.g. AN8046) acted as the major suberin degrading enzymes (regulated by FarA and possibly by some unknown regulatory elements). Suberin also induced the onset of sexual development and the boost of secondary metabolism.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'Portugal. spereira@itqb.unl.pt',\n",
       "  'Year': '2014 Jul 21'},\n",
       " {'PMID': '25144221',\n",
       "  'Title': 'VIB1, a link between glucose signaling and carbon catabolite repression, is essential for plant cell wall degradation by Neurospora crassa.',\n",
       "  'Abstract': 'Filamentous fungi that thrive on plant biomass are the major producers of hydrolytic enzymes used to decompose lignocellulose for biofuel production. Although induction of cellulases is regulated at the transcriptional level, how filamentous fungi sense and signal carbon-limited conditions to coordinate cell metabolism and regulate cellulolytic enzyme production is not well characterized. By screening a transcription factor deletion set in the filamentous fungus Neurospora crassa for mutants unable to grow on cellulosic materials, we identified a role for the transcription factor, VIB1, as essential for cellulose utilization. VIB1 does not directly regulate hydrolytic enzyme gene expression or function in cellulosic inducer signaling/processing, but affects the expression level of an essential regulator of hydrolytic enzyme genes, CLR2. Transcriptional profiling of a Deltavib-1 mutant suggests that it has an improper expression of genes functioning in metabolism and energy and a deregulation of carbon catabolite repression (CCR). By characterizing new genes, we demonstrate that the transcription factor, COL26, is critical for intracellular glucose sensing/metabolism and plays a role in CCR by negatively regulating cre-1 expression. Deletion of the major player in CCR, cre-1, or a deletion of col-26, did not rescue the growth of Deltavib-1 on cellulose. However, the synergistic effect of the Deltacre-1; Deltacol-26 mutations circumvented the requirement of VIB1 for cellulase gene expression, enzyme secretion and cellulose deconstruction. Our findings support a function of VIB1 in repressing both glucose signaling and CCR under carbon-limited conditions, thus enabling a proper cellular response for plant biomass deconstruction and utilization.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PLoS genetics',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2014 Aug'},\n",
       " {'PMID': '24717808',\n",
       "  'Title': 'CZT-1 is a novel transcription factor controlling cell death and natural drug resistance in Neurospora crassa.',\n",
       "  'Abstract': 'We pinpoint CZT-1 (cell death-activated zinc cluster transcription factor) as a novel transcription factor involved in tolerance to cell death induced by the protein kinase inhibitor staurosporine in Neurospora crassa. Transcriptional profiling of staurosporine-treated wild-type cells by RNA-sequencing showed that genes encoding the machinery for protein synthesis are enriched among the genes repressed by the drug. Functional category enrichment analyses also show that genes encoding components of the mitochondrial respiratory chain are downregulated by staurosporine, whereas genes involved in endoplasmic reticulum activities are upregulated. In contrast, a staurosporine-treated Deltaczt-1 deletion strain is unable to repress the genes for the respiratory chain and to induce the genes related to the endoplasmic reticulum, indicating a role for CZT-1 in the regulation of activity of these organelles. The Deltaczt-1 mutant strain displays increased reactive oxygen species accumulation on insult with staurosporine. A genome-wide association study of a wild population of N. crassa isolates pointed out genes associated with a cell death role of CZT-1, including catalase-1 (cat-1) and apoptosis-inducing factor-homologous mitochondrion-associated inducer of death 2 (amid-2). Importantly, differences in the expression of czt-1 correlates with resistance to staurosporine among wild isolate strains. Our results reveal a novel transcription factor that regulates drug resistance and cell death in response to staurosporine in laboratory strains as well as in wild isolates of N. crassa.',\n",
       "  'Keywords': ['RNA-seq',\n",
       "   'cell death',\n",
       "   'genome-wide association study',\n",
       "   'multidrug resistance',\n",
       "   'zinc cluster transcription factor'],\n",
       "  'Journal': 'G3 (Bethesda, Md.)',\n",
       "  'Country': 'Portugal apgoncalves@ibmc.up.pt avideira@ibmc.up.pt',\n",
       "  'Year': '2014 Apr 8'},\n",
       " {'PMID': '24581151',\n",
       "  'Title': 'Transcriptional comparison of the filamentous fungus Neurospora crassa growing on three major monosaccharides D-glucose, D-xylose and L-arabinose.',\n",
       "  'Abstract': 'BACKGROUND: D-glucose, D-xylose and L-arabinose are the three major monosaccharides in plant cell walls. Complete utilization of all three sugars is still a bottleneck for second-generation cellulolytic bioethanol production, especially for L-arabinose. However, little is known about gene expression profiles during L-arabinose utilization in fungi and a comparison of the genome-wide fungal response to these three major monosaccharides has not yet been reported. RESULTS: Using next-generation sequencing technology, we have analyzed the transcriptome of N. crassa grown on L-arabinose versus D-xylose, with D-glucose as the reference. We found that the gene expression profiles on L-arabinose were dramatically different from those on D-xylose. It appears that L-arabinose can rewire the fungal cell metabolic pathway widely and provoke the expression of many kinds of sugar transporters, hemicellulase genes and transcription factors. In contrast, many fewer genes, mainly related to the pentose metabolic pathway, were upregulated on D-xylose. The rewired metabolic response to L-arabinose was significantly different and wider than that under no carbon conditions, although the carbon starvation response was initiated on L-arabinose. Three novel sugar transporters were identified and characterized for their substrates here, including one glucose transporter GLT-1 (NCU01633) and two novel pentose transporters, XAT-1 (NCU01132), XYT-1 (NCU05627). One transcription factor associated with the regulation of hemicellulase genes, HCR-1 (NCU05064) was also characterized in the present study. CONCLUSIONS: We conducted the first transcriptome analysis of Neurospora crassa grown on L-arabinose and performed a comparative analysis with cells grown on D-xylose and D-glucose, which deepens the understanding of the utilization of L-arabinose and D-xylose in filamentous fungi. The dataset generated by this research will be useful for mining target genes for D-xylose and L-arabinose utilization engineering and the novel sugar transportes identified are good targets for pentose untilization and biofuels production. Moreover, hemicellulase production by fungi could be improved by modifying the hemicellulase regulator discovered here.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Biotechnology for biofuels',\n",
       "  'Country': 'China. tian_cg@tib.cas.cn',\n",
       "  'Year': '2014 Feb 28'},\n",
       " {'PMID': '24224966',\n",
       "  'Title': 'A comparative systems analysis of polysaccharide-elicited responses in Neurospora crassa reveals carbon source-specific cellular adaptations.',\n",
       "  'Abstract': 'Filamentous fungi are powerful producers of hydrolytic enzymes for the deconstruction of plant cell wall polysaccharides. However, the central question of how these sugars are perceived in the context of the complex cell wall matrix remains largely elusive. To address this question in a systematic fashion we performed an extensive comparative systems analysis of how the model filamentous fungus Neurospora crassa responds to the three main cell wall polysaccharides: pectin, hemicellulose and cellulose. We found the pectic response to be largely independent of the cellulolytic one with some overlap to hemicellulose, and in its extent surprisingly high, suggesting advantages for the fungus beyond being a mere carbon source. Our approach furthermore allowed us to identify carbon source-specific adaptations, such as the induction of the unfolded protein response on cellulose, and a commonly induced set of 29 genes likely involved in carbon scouting. Moreover, by hierarchical clustering we generated a coexpression matrix useful for the discovery of new components involved in polysaccharide utilization. This is exemplified by the identification of lat-1, which we demonstrate to encode for the physiologically relevant arabinose transporter in Neurospora. The analyses presented here are an important step towards understanding fungal degradation processes of complex biomass.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Molecular microbiology',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2014 Jan'},\n",
       " {'PMID': '22626843',\n",
       "  'Title': 'Sex-specific gene expression during asexual development of Neurospora crassa.',\n",
       "  'Abstract': 'The impact of loci that determine sexual identity upon the asexual, dominant stage of fungal life history has been well studied. To investigate their impact, expression differences between strains of different mating type during asexual development were assayed, with RNA sampled from otherwise largely isogenic mat A and mat a strains of Neurospora crassa at early, middle, and late clonal stages of development. We observed significant differences in overall gene expression between mating types across clonal development, especially at late development stages. The expression levels of mating-type genes and pheromone genes were assayed by reverse transcription and quantitative PCR, revealing expression of pheromone and receptor genes in strains of both mating types in all development stages, and revealing that mating type (mat) genes were increasingly expressed over the course of asexual development. Interestingly, among differentially expressed genes, the mat A genotype more frequently exhibited a higher expression level than mat a, and demonstrated greater transcriptional regulatory dynamism. Significant up-regulation of expression was observed for many late light-responsive genes at late asexual development stages. Further investigation of the impact of light and the roles of light response genes in asexual development of both mating types are warranted.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Fungal genetics and biology : FG & B',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2012 Jul'},\n",
       " {'PMID': '23766336',\n",
       "  'Title': 'Analysis of a conserved cellulase transcriptional regulator reveals inducer-independent production of cellulolytic enzymes in Neurospora crassa.',\n",
       "  'Abstract': 'Cellulose is recalcitrant to deconstruction to glucose for use in fermentation strategies for biofuels and chemicals derived from lignocellulose. In Neurospora crassa, the transcriptional regulator, CLR-2, is required for cellulolytic gene expression and cellulose deconstruction. To assess conservation and divergence of cellulase gene regulation between fungi from different ecological niches, we compared clr-2 function with its ortholog (clrB) in the distantly related species, Aspergillus nidulans. Transcriptional profiles induced by exposure to crystalline cellulose were similar in both species. Approximately 50% of the cellulose-responsive genes showed strict dependence on functional clr-2/clrB, with a subset of 28 genes encoding plant biomass degrading enzymes that were conserved between N. crassa and A. nidulans. Importantly, misexpression of clr-2 under noninducing conditions was sufficient to drive cellulase gene expression, secretion, and activity in N. crassa, to a level comparable to wild type exposed to Avicel. However, misexpression of clrB in A. nidulans was not sufficient to drive cellulase gene expression under noninducing conditions, although an increase in cellulase activity was observed under crystalline cellulose conditions. Manipulation of clr-2 orthologs among filamentous fungi may enable regulated cellulosic enzyme production in a wide array of culture conditions and host strains, potentially reducing costs associated with enzyme production for plant cell wall deconstruction. However, this functionality may require additional engineering in some species.',\n",
       "  'Keywords': ['Aspergillus',\n",
       "   'Neurospora',\n",
       "   'biofuels',\n",
       "   'cellulase',\n",
       "   'transcriptional regulation'],\n",
       "  'Journal': 'MicrobiologyOpen',\n",
       "  'Country': '94720-3102',\n",
       "  'Year': '2013 Aug'},\n",
       " {'PMID': '24037267',\n",
       "  'Title': 'Early colony establishment in Neurospora crassa requires a MAP kinase regulatory network.',\n",
       "  'Abstract': 'Vegetative fusion is essential for the development of an interconnected colony in many filamentous fungi. In the ascomycete fungus Neurospora crassa, vegetative fusion occurs between germinated conidia (germlings) via specialized structures termed \"conidial anastomosis tubes\" (CATs) and between hyphae within a mature colony. In N. crassa, both CAT and hyphal fusion are under the regulation of a conserved MAP kinase cascade (NRC1, MEK2, and MAK2). Here we show that the predicted downstream target of the MAK2 kinase pathway, a Ste12-like transcription factor known as PP1, regulates elements required for CAT and hyphal fusion. The PP1 regulatory network was revealed by expression profiling of wild type and the Deltapp-1 mutant during conidial germination and colony establishment. To identify targets required for cell fusion more specifically, expression-profiling differences were assessed via inhibition of MAK2 kinase activity during chemotropic interactions and cell fusion. These approaches led to the identification of new targets of the cell fusion pathway that, when mutated, showed alterations in chemotropic signaling and cell fusion. In particular, conidial germlings carrying a deletion of NCU04732 (Deltaham-11) failed to show chemotropic interactions and cell fusion. However, signaling (as shown by oscillation of MAK2 and SO to CAT tips), chemotropism, and cell fusion were restored in Deltaham-11 germlings when matched with wild-type partner germlings. These data reveal novel insights into the complex process of self-signaling, germling fusion, and colony establishment in filamentous fungi.',\n",
       "  'Keywords': ['Neurospora crassa',\n",
       "   'STE12',\n",
       "   'cell fusion',\n",
       "   'chemotropism',\n",
       "   'signal transduction',\n",
       "   'transcriptional profiling'],\n",
       "  'Journal': 'Genetics',\n",
       "  'Country': 'California 94720-3102',\n",
       "  'Year': '2013 Nov'},\n",
       " {'PMID': '24037267',\n",
       "  'Title': 'Early colony establishment in Neurospora crassa requires a MAP kinase regulatory network.',\n",
       "  'Abstract': 'Vegetative fusion is essential for the development of an interconnected colony in many filamentous fungi. In the ascomycete fungus Neurospora crassa, vegetative fusion occurs between germinated conidia (germlings) via specialized structures termed \"conidial anastomosis tubes\" (CATs) and between hyphae within a mature colony. In N. crassa, both CAT and hyphal fusion are under the regulation of a conserved MAP kinase cascade (NRC1, MEK2, and MAK2). Here we show that the predicted downstream target of the MAK2 kinase pathway, a Ste12-like transcription factor known as PP1, regulates elements required for CAT and hyphal fusion. The PP1 regulatory network was revealed by expression profiling of wild type and the Deltapp-1 mutant during conidial germination and colony establishment. To identify targets required for cell fusion more specifically, expression-profiling differences were assessed via inhibition of MAK2 kinase activity during chemotropic interactions and cell fusion. These approaches led to the identification of new targets of the cell fusion pathway that, when mutated, showed alterations in chemotropic signaling and cell fusion. In particular, conidial germlings carrying a deletion of NCU04732 (Deltaham-11) failed to show chemotropic interactions and cell fusion. However, signaling (as shown by oscillation of MAK2 and SO to CAT tips), chemotropism, and cell fusion were restored in Deltaham-11 germlings when matched with wild-type partner germlings. These data reveal novel insights into the complex process of self-signaling, germling fusion, and colony establishment in filamentous fungi.',\n",
       "  'Keywords': ['Neurospora crassa',\n",
       "   'STE12',\n",
       "   'cell fusion',\n",
       "   'chemotropism',\n",
       "   'signal transduction',\n",
       "   'transcriptional profiling'],\n",
       "  'Journal': 'Genetics',\n",
       "  'Country': 'California 94720-3102',\n",
       "  'Year': '2013 Nov'},\n",
       " {'PMID': '23733042',\n",
       "  'Title': 'Transcriptional profiling of Neurospora crassa Deltamak-2 reveals that mitogen-activated protein kinase MAK-2 participates in the phosphate signaling pathway.',\n",
       "  'Abstract': 'The filamentous fungus Neurospora crassa is an excellent model system for examining molecular responses to ambient signals in eukaryotic microorganisms. Inorganic phosphate (Pi) is an essential growth-limiting nutrient in nature and is crucial for the synthesis of nucleic acids and the flow of genetic information. The genetic and molecular mechanisms controlling the response to Pi starvation in N. crassa include at least four genes (nuc-2, preg, pogv, and nuc-1), which are involved in a hierarchical regulatory activation network. In a previous work, we identified a number of genes modulated by NUC-2 protein, including the mak-2 gene, which codes for a mitogen-activated protein kinase (MAPK), suggesting its participation in the phosphate signaling pathway. Thus, to identify other genes involved in metabolic responses to exogenous phosphate sensing and the functioning of the MAPK MAK-2, we performed microarray experiments using a mak-2 knockout strain (Deltamak-2) grown under phosphate-shortage conditions by comparing its transcription profile to that of a control strain grown in low- and high-phosphate cultures. These experiments revealed 912 unique differentially expressed genes involved in a number of physiological processes related to phosphate transport, metabolism, and regulation as well as posttranslational modification of proteins, and MAPK signaling pathways. Quantitative Real-time PCR gene expression analysis of 18 selected genes, using independent RNA samples, validated our microarray results. A high Pearson correlation between microarray and quantitative Real-time PCR data was observed. The analysis of these differentially expressed genes in the Deltamak-2 strain provide evidence that the mak-2 gene participates in the hierarchical phosphate-signaling pathway in N. crassa in addition to its involvement in other metabolic routes such as the isoprenylation pathway, thus revealing novel aspects of the N. crassa phosphorus-sensing network.',\n",
       "  'Keywords': ['MAK-2', 'Neurospora crassa', 'Phosphate sensing'],\n",
       "  'Journal': 'Fungal genetics and biology : FG & B',\n",
       "  'Country': 'Brazil',\n",
       "  'Year': '2013 Nov'},\n",
       " {'PMID': '24037267',\n",
       "  'Title': 'Early colony establishment in Neurospora crassa requires a MAP kinase regulatory network.',\n",
       "  'Abstract': 'Vegetative fusion is essential for the development of an interconnected colony in many filamentous fungi. In the ascomycete fungus Neurospora crassa, vegetative fusion occurs between germinated conidia (germlings) via specialized structures termed \"conidial anastomosis tubes\" (CATs) and between hyphae within a mature colony. In N. crassa, both CAT and hyphal fusion are under the regulation of a conserved MAP kinase cascade (NRC1, MEK2, and MAK2). Here we show that the predicted downstream target of the MAK2 kinase pathway, a Ste12-like transcription factor known as PP1, regulates elements required for CAT and hyphal fusion. The PP1 regulatory network was revealed by expression profiling of wild type and the Deltapp-1 mutant during conidial germination and colony establishment. To identify targets required for cell fusion more specifically, expression-profiling differences were assessed via inhibition of MAK2 kinase activity during chemotropic interactions and cell fusion. These approaches led to the identification of new targets of the cell fusion pathway that, when mutated, showed alterations in chemotropic signaling and cell fusion. In particular, conidial germlings carrying a deletion of NCU04732 (Deltaham-11) failed to show chemotropic interactions and cell fusion. However, signaling (as shown by oscillation of MAK2 and SO to CAT tips), chemotropism, and cell fusion were restored in Deltaham-11 germlings when matched with wild-type partner germlings. These data reveal novel insights into the complex process of self-signaling, germling fusion, and colony establishment in filamentous fungi.',\n",
       "  'Keywords': ['Neurospora crassa',\n",
       "   'STE12',\n",
       "   'cell fusion',\n",
       "   'chemotropism',\n",
       "   'signal transduction',\n",
       "   'transcriptional profiling'],\n",
       "  'Journal': 'Genetics',\n",
       "  'Country': 'California 94720-3102',\n",
       "  'Year': '2013 Nov'},\n",
       " {'PMID': '23125351',\n",
       "  'Title': 'Circadian activation of the mitogen-activated protein kinase MAK-1 facilitates rhythms in clock-controlled genes in Neurospora crassa.',\n",
       "  'Abstract': 'The circadian clock regulates the expression of many genes involved in a wide range of biological functions through output pathways such as mitogen-activated protein kinase (MAPK) pathways. We demonstrate here that the clock regulates the phosphorylation, and thus activation, of the MAPKs MAK-1 and MAK-2 in the filamentous fungus Neurospora crassa. In this study, we identified genetic targets of the MAK-1 pathway, which is homologous to the cell wall integrity pathway in Saccharomyces cerevisiae and the extracellular signal-regulated kinase 1/2 (ERK1/2) pathway in mammals. When MAK-1 was deleted from Neurospora cells, vegetative growth was reduced and the transcript levels for over 500 genes were affected, with significant enrichment for genes involved in protein synthesis, biogenesis of cellular components, metabolism, energy production, and transcription. Additionally, of the ~500 genes affected by the disruption of MAK-1, more than 25% were previously identified as putative clock-controlled genes. We show that MAK-1 is necessary for robust rhythms of two morning-specific genes, i.e., ccg-1 and the mitochondrial phosphate carrier protein gene NCU07465. Additionally, we show clock regulation of a predicted chitin synthase gene, NCU04352, whose rhythmic accumulation is also dependent upon MAK-1. Together, these data establish a role for the MAK-1 pathway as an output pathway of the circadian clock and suggest a link between rhythmic MAK-1 activity and circadian control of cellular growth.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Eukaryotic cell',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2013 Jan'},\n",
       " {'PMID': '22773104',\n",
       "  'Title': 'The translin-TRAX complex (C3PO) is a ribonuclease in tRNA processing.',\n",
       "  'Abstract': \"Conserved translin-TRAX complexes, also known as C3POs, have been implicated in many biological processes, but how they function remains unclear. Recently, C3PO was shown to be an endoRNase that promotes RNA interference (RNAi) in animal cells. Here, we show that C3PO does not play a significant role in RNAi in the filamentous fungus Neurospora crassa. Instead, the Neurospora C3PO functions as an RNase that removes the 5' pre-tRNA fragments after the processing of pre-tRNAs by RNase P. In addition, translin and trax mutants have elevated levels of tRNA and protein translation and are more resistant to a cell death-inducing agent. Finally, we show that C3PO is also involved in tRNA processing in mouse embryonic fibroblast cells. This study identifies the endogenous RNA substrates of C3PO and provides a potential explanation for its roles in apparently diverse biological processes.\",\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Nature structural & molecular biology',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2012 Aug'},\n",
       " {'PMID': '22469835',\n",
       "  'Title': 'Differential impact of nutrition on developmental and metabolic gene expression during fruiting body development in Neurospora crassa.',\n",
       "  'Abstract': 'Fungal fruiting body size and form are influenced by the ecology of the species, including diverse environmental stimuli. Accordingly, nutritional resources available to the fungus during development can be vital to successful production of fruiting bodies. To investigate the effect of nutrition, perithecial development of Neurospora crassa was induced on two different media, a chemically sparsely nutritive Synthetic Crossing Medium (SCM) and a natural Carrot Agar (CA). Protoperithecia were collected before crossing, and perithecia were collected at 2, 24, 48, 72, 96, 120, and at full maturity 144 h after crossing. No differences in fruiting body morphology were observed between the two media at any time point. A circuit of microarray hybridizations comparing cDNA from all neighboring stages was performed. For a majority of differentially expressed genes, expression was higher in SCM than in CA, and expression of core metabolic genes was particularly affected. Effects of nutrition were highest in magnitude before crossing, lowering in magnitude during early perithecial development. Interestingly, metabolic effects of the media were also large in magnitude during late perithecial development, at which stage the lower expression in CA presumably reflected the continued intake of diverse complex initial compounds, diminishing the need for expression of anabolic pathways. However, for genes with key regulatory roles in sexual development, including pheromone precursor ccg-4 and poi2, expression patterns were similar between treatments. When possible, a common nutritional environment is ideal for comparing transcriptional profiles between different fungi. Nevertheless, the observed consistency of the developmental program across media, despite considerable metabolic differentiation is reassuring. This result facilitates comparative studies that will require different nutritional resources for sexual development in different fungi.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Fungal genetics and biology : FG & B',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2012 May'},\n",
       " {'PMID': '22532664',\n",
       "  'Title': 'Conserved and essential transcription factors for cellulase gene expression in ascomycete fungi.',\n",
       "  'Abstract': 'Rational engineering of filamentous fungi for improved cellulase production is hampered by our incomplete knowledge of transcriptional regulatory networks. We therefore used the model filamentous fungus Neurospora crassa to search for uncharacterized transcription factors associated with cellulose deconstruction. A screen of a N. crassa transcription factor deletion collection identified two uncharacterized zinc binuclear cluster transcription factors (clr-1 and clr-2) that were required for growth and enzymatic activity on cellulose, but were not required for growth or hemicellulase activity on xylan. Transcriptional profiling with next-generation sequencing methods refined our understanding of the N. crassa transcriptional response to cellulose and demonstrated that clr-1 and clr-2 were required for the bulk of that response, including induction of all major cellulase and some major hemicellulase genes. Functional CLR-1 was necessary for expression of clr-2 and efficient cellobiose utilization. Phylogenetic analyses showed that CLR-1 and CLR-2 are conserved in the genomes of most filamentous ascomycete fungi capable of degrading cellulose. In Aspergillus nidulans, a strain carrying a deletion of the clr-2 homolog (clrB) failed to induce cellulase gene expression and lacked cellulolytic activity on Avicel. Further manipulation of this control system in industrial production strains may significantly improve yields of cellulases for cellulosic biofuel production.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2012 May 8'},\n",
       " {'PMID': '23186325',\n",
       "  'Title': 'Analyses of expressed sequence tags in Neurospora reveal rapid evolution of genes associated with the early stages of sexual reproduction in fungi.',\n",
       "  'Abstract': 'BACKGROUND: The broadly accepted pattern of rapid evolution of reproductive genes is primarily based on studies of animal systems, although several examples of rapidly evolving genes involved in reproduction are found in diverse additional taxa. In fungi, genes involved in mate recognition have been found to evolve rapidly. However, the examples are too few to draw conclusions on a genome scale. RESULTS: In this study, we performed microarray hybridizations between RNA from sexual and vegetative tissues of two strains of the heterothallic (self-sterile) filamentous ascomycete Neurospora intermedia, to identify a set of sex-associated genes in this species. We aligned Expressed Sequence Tags (ESTs) from sexual and vegetative tissue of N. intermedia to orthologs from three closely related species: N. crassa, N. discreta and N. tetrasperma. The resulting four-species alignments provided a dataset for molecular evolutionary analyses. Our results confirm a general pattern of rapid evolution of fungal sex-associated genes, compared to control genes with constitutive expression or a high relative expression during vegetative growth. Among the rapidly evolving sex-associated genes, we identified candidates that could be of importance for mating or fruiting-body development. Analyses of five of these candidate genes from additional species of heterothallic Neurospora revealed that three of them evolve under positive selection. CONCLUSIONS: Taken together, our study represents a novel finding of a genome-wide pattern of rapid evolution of sex-associated genes in the fungal kingdom, and provides a list of candidate genes important for reproductive isolation in Neurospora.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'BMC evolutionary biology',\n",
       "  'Country': 'Sweden',\n",
       "  'Year': '2012 Nov 27'},\n",
       " {'PMID': '22474347',\n",
       "  'Title': 'Induction of lignocellulose-degrading enzymes in Neurospora crassa by cellodextrins.',\n",
       "  'Abstract': 'Neurospora crassa colonizes burnt grasslands in the wild and metabolizes both cellulose and hemicellulose from plant cell walls. When switched from a favored carbon source such as sucrose to cellulose, N. crassa dramatically upregulates expression and secretion of a wide variety of genes encoding lignocellulolytic enzymes. However, the means by which N. crassa and other filamentous fungi sense the presence of cellulose in the environment remains unclear. Here, we show that an N. crassa mutant carrying deletions of two genes encoding extracellular beta-glucosidase enzymes and one intracellular beta-glucosidase lacks beta-glucosidase activity, but efficiently induces cellulase gene expression in the presence of cellobiose, cellotriose, or cellotetraose as a sole carbon source. These data indicate that cellobiose, or a modified version of cellobiose, functions as an inducer of lignocellulolytic gene expression in N. crassa. Furthermore, the inclusion of a deletion of the catabolite repressor gene, cre-1, in the triple beta-glucosidase mutant resulted in a strain that produces higher concentrations of secreted active cellulases on cellobiose. Thus, the ability to induce cellulase gene expression using a common and soluble carbon source simplifies enzyme production and characterization, which could be applied to other cellulolytic filamentous fungi.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2012 Apr 17'},\n",
       " {'PMID': '22345350',\n",
       "  'Title': 'Deciphering transcriptional regulatory mechanisms associated with hemicellulose degradation in Neurospora crassa.',\n",
       "  'Abstract': 'Hemicellulose, the second most abundant plant biomass fraction after cellulose, is widely viewed as a potential substrate for the production of liquid fuels and other value-added materials. Degradation of hemicellulose by filamentous fungi requires production of many different enzymes, which are induced by biopolymers or its derivatives and regulated mainly at the transcriptional level through transcription factors (TFs). Neurospora crassa, a model filamentous fungus, expresses and secretes enzymes required for plant cell wall deconstruction. To better understand genes specifically associated with degradation of hemicellulose, we applied secretome and transcriptome analysis to N. crassa grown on beechwood xylan. We identified 34 secreted proteins and 353 genes with elevated transcription on xylan. The xylanolytic phenotype of strains with deletions in genes identified from the secretome and transcriptome analysis of the wild type was assessed, revealing functions for known and unknown proteins associated with hemicellulose degradation. By evaluating phenotypes of strains containing deletions of predicted TF genes in N. crassa, we identified a TF (XLR-1; xylan degradation regulator 1) essential for hemicellulose degradation that is an ortholog to XlnR/XYR1 in Aspergillus and Trichoderma species, respectively, a major transcriptional regulator of genes encoding both cellulases and hemicellulases. Deletion of xlr-1 in N. crassa abolished growth on xylan and xylose, but growth on cellulose and cellulolytic activity were only slightly affected. To determine the regulatory mechanisms for hemicellulose degradation, we explored the transcriptional regulon of XLR-1 under xylose, xylanolytic, and cellulolytic conditions. XLR-1 regulated only some predicted hemicellulase genes in N. crassa and was required for a full induction of several cellulase genes. Hemicellulase gene expression was induced by a combination of release from carbon catabolite repression (CCR) and induction. This systematic analysis illustrates the similarities and differences in regulation of hemicellulose degradation among filamentous fungi.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Eukaryotic cell',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2012 Apr'},\n",
       " {'PMID': '22590528',\n",
       "  'Title': 'The functions of grainy head-like proteins in animals and fungi and the evolution of apical extracellular barriers.',\n",
       "  'Abstract': 'The Grainy head (GRH) family of transcription factors are crucial for the development and repair of epidermal barriers in all animals in which they have been studied. This is a high-level functional conservation, as the known structural and enzymatic genes regulated by GRH proteins differ between species depending on the type of epidermal barrier being formed. Interestingly, members of the CP2 superfamily of transcription factors, which encompasses the GRH and LSF families in animals, are also found in fungi--organisms that lack epidermal tissues. To shed light on CP2 protein function in fungi, we characterized a Neurospora crassa mutant lacking the CP2 member we refer to as grainy head-like (grhl). We show that Neurospora GRHL has a DNA-binding specificity similar to that of animal GRH proteins and dissimilar to that of animal LSF proteins. Neurospora grhl mutants are defective in conidial-spore dispersal due to an inability to remodel the cell wall, and we show that grhl mutants and the long-known conidial separation-2 (csp-2) mutants are allelic. We then characterized the transcriptomes of both Neurospora grhl mutants and Drosophila grh mutant embryos to look for similarities in the affected genes. Neurospora grhl appears to play a role in the development and remodeling of the cell wall, as well as in the activation of genes involved in defense and virulence. Drosophila GRH is required to activate the expression of many genes involved in cuticular/epidermal-barrier formation. We also present evidence that GRH plays a role in adult antimicrobial defense. These results, along with previous studies of animal GRH proteins, suggest the fascinating possibility that the apical extracellular barriers of some animals and fungi might share an evolutionary connection, and that the formation of physical barriers in the last common ancestor was under the control of a transcriptional code that included GRH-like proteins.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PloS one',\n",
       "  'Country': 'United States of America. adamcpare@gmail.com',\n",
       "  'Year': '2012'},\n",
       " {'PMID': '22276181',\n",
       "  'Title': 'Bisulfite sequencing reveals that Aspergillus flavus holds a hollow in DNA methylation.',\n",
       "  'Abstract': \"Aspergillus flavus first gained scientific attention for its production of aflatoxin. The underlying regulation of aflatoxin biosynthesis has been serving as a theoretical model for biosynthesis of other microbial secondary metabolites. Nevertheless, for several decades, the DNA methylation status, one of the important epigenomic modifications involved in gene regulation, in A. flavus remains to be controversial. Here, we applied bisulfite sequencing in conjunction with a biological replicate strategy to investigate the DNA methylation profiling of A. flavus genome. Both the bisulfite sequencing data and the methylome comparisons with other fungi confirm that the DNA methylation level of this fungus is negligible. Further investigation into the DNA methyltransferase of Aspergillus uncovers its close relationship with RID-like enzymes as well as its divergence with the methyltransferase of species with validated DNA methylation. The lack of repeat contents of the A. flavus' genome and the high RIP-index of the small amount of remanent repeat potentially support our speculation that DNA methylation may be absent in A. flavus or that it may possess de novo DNA methylation which occurs very transiently during the obscure sexual stage of this fungal species. This work contributes to our understanding on the DNA methylation status of A. flavus, as well as reinforces our views on the DNA methylation in fungal species. In addition, our strategy of applying bisulfite sequencing to DNA methylation detection in species with low DNA methylation may serve as a reference for later scientific investigations in other hypomethylated species.\",\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PloS one',\n",
       "  'Country': 'China',\n",
       "  'Year': '2012'},\n",
       " {'PMID': '22813893',\n",
       "  'Title': 'Diversification of a protein kinase cascade: IME-2 is involved in nonself recognition and programmed cell death in Neurospora crassa.',\n",
       "  'Abstract': 'Kinase cascades and the modification of proteins by phosphorylation are major mechanisms for cell signaling and communication, and evolution of these signaling pathways can contribute to new developmental or environmental response pathways. The Saccharomyces cerevisiae kinase Ime2 has been well characterized for its role in meiosis. However, recent studies have revealed alternative functions for Ime2 in both S. cerevisiae and other fungi. In the filamentous fungus Neurospora crassa, the IME2 homolog (ime-2) is not required for meiosis. Here we determine that ime-2 interacts genetically with a transcription factor vib-1 during nonself recognition and programmed cell death (PCD). Mutations in vib-1 (Deltavib-1) suppress PCD due to nonself recognition events; however, a Deltavib-1 Deltaime-2 mutant restored wild-type levels of cell death. A role for ime-2 in the post-translational processing and localization of a mitochondrial matrix protein was identified, which may implicate mitochondria in N. crassa nonself recognition and PCD. Further, Deltavib-1 strains do not produce extracellular proteases, but protease secretion reverted to near wild-type levels in a Deltavib-1 Deltaime-2 strain. Mass spectrometry analysis revealed that the VIB-1 protein is phosphorylated at several sites, including a site that matches the IME-2 consensus. The genetic and biochemical data for ime-2 and vib-1 indicate that IME-2 is a negative regulator of VIB-1 and suggest parallel negative regulation by IME-2 of a cell death pathway in N. crassa that functions in concert with the VIB-1 cell death pathway. Thus, IME2 kinase function has evolved following the divergence of S. cerevisiae and N. crassa and provides insight into the evolution of kinases and their regulatory targets.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Genetics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2012 Oct'},\n",
       " {'PMID': '22247264',\n",
       "  'Title': 'The Ustilago maydis Nit2 homolog regulates nitrogen utilization and is required for efficient induction of filamentous growth.',\n",
       "  'Abstract': 'Nitrogen catabolite repression (NCR) is a regulatory strategy found in microorganisms that restricts the utilization of complex and unfavored nitrogen sources in the presence of favored nitrogen sources. In fungi, this concept has been best studied in yeasts and filamentous ascomycetes, where the GATA transcription factors Gln3p and Gat1p (in yeasts) and Nit2/AreA (in ascomycetes) constitute the main positive regulators of NCR. The reason why functional Nit2 homologs of some phytopathogenic fungi are required for full virulence in their hosts has remained elusive. We have identified the Nit2 homolog in the basidiomycetous phytopathogen Ustilago maydis and show that it is a major, but not the exclusive, positive regulator of nitrogen utilization. By transcriptome analysis of sporidia grown on artificial media devoid of favored nitrogen sources, we show that only a subset of nitrogen-responsive genes are regulated by Nit2, including the Gal4-like transcription factor Ton1 (a target of Nit2). Ustilagic acid biosynthesis is not under the control of Nit2, while nitrogen starvation-induced filamentous growth is largely dependent on functional Nit2. nit2 deletion mutants show the delayed initiation of filamentous growth on maize leaves and exhibit strongly compromised virulence, demonstrating that Nit2 is required to efficiently initiate the pathogenicity program of U. maydis.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Eukaryotic cell',\n",
       "  'Country': 'Germany',\n",
       "  'Year': '2012 Mar'},\n",
       " {'PMID': '22462823',\n",
       "  'Title': 'Unravelling the molecular basis for light modulated cellulase gene expression - the role of photoreceptors in Neurospora crassa.',\n",
       "  'Abstract': 'BACKGROUND: Light represents an important environmental cue, which exerts considerable influence on the metabolism of fungi. Studies with the biotechnological fungal workhorse Trichoderma reesei (Hypocrea jecorina) have revealed an interconnection between transcriptional regulation of cellulolytic enzymes and the light response. Neurospora crassa has been used as a model organism to study light and circadian rhythm biology. We therefore investigated whether light also regulates transcriptional regulation of cellulolytic enzymes in N. crassa. RESULTS: We show that the N. crassa photoreceptor genes wc-1, wc-2 and vvd are involved in regulation of cellulase gene expression, indicating that this phenomenon is conserved among filamentous fungi. The negative effect of VVD on production of cellulolytic enzymes is thereby accomplished by its role in photoadaptation and hence its function in White collar complex (WCC) formation. In contrast, the induction of vvd expression by the WCC does not seem to be crucial in this process. Additionally, we found that WC-1 and WC-2 not only act as a complex, but also have individual functions upon growth on cellulose. CONCLUSIONS: Genome wide transcriptome analysis of photoreceptor mutants and evaluation of results by analysis of mutant strains identified several candidate genes likely to play a role in light modulated cellulase gene expression. Genes with functions in amino acid metabolism, glycogen metabolism, energy supply and protein folding are enriched among genes with decreased expression levels in the wc-1 and wc-2 mutants. The ability to properly respond to amino acid starvation, i. e. up-regulation of the cross pathway control protein cpc-1, was found to be beneficial for cellulase gene expression. Our results further suggest a contribution of oxidative depolymerization of cellulose to plant cell wall degradation in N. crassa.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA. monika.schmoll@tuwien.ac.at',\n",
       "  'Year': '2012 Mar 31'},\n",
       " {'PMID': '22001288',\n",
       "  'Title': 'Modulation of fungal sensitivity to staurosporine by targeting proteins identified by transcriptional profiling.',\n",
       "  'Abstract': 'An analysis of the time-dependent genetic response to the death-inducer staurosporine was performed in Neurospora crassa by transcriptional profiling. Staurosporine induced two major genes encoding an ABC transporter and a protein with similarity to regulatory subunits of potassium channels. The transcriptional response is dependent on the activity of a novel transcription factor. Deletion mutants in differentially expressed genes displayed altered sensitivity to staurosporine, underscoring significant proteins involved in the response to the drug. A null-mutant of the ABC transporter (abc3) is extremely sensitive to staurosporine, accumulates more staurosporine than the wild type strain and is defective in energy-dependent export of the drug, indicating that the ABC3 protein is the first described staurosporine transporter. It was located in the plasma membrane by immunofluorescence microscopy. The combination of inhibitors of ABC transporters or of potassium channels with staurosporine leads to an enhanced activity against N. crassa and pathogenic fungi paving the way to the development of more potent and specific antifungals. Our results highlight the general use of transcriptional profiling for the identification of novel proteins involved in cell death and their potential use as drug targets.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Fungal genetics and biology : FG & B',\n",
       "  'Country': 'Portugal',\n",
       "  'Year': '2011 Dec'},\n",
       " {'PMID': '21980519',\n",
       "  'Title': 'Identification of the CRE-1 cellulolytic regulon in Neurospora crassa.',\n",
       "  'Abstract': \"BACKGROUND: In filamentous ascomycete fungi, the utilization of alternate carbon sources is influenced by the zinc finger transcription factor CreA/CRE-1, which encodes a carbon catabolite repressor protein homologous to Mig1 from Saccharomyces cerevisiae. In Neurospora crassa, deletion of cre-1 results in increased secretion of amylase and beta-galactosidase. METHODOLOGY/PRINCIPAL FINDINGS: Here we show that a strain carrying a deletion of cre-1 has increased cellulolytic activity and increased expression of cellulolytic genes during growth on crystalline cellulose (Avicel). Constitutive expression of cre-1 complements the phenotype of a N. crassa Deltacre-1 strain grown on Avicel, and also results in stronger repression of cellulolytic protein secretion and enzyme activity. We determined the CRE-1 regulon by investigating the secretome and transcriptome of a Deltacre-1 strain as compared to wild type when grown on Avicel versus minimal medium. Chromatin immunoprecipitation-PCR of putative target genes showed that CRE-1 binds to only some adjacent 5'-SYGGRG-3' motifs, consistent with previous findings in other fungi, and suggests that unidentified additional regulatory factors affect CRE-1 binding to promoter regions. Characterization of 30 mutants containing deletions in genes whose expression level increased in a Deltacre-1 strain under cellulolytic conditions identified novel genes that affect cellulase activity and protein secretion. CONCLUSIONS/SIGNIFICANCE: Our data provide comprehensive information on the CRE-1 regulon in N. crassa and contribute to deciphering the global role of carbon catabolite repression in filamentous ascomycete fungi during plant cell wall deconstruction.\",\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'PloS one',\n",
       "  'Country': 'United States of America',\n",
       "  'Year': '2011'},\n",
       " {'PMID': '20716356',\n",
       "  'Title': 'Multi-targeted priming for genome-wide gene expression assays.',\n",
       "  'Abstract': 'BACKGROUND: Complementary approaches to assaying global gene expression are needed to assess gene expression in regions that are poorly assayed by current methodologies. A key component of nearly all gene expression assays is the reverse transcription of transcribed sequences that has traditionally been performed by priming the poly-A tails on many of the transcribed genes in eukaryotes with oligo-dT, or by priming RNA indiscriminately with random hexamers. We designed an algorithm to find common sequence motifs that were present within most protein-coding genes of Saccharomyces cerevisiae and of Neurospora crassa, but that were not present within their ribosomal RNA or transfer RNA genes. We then experimentally tested whether degenerately priming these motifs with multi-targeted primers improved the accuracy and completeness of transcriptomic assays. RESULTS: We discovered two multi-targeted primers that would prime a preponderance of genes in the genomes of Saccharomyces cerevisiae and Neurospora crassa while avoiding priming ribosomal RNA or transfer RNA. Examining the response of Saccharomyces cerevisiae to nitrogen deficiency and profiling Neurospora crassa early sexual development, we demonstrated that using multi-targeted primers in reverse transcription led to superior performance of microarray profiling and next-generation RNA tag sequencing. Priming with multi-targeted primers in addition to oligo-dT resulted in higher sensitivity, a larger number of well-measured genes and greater power to detect differences in gene expression. CONCLUSIONS: Our results provide the most complete and detailed expression profiles of the yeast nitrogen starvation response and N. crassa early sexual development to date. Furthermore, our multi-targeting priming methodology for genome-wide gene expression assays provides selective targeting of multiple sequences and counter-selection against undesirable sequences, facilitating a more complete and precise assay of the transcribed sequences within the genome.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2010 Aug 17'},\n",
       " {'PMID': '20716356',\n",
       "  'Title': 'Multi-targeted priming for genome-wide gene expression assays.',\n",
       "  'Abstract': 'BACKGROUND: Complementary approaches to assaying global gene expression are needed to assess gene expression in regions that are poorly assayed by current methodologies. A key component of nearly all gene expression assays is the reverse transcription of transcribed sequences that has traditionally been performed by priming the poly-A tails on many of the transcribed genes in eukaryotes with oligo-dT, or by priming RNA indiscriminately with random hexamers. We designed an algorithm to find common sequence motifs that were present within most protein-coding genes of Saccharomyces cerevisiae and of Neurospora crassa, but that were not present within their ribosomal RNA or transfer RNA genes. We then experimentally tested whether degenerately priming these motifs with multi-targeted primers improved the accuracy and completeness of transcriptomic assays. RESULTS: We discovered two multi-targeted primers that would prime a preponderance of genes in the genomes of Saccharomyces cerevisiae and Neurospora crassa while avoiding priming ribosomal RNA or transfer RNA. Examining the response of Saccharomyces cerevisiae to nitrogen deficiency and profiling Neurospora crassa early sexual development, we demonstrated that using multi-targeted primers in reverse transcription led to superior performance of microarray profiling and next-generation RNA tag sequencing. Priming with multi-targeted primers in addition to oligo-dT resulted in higher sensitivity, a larger number of well-measured genes and greater power to detect differences in gene expression. CONCLUSIONS: Our results provide the most complete and detailed expression profiles of the yeast nitrogen starvation response and N. crassa early sexual development to date. Furthermore, our multi-targeting priming methodology for genome-wide gene expression assays provides selective targeting of multiple sequences and counter-selection against undesirable sequences, facilitating a more complete and precise assay of the transcribed sequences within the genome.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2010 Aug 17'},\n",
       " {'PMID': '20716356',\n",
       "  'Title': 'Multi-targeted priming for genome-wide gene expression assays.',\n",
       "  'Abstract': 'BACKGROUND: Complementary approaches to assaying global gene expression are needed to assess gene expression in regions that are poorly assayed by current methodologies. A key component of nearly all gene expression assays is the reverse transcription of transcribed sequences that has traditionally been performed by priming the poly-A tails on many of the transcribed genes in eukaryotes with oligo-dT, or by priming RNA indiscriminately with random hexamers. We designed an algorithm to find common sequence motifs that were present within most protein-coding genes of Saccharomyces cerevisiae and of Neurospora crassa, but that were not present within their ribosomal RNA or transfer RNA genes. We then experimentally tested whether degenerately priming these motifs with multi-targeted primers improved the accuracy and completeness of transcriptomic assays. RESULTS: We discovered two multi-targeted primers that would prime a preponderance of genes in the genomes of Saccharomyces cerevisiae and Neurospora crassa while avoiding priming ribosomal RNA or transfer RNA. Examining the response of Saccharomyces cerevisiae to nitrogen deficiency and profiling Neurospora crassa early sexual development, we demonstrated that using multi-targeted primers in reverse transcription led to superior performance of microarray profiling and next-generation RNA tag sequencing. Priming with multi-targeted primers in addition to oligo-dT resulted in higher sensitivity, a larger number of well-measured genes and greater power to detect differences in gene expression. CONCLUSIONS: Our results provide the most complete and detailed expression profiles of the yeast nitrogen starvation response and N. crassa early sexual development to date. Furthermore, our multi-targeting priming methodology for genome-wide gene expression assays provides selective targeting of multiple sequences and counter-selection against undesirable sequences, facilitating a more complete and precise assay of the transcribed sequences within the genome.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2010 Aug 17'},\n",
       " {'PMID': '20716356',\n",
       "  'Title': 'Multi-targeted priming for genome-wide gene expression assays.',\n",
       "  'Abstract': 'BACKGROUND: Complementary approaches to assaying global gene expression are needed to assess gene expression in regions that are poorly assayed by current methodologies. A key component of nearly all gene expression assays is the reverse transcription of transcribed sequences that has traditionally been performed by priming the poly-A tails on many of the transcribed genes in eukaryotes with oligo-dT, or by priming RNA indiscriminately with random hexamers. We designed an algorithm to find common sequence motifs that were present within most protein-coding genes of Saccharomyces cerevisiae and of Neurospora crassa, but that were not present within their ribosomal RNA or transfer RNA genes. We then experimentally tested whether degenerately priming these motifs with multi-targeted primers improved the accuracy and completeness of transcriptomic assays. RESULTS: We discovered two multi-targeted primers that would prime a preponderance of genes in the genomes of Saccharomyces cerevisiae and Neurospora crassa while avoiding priming ribosomal RNA or transfer RNA. Examining the response of Saccharomyces cerevisiae to nitrogen deficiency and profiling Neurospora crassa early sexual development, we demonstrated that using multi-targeted primers in reverse transcription led to superior performance of microarray profiling and next-generation RNA tag sequencing. Priming with multi-targeted primers in addition to oligo-dT resulted in higher sensitivity, a larger number of well-measured genes and greater power to detect differences in gene expression. CONCLUSIONS: Our results provide the most complete and detailed expression profiles of the yeast nitrogen starvation response and N. crassa early sexual development to date. Furthermore, our multi-targeting priming methodology for genome-wide gene expression assays provides selective targeting of multiple sequences and counter-selection against undesirable sequences, facilitating a more complete and precise assay of the transcribed sequences within the genome.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2010 Aug 17'},\n",
       " {'PMID': '20716356',\n",
       "  'Title': 'Multi-targeted priming for genome-wide gene expression assays.',\n",
       "  'Abstract': 'BACKGROUND: Complementary approaches to assaying global gene expression are needed to assess gene expression in regions that are poorly assayed by current methodologies. A key component of nearly all gene expression assays is the reverse transcription of transcribed sequences that has traditionally been performed by priming the poly-A tails on many of the transcribed genes in eukaryotes with oligo-dT, or by priming RNA indiscriminately with random hexamers. We designed an algorithm to find common sequence motifs that were present within most protein-coding genes of Saccharomyces cerevisiae and of Neurospora crassa, but that were not present within their ribosomal RNA or transfer RNA genes. We then experimentally tested whether degenerately priming these motifs with multi-targeted primers improved the accuracy and completeness of transcriptomic assays. RESULTS: We discovered two multi-targeted primers that would prime a preponderance of genes in the genomes of Saccharomyces cerevisiae and Neurospora crassa while avoiding priming ribosomal RNA or transfer RNA. Examining the response of Saccharomyces cerevisiae to nitrogen deficiency and profiling Neurospora crassa early sexual development, we demonstrated that using multi-targeted primers in reverse transcription led to superior performance of microarray profiling and next-generation RNA tag sequencing. Priming with multi-targeted primers in addition to oligo-dT resulted in higher sensitivity, a larger number of well-measured genes and greater power to detect differences in gene expression. CONCLUSIONS: Our results provide the most complete and detailed expression profiles of the yeast nitrogen starvation response and N. crassa early sexual development to date. Furthermore, our multi-targeting priming methodology for genome-wide gene expression assays provides selective targeting of multiple sequences and counter-selection against undesirable sequences, facilitating a more complete and precise assay of the transcribed sequences within the genome.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'BMC genomics',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2010 Aug 17'},\n",
       " {'PMID': '20417140',\n",
       "  'Title': 'Diverse pathways generate microRNA-like RNAs and Dicer-independent small interfering RNAs in fungi.',\n",
       "  'Abstract': 'A variety of small RNAs, including the Dicer-dependent miRNAs and the Dicer-independent Piwi-interacting RNAs, associate with Argonaute family proteins to regulate gene expression in diverse cellular processes. These two species of small RNA have not been found in fungi. Here, by analyzing small RNAs associated with the Neurospora Argonaute protein QDE-2, we show that diverse pathways generate miRNA-like small RNAs (milRNAs) and Dicer-independent small interfering RNAs (disiRNAs) in this filamentous fungus. Surprisingly, milRNAs are produced by at least four different mechanisms that use a distinct combination of factors, including Dicers, QDE-2, the exonuclease QIP, and an RNase III domain-containing protein, MRPL3. In contrast, disiRNAs originate from loci producing overlapping sense and antisense transcripts, and do not require the known RNAi components for their production. Taken together, these results uncover several pathways for small RNA production in filamentous fungi, shedding light on the diversity and evolutionary origins of eukaryotic small RNAs.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Molecular cell',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2010 Jun 25'},\n",
       " {'PMID': '20395474',\n",
       "  'Title': 'Genome-wide evolutionary analysis of eukaryotic DNA methylation.',\n",
       "  'Abstract': 'Eukaryotic cytosine methylation represses transcription but also occurs in the bodies of active genes, and the extent of methylation biology conservation is unclear. We quantified DNA methylation in 17 eukaryotic genomes and found that gene body methylation is conserved between plants and animals, whereas selective methylation of transposons is not. We show that methylation of plant transposons in the CHG context extends to green algae and that exclusion of histone H2A.Z from methylated DNA is conserved between plants and animals, and we present evidence for RNA-directed DNA methylation of fungal genes. Our data demonstrate that extant DNA methylation systems are mosaics of conserved and derived features, and indicate that gene body methylation is an ancient property of eukaryotic genomes.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Science (New York, N.Y.)',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2010 May 14'},\n",
       " {'PMID': '20305004',\n",
       "  'Title': 'Genetic and molecular characterization of a cryptochrome from the filamentous fungus Neurospora crassa.',\n",
       "  'Abstract': 'In plants and animals, cryptochromes function as either photoreceptors or circadian clock components. We have examined the cryptochrome from the filamentous fungus Neurospora crassa and demonstrate that Neurospora cry encodes a DASH-type cryptochrome that appears capable of binding flavin adenine dinucleotide (FAD) and methenyltetrahydrofolate (MTHF). The cry transcript and CRY protein levels are strongly induced by blue light in a wc-1-dependent manner, and cry transcript is circadianly regulated, with a peak abundance opposite in phase to frq. Neither deletion nor overexpression of cry appears to perturb the free-running circadian clock. However, cry disruption knockout mutants show a small phase delay under circadian entrainment. Using electrophoretic mobility shift assays (EMSA), we show that CRY is capable of binding single- and double-stranded DNA (ssDNA and dsDNA, respectively) and ssRNA and dsRNA. Whole-genome microarray experiments failed to identify substantive transcriptional regulatory activity of cry under our laboratory conditions.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Eukaryotic cell',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2010 May'},\n",
       " {'PMID': '19269344',\n",
       "  'Title': 'Construction of an hdfA Penicillium chrysogenum strain impaired in non-homologous end-joining and analysis of its potential for functional analysis studies.',\n",
       "  'Abstract': 'The homologous recombination mechanism for DNA-repair is not predominant in most filamentous fungi, resulting in extremely low targeting efficiencies for molecular engineering. To increase the gene targeting efficiency, it is becoming common practice to inactivate the non-homologous end-joining (NHEJ) pathway that causes random integration, by deleting the fungal homologs of the human KU70 and KU80 genes that encode proteins functioning in the NHEJ pathway. This has been described for several filamentous fungi, but limited knowledge on the physiological consequences is available. In this study we characterized targeting efficiency and physiology of penicillinG producing Penicillium chrysogenum strains, in which the KU70 or KU80 homologues hdfA and hdfB had been deleted. Targeting efficiency was increased from ca. 1% in the reference strain to 47% and 56% in the hdfA and hdfB mutant strains, respectively, using an ends-out construct. Physiological and transcriptome analysis of glucose-limited chemostat cultures of the hdfA deletion strain and the reference strain showed minimal differences. Although, in a direct competition experiment to assess strain fitness, the reference strain had a clear advantage over the deletion strain, the results demonstrate the potential of DeltahdfAP. chrysogenum strains for the functional analysis of the recently completed P. chrysogenum genome sequence and in further metabolic engineering of antibiotics production.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Fungal genetics and biology : FG & B',\n",
       "  'Country': 'The Netherlands',\n",
       "  'Year': '2009 May'},\n",
       " {'PMID': '19262566',\n",
       "  'Title': 'Genome-wide analysis of light-inducible responses reveals hierarchical light signalling in Neurospora.',\n",
       "  'Abstract': 'White collar-1 (WC-1) and white collar-2 (WC-2) are essential for light-mediated responses in Neurospora crassa, but the molecular mechanisms underlying gene induction and the roles of other real and putative photoreceptors remain poorly characterized. Unsupervised hierarchical clustering of genome-wide microarrays reveals 5.6% of detectable transcripts, including several novel mediators, that are either early or late light responsive. Evidence is shown for photoreception in the absence of the dominant, and here confirmed, white collar complex (WCC) that regulates both types of light responses. VVD primarily modulates late responses, whereas light-responsive submerged protoperithecia-1 (SUB-1), a GATA family transcription factor, is essential for most late light gene expression. After a 15-min light stimulus, the WCC directly binds the sub-1 promoter. Bioinformatics analysis detects many early light response elements (ELREs), as well as identifying a late light response element (LLRE) required for wild-type activity of late light response promoters. The data provide a global picture of transcriptional response to light, as well as illuminating the cis- and trans-acting elements comprising the regulatory signalling cascade that governs the photobiological response.',\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'The EMBO journal',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2009 Apr 22'},\n",
       " {'PMID': '19092133',\n",
       "  'Title': 'Relics of repeat-induced point mutation direct heterochromatin formation in Neurospora crassa.',\n",
       "  'Abstract': \"Both RNAi-dependent and -independent mechanisms have been implicated in the establishment of heterochromatin domains, which may be stabilized by feedback loops involving chromatin proteins and modifications of histones and DNA. Neurospora crassa sports features of heterochromatin found in higher eukaryotes, namely cytosine methylation (5mC), methylation of histone H3 lysine 9 (H3K9me), and heterochromatin protein 1 (HP1), and is a model to investigate heterochromatin establishment and maintenance. We mapped the distribution of HP1, 5mC, H3K9me3, and H3K4me2 at 100 bp resolution and explored their interplay. HP1, H3K9me3, and 5mC were extensively co-localized and defined 44 heterochromatic domains on linkage group VII, all relics of repeat-induced point mutation. Interestingly, the centromere was found in an approximately 350 kb heterochromatic domain with no detectable H3K4me2. 5mC was not found in genes, in contrast to the situation in plants and animals. H3K9me3 is required for HP1 localization and DNA methylation in N. crassa. In contrast, we found that localization of H3K9me3 was independent of 5mC or HP1 at virtually all heterochromatin regions. In addition, we observed complete restoration of DNA methylation patterns after depletion and reintroduction of the H3K9 methylation machinery. These data show that A:T-rich RIP'd DNA efficiently directs methylation of H3K9, which in turn, directs methylation of associated cytosines.\",\n",
       "  'Keywords': '?',\n",
       "  'Journal': 'Genome research',\n",
       "  'Country': 'USA',\n",
       "  'Year': '2009 Mar'}]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "get_keywords(list(df.iloc[:, 4]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PMID</th>\n",
       "      <th>Title</th>\n",
       "      <th>Abstract</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Country</th>\n",
       "      <th>Year</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>37083494</td>\n",
       "      <td>The nutrient-sensing GCN2 signaling pathway is...</td>\n",
       "      <td>Circadian clocks are evolved to adapt to the d...</td>\n",
       "      <td>[CPC-3/CPC-1 pathway, N. crassa, Neurospora cr...</td>\n",
       "      <td>eLife</td>\n",
       "      <td>China</td>\n",
       "      <td>2023 Apr 21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>36603054</td>\n",
       "      <td>Nutritional compensation of the circadian cloc...</td>\n",
       "      <td>Compensation is a defining principle of a true...</td>\n",
       "      <td>?</td>\n",
       "      <td>PLoS biology</td>\n",
       "      <td>United States of America</td>\n",
       "      <td>2023 Jan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>36577368</td>\n",
       "      <td>A circadian clock translational control mechan...</td>\n",
       "      <td>Phosphorylation of Neurospora crassa eukaryoti...</td>\n",
       "      <td>[CP: Molecular biology, CPC-3, GCN2, Neurospor...</td>\n",
       "      <td>Cell reports</td>\n",
       "      <td>USA</td>\n",
       "      <td>2022 Dec 27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>36577368</td>\n",
       "      <td>A circadian clock translational control mechan...</td>\n",
       "      <td>Phosphorylation of Neurospora crassa eukaryoti...</td>\n",
       "      <td>[CP: Molecular biology, CPC-3, GCN2, Neurospor...</td>\n",
       "      <td>Cell reports</td>\n",
       "      <td>USA</td>\n",
       "      <td>2022 Dec 27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>36577368</td>\n",
       "      <td>A circadian clock translational control mechan...</td>\n",
       "      <td>Phosphorylation of Neurospora crassa eukaryoti...</td>\n",
       "      <td>[CP: Molecular biology, CPC-3, GCN2, Neurospor...</td>\n",
       "      <td>Cell reports</td>\n",
       "      <td>USA</td>\n",
       "      <td>2022 Dec 27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>20395474</td>\n",
       "      <td>Genome-wide evolutionary analysis of eukaryoti...</td>\n",
       "      <td>Eukaryotic cytosine methylation represses tran...</td>\n",
       "      <td>?</td>\n",
       "      <td>Science (New York, N.Y.)</td>\n",
       "      <td>USA</td>\n",
       "      <td>2010 May 14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>20305004</td>\n",
       "      <td>Genetic and molecular characterization of a cr...</td>\n",
       "      <td>In plants and animals, cryptochromes function ...</td>\n",
       "      <td>?</td>\n",
       "      <td>Eukaryotic cell</td>\n",
       "      <td>USA</td>\n",
       "      <td>2010 May</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>19269344</td>\n",
       "      <td>Construction of an hdfA Penicillium chrysogenu...</td>\n",
       "      <td>The homologous recombination mechanism for DNA...</td>\n",
       "      <td>?</td>\n",
       "      <td>Fungal genetics and biology : FG &amp; B</td>\n",
       "      <td>The Netherlands</td>\n",
       "      <td>2009 May</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>19262566</td>\n",
       "      <td>Genome-wide analysis of light-inducible respon...</td>\n",
       "      <td>White collar-1 (WC-1) and white collar-2 (WC-2...</td>\n",
       "      <td>?</td>\n",
       "      <td>The EMBO journal</td>\n",
       "      <td>USA</td>\n",
       "      <td>2009 Apr 22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>19092133</td>\n",
       "      <td>Relics of repeat-induced point mutation direct...</td>\n",
       "      <td>Both RNAi-dependent and -independent mechanism...</td>\n",
       "      <td>?</td>\n",
       "      <td>Genome research</td>\n",
       "      <td>USA</td>\n",
       "      <td>2009 Mar</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>129 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         PMID                                              Title   \n",
       "0    37083494  The nutrient-sensing GCN2 signaling pathway is...  \\\n",
       "1    36603054  Nutritional compensation of the circadian cloc...   \n",
       "2    36577368  A circadian clock translational control mechan...   \n",
       "3    36577368  A circadian clock translational control mechan...   \n",
       "4    36577368  A circadian clock translational control mechan...   \n",
       "..        ...                                                ...   \n",
       "124  20395474  Genome-wide evolutionary analysis of eukaryoti...   \n",
       "125  20305004  Genetic and molecular characterization of a cr...   \n",
       "126  19269344  Construction of an hdfA Penicillium chrysogenu...   \n",
       "127  19262566  Genome-wide analysis of light-inducible respon...   \n",
       "128  19092133  Relics of repeat-induced point mutation direct...   \n",
       "\n",
       "                                              Abstract   \n",
       "0    Circadian clocks are evolved to adapt to the d...  \\\n",
       "1    Compensation is a defining principle of a true...   \n",
       "2    Phosphorylation of Neurospora crassa eukaryoti...   \n",
       "3    Phosphorylation of Neurospora crassa eukaryoti...   \n",
       "4    Phosphorylation of Neurospora crassa eukaryoti...   \n",
       "..                                                 ...   \n",
       "124  Eukaryotic cytosine methylation represses tran...   \n",
       "125  In plants and animals, cryptochromes function ...   \n",
       "126  The homologous recombination mechanism for DNA...   \n",
       "127  White collar-1 (WC-1) and white collar-2 (WC-2...   \n",
       "128  Both RNAi-dependent and -independent mechanism...   \n",
       "\n",
       "                                              Keywords   \n",
       "0    [CPC-3/CPC-1 pathway, N. crassa, Neurospora cr...  \\\n",
       "1                                                    ?   \n",
       "2    [CP: Molecular biology, CPC-3, GCN2, Neurospor...   \n",
       "3    [CP: Molecular biology, CPC-3, GCN2, Neurospor...   \n",
       "4    [CP: Molecular biology, CPC-3, GCN2, Neurospor...   \n",
       "..                                                 ...   \n",
       "124                                                  ?   \n",
       "125                                                  ?   \n",
       "126                                                  ?   \n",
       "127                                                  ?   \n",
       "128                                                  ?   \n",
       "\n",
       "                                  Journal                   Country   \n",
       "0                                   eLife                     China  \\\n",
       "1                            PLoS biology  United States of America   \n",
       "2                            Cell reports                       USA   \n",
       "3                            Cell reports                       USA   \n",
       "4                            Cell reports                       USA   \n",
       "..                                    ...                       ...   \n",
       "124              Science (New York, N.Y.)                       USA   \n",
       "125                       Eukaryotic cell                       USA   \n",
       "126  Fungal genetics and biology : FG & B           The Netherlands   \n",
       "127                      The EMBO journal                       USA   \n",
       "128                       Genome research                       USA   \n",
       "\n",
       "            Year  \n",
       "0    2023 Apr 21  \n",
       "1       2023 Jan  \n",
       "2    2022 Dec 27  \n",
       "3    2022 Dec 27  \n",
       "4    2022 Dec 27  \n",
       "..           ...  \n",
       "124  2010 May 14  \n",
       "125     2010 May  \n",
       "126     2009 May  \n",
       "127  2009 Apr 22  \n",
       "128     2009 Mar  \n",
       "\n",
       "[129 rows x 7 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = pd.DataFrame(PMID_info)\n",
    "df_info"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "删除重复文献"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PMID</th>\n",
       "      <th>Title</th>\n",
       "      <th>Abstract</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Journal</th>\n",
       "      <th>Country</th>\n",
       "      <th>Year</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>37083494</td>\n",
       "      <td>The nutrient-sensing GCN2 signaling pathway is...</td>\n",
       "      <td>Circadian clocks are evolved to adapt to the d...</td>\n",
       "      <td>[CPC-3/CPC-1 pathway, N. crassa, Neurospora cr...</td>\n",
       "      <td>eLife</td>\n",
       "      <td>China</td>\n",
       "      <td>2023 Apr 21</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>36603054</td>\n",
       "      <td>Nutritional compensation of the circadian cloc...</td>\n",
       "      <td>Compensation is a defining principle of a true...</td>\n",
       "      <td>?</td>\n",
       "      <td>PLoS biology</td>\n",
       "      <td>United States of America</td>\n",
       "      <td>2023 Jan</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>36577368</td>\n",
       "      <td>A circadian clock translational control mechan...</td>\n",
       "      <td>Phosphorylation of Neurospora crassa eukaryoti...</td>\n",
       "      <td>[CP: Molecular biology, CPC-3, GCN2, Neurospor...</td>\n",
       "      <td>Cell reports</td>\n",
       "      <td>USA</td>\n",
       "      <td>2022 Dec 27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>35913159</td>\n",
       "      <td>Transcriptional Regulation by the Velvet Prote...</td>\n",
       "      <td>Asexual reproduction in fungi facilitates the ...</td>\n",
       "      <td>[RNA-seq, conidiation, light, transcriptional ...</td>\n",
       "      <td>mBio</td>\n",
       "      <td>Spain</td>\n",
       "      <td>2022 Aug 30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>35638725</td>\n",
       "      <td>Secondary Metabolism Gene Clusters Exhibit Inc...</td>\n",
       "      <td>Secondary metabolite clusters (SMCs) encode th...</td>\n",
       "      <td>[Neurospora crassa, asexual development, envir...</td>\n",
       "      <td>mSystems</td>\n",
       "      <td>USA</td>\n",
       "      <td>2022 Jun 28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>20395474</td>\n",
       "      <td>Genome-wide evolutionary analysis of eukaryoti...</td>\n",
       "      <td>Eukaryotic cytosine methylation represses tran...</td>\n",
       "      <td>?</td>\n",
       "      <td>Science (New York, N.Y.)</td>\n",
       "      <td>USA</td>\n",
       "      <td>2010 May 14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>20305004</td>\n",
       "      <td>Genetic and molecular characterization of a cr...</td>\n",
       "      <td>In plants and animals, cryptochromes function ...</td>\n",
       "      <td>?</td>\n",
       "      <td>Eukaryotic cell</td>\n",
       "      <td>USA</td>\n",
       "      <td>2010 May</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>19269344</td>\n",
       "      <td>Construction of an hdfA Penicillium chrysogenu...</td>\n",
       "      <td>The homologous recombination mechanism for DNA...</td>\n",
       "      <td>?</td>\n",
       "      <td>Fungal genetics and biology : FG &amp; B</td>\n",
       "      <td>The Netherlands</td>\n",
       "      <td>2009 May</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>19262566</td>\n",
       "      <td>Genome-wide analysis of light-inducible respon...</td>\n",
       "      <td>White collar-1 (WC-1) and white collar-2 (WC-2...</td>\n",
       "      <td>?</td>\n",
       "      <td>The EMBO journal</td>\n",
       "      <td>USA</td>\n",
       "      <td>2009 Apr 22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>19092133</td>\n",
       "      <td>Relics of repeat-induced point mutation direct...</td>\n",
       "      <td>Both RNAi-dependent and -independent mechanism...</td>\n",
       "      <td>?</td>\n",
       "      <td>Genome research</td>\n",
       "      <td>USA</td>\n",
       "      <td>2009 Mar</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>109 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         PMID                                              Title   \n",
       "0    37083494  The nutrient-sensing GCN2 signaling pathway is...  \\\n",
       "1    36603054  Nutritional compensation of the circadian cloc...   \n",
       "2    36577368  A circadian clock translational control mechan...   \n",
       "3    35913159  Transcriptional Regulation by the Velvet Prote...   \n",
       "4    35638725  Secondary Metabolism Gene Clusters Exhibit Inc...   \n",
       "..        ...                                                ...   \n",
       "104  20395474  Genome-wide evolutionary analysis of eukaryoti...   \n",
       "105  20305004  Genetic and molecular characterization of a cr...   \n",
       "106  19269344  Construction of an hdfA Penicillium chrysogenu...   \n",
       "107  19262566  Genome-wide analysis of light-inducible respon...   \n",
       "108  19092133  Relics of repeat-induced point mutation direct...   \n",
       "\n",
       "                                              Abstract   \n",
       "0    Circadian clocks are evolved to adapt to the d...  \\\n",
       "1    Compensation is a defining principle of a true...   \n",
       "2    Phosphorylation of Neurospora crassa eukaryoti...   \n",
       "3    Asexual reproduction in fungi facilitates the ...   \n",
       "4    Secondary metabolite clusters (SMCs) encode th...   \n",
       "..                                                 ...   \n",
       "104  Eukaryotic cytosine methylation represses tran...   \n",
       "105  In plants and animals, cryptochromes function ...   \n",
       "106  The homologous recombination mechanism for DNA...   \n",
       "107  White collar-1 (WC-1) and white collar-2 (WC-2...   \n",
       "108  Both RNAi-dependent and -independent mechanism...   \n",
       "\n",
       "                                              Keywords   \n",
       "0    [CPC-3/CPC-1 pathway, N. crassa, Neurospora cr...  \\\n",
       "1                                                    ?   \n",
       "2    [CP: Molecular biology, CPC-3, GCN2, Neurospor...   \n",
       "3    [RNA-seq, conidiation, light, transcriptional ...   \n",
       "4    [Neurospora crassa, asexual development, envir...   \n",
       "..                                                 ...   \n",
       "104                                                  ?   \n",
       "105                                                  ?   \n",
       "106                                                  ?   \n",
       "107                                                  ?   \n",
       "108                                                  ?   \n",
       "\n",
       "                                  Journal                   Country   \n",
       "0                                   eLife                     China  \\\n",
       "1                            PLoS biology  United States of America   \n",
       "2                            Cell reports                       USA   \n",
       "3                                    mBio                     Spain   \n",
       "4                                mSystems                       USA   \n",
       "..                                    ...                       ...   \n",
       "104              Science (New York, N.Y.)                       USA   \n",
       "105                       Eukaryotic cell                       USA   \n",
       "106  Fungal genetics and biology : FG & B           The Netherlands   \n",
       "107                      The EMBO journal                       USA   \n",
       "108                       Genome research                       USA   \n",
       "\n",
       "            Year  \n",
       "0    2023 Apr 21  \n",
       "1       2023 Jan  \n",
       "2    2022 Dec 27  \n",
       "3    2022 Aug 30  \n",
       "4    2022 Jun 28  \n",
       "..           ...  \n",
       "104  2010 May 14  \n",
       "105     2010 May  \n",
       "106     2009 May  \n",
       "107  2009 Apr 22  \n",
       "108     2009 Mar  \n",
       "\n",
       "[109 rows x 7 columns]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = df_info.drop_duplicates(subset=['PMID'], keep='first')\n",
    "df_info = df_info.reset_index(drop=True)\n",
    "df_info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# df_info保存为csv\n",
    "df_info.to_csv(key_name +'_GSE_Info_Pub.csv', index=False)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "code",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
