{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{24881579: {'Title': 'Transcriptome and exoproteome analysis of utilization of plant-derived biomass by Myceliophthora thermophila.',\n",
       "  'Abstract': 'Myceliophthora thermophila is a thermophilic fungus whose genome encodes a wide range of carbohydrate-active enzymes (CAZymes) involved in plant biomass degradation. Such enzymes have potential applications in turning different kinds of lignocellulosic feedstock into sugar precursors for biofuels and chemicals. The present study examined and compared the transcriptomes and exoproteomes of M. thermophila during cultivation on different types of complex biomass to gain insight into how its secreted enzymatic machinery varies with different sources of lignocellulose. In the transcriptome analysis three monocot (barley, oat, triticale) and three dicot (alfalfa, canola, flax) plants were used whereas in the proteome analysis additional substrates, i.e. wood and corn stover pulps, were included. A core set of 59 genes encoding CAZymes was up-regulated in response to both monocot and dicot straws, including nine polysaccharide monooxygenases and GH10, but not GH11, xylanases. Genes encoding additional xylanolytic enzymes were up-regulated during growth on monocot straws, while genes encoding additional pectinolytic enzymes were up-regulated in response to dicot biomass. Exoproteome analysis was generally consistent with the conclusions drawn from transcriptome analysis, but additional CAZymes that accumulated to high levels were identified. Despite the wide variety of biomass sources tested some CAZy family members were not expressed under any condition. The results of this study provide a comprehensive view from both transcriptome and exoproteome levels, of how M. thermophila responds to a wide range of biomass sources using its genomic resources.',\n",
       "  'Authors': ['Kolbusz MA',\n",
       "   'Di Falco M',\n",
       "   'Ishmael N',\n",
       "   'Marqueteau S',\n",
       "   'Moisan MC',\n",
       "   'Baptista CDS',\n",
       "   'Powlowski J',\n",
       "   'Tsang A'],\n",
       "  'Year': '2014 Nov',\n",
       "  'Keywords': ['Biomass degradation',\n",
       "   'Carbohydrate-active enzymes',\n",
       "   'Mass spectrometry',\n",
       "   'Myceliophthora thermophila',\n",
       "   'RNA-Seq'],\n",
       "  'Unit': 'Centre for Structural and Functional Genomics',\n",
       "  'Country': 'Canada. Electronic address: magdalena.kolbusz@concordia.ca',\n",
       "  'Publisher': 'Fungal genetics and biology : FG & B'}}"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import json\n",
    "import re\n",
    "import xlrd\n",
    "from Bio import Medline\n",
    "from Bio import Entrez\n",
    "import string \n",
    "import wget\n",
    "\n",
    "\n",
    "Entrez.email = \"1219131962@qq.com\"\n",
    "Entrez.api_key = \"6fcb01c2989c35aa1da3d9d0118abedfc409\"\n",
    "\n",
    "def json_load(path):\n",
    "    \"\"\"Loads the given JSON file and returns it as dictionary.\n",
    "    Arguments\n",
    "    ----------\n",
    "    * path: The path of the JSON file\n",
    "    \"\"\"\n",
    "    with open(path) as f:\n",
    "        dictionary = json.load(f)\n",
    "    return dictionary\n",
    "\n",
    "\n",
    "#根据DOI获得pmid\n",
    "CGM_result_dict={}\n",
    "\n",
    "# DOI='10.1038/nbt.1976'\n",
    "# url='https://www.ncbi.nlm.nih.gov/pmc/utils/idconv/v1.0/?tool=my_tool&email=%s&ids=%s&format=json'%(Entrez.email,DOI)\n",
    "\n",
    "# target_name=DOI.replace('/','#')+'.json'\n",
    "# file_name=wget.download(url,out=target_name)\n",
    "# doi_data=json_load(file_name)\n",
    "# #print(doi_data['records'][0])\n",
    "# if 'status' not in doi_data['records'][0].keys():#得到pmid了\n",
    "# \t#print(DOI,doi_data['records'],doi_data['status'])\n",
    "# \tPMID=doi_data['records'][0]['pmid']\n",
    "\n",
    "PMID = 35257374\n",
    "\n",
    "#根据pmid获取文献信息\n",
    "if PMID:\n",
    "\thandle = Entrez.efetch(db=\"pubmed\", id=PMID, rettype=\"medline\", retmode=\"text\")\n",
    "\trecords = Medline.parse(handle)\n",
    "\trecords = list(records)\n",
    "\tCGM_result_dict[PMID]={}\n",
    "\tfor record in records:\n",
    "\t\t#print(record)\n",
    "\t\tCGM_result_dict[PMID]['Title']=record.get(\"TI\",\"?\")\n",
    "\t\tCGM_result_dict[PMID]['Abstract']=record.get(\"AB\",\"?\")\n",
    "\t\tCGM_result_dict[PMID]['Authors']=record.get(\"AU\",\"?\")\n",
    "\t\tCGM_result_dict[PMID]['Year']=record.get(\"DP\",\"?\")\n",
    "\t\tCGM_result_dict[PMID]['Keywords']=record.get(\"OT\", \"?\")\n",
    "\t\tCGM_result_dict[PMID]['Unit']=record.get(\"AD\", \"?\")[0].split(',')[0]\n",
    "\t\tCGM_result_dict[PMID]['Country']=record.get(\"AD\", \"?\")[0].split(', ')[-1].strip(string.punctuation)\n",
    "\t\tCGM_result_dict[PMID]['Publisher']=record.get(\"JT\", \"?\")\n",
    "\n",
    "CGM_result_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'PMID': '24881579',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20150715',\n",
       "  'LR': '20220410',\n",
       "  'IS': '1096-0937 (Electronic) 1087-1845 (Linking)',\n",
       "  'VI': '72',\n",
       "  'DP': '2014 Nov',\n",
       "  'TI': 'Transcriptome and exoproteome analysis of utilization of plant-derived biomass by Myceliophthora thermophila.',\n",
       "  'PG': '10-20',\n",
       "  'LID': 'S1087-1845(14)00084-X [pii] 10.1016/j.fgb.2014.05.006 [doi]',\n",
       "  'AB': 'Myceliophthora thermophila is a thermophilic fungus whose genome encodes a wide range of carbohydrate-active enzymes (CAZymes) involved in plant biomass degradation. Such enzymes have potential applications in turning different kinds of lignocellulosic feedstock into sugar precursors for biofuels and chemicals. The present study examined and compared the transcriptomes and exoproteomes of M. thermophila during cultivation on different types of complex biomass to gain insight into how its secreted enzymatic machinery varies with different sources of lignocellulose. In the transcriptome analysis three monocot (barley, oat, triticale) and three dicot (alfalfa, canola, flax) plants were used whereas in the proteome analysis additional substrates, i.e. wood and corn stover pulps, were included. A core set of 59 genes encoding CAZymes was up-regulated in response to both monocot and dicot straws, including nine polysaccharide monooxygenases and GH10, but not GH11, xylanases. Genes encoding additional xylanolytic enzymes were up-regulated during growth on monocot straws, while genes encoding additional pectinolytic enzymes were up-regulated in response to dicot biomass. Exoproteome analysis was generally consistent with the conclusions drawn from transcriptome analysis, but additional CAZymes that accumulated to high levels were identified. Despite the wide variety of biomass sources tested some CAZy family members were not expressed under any condition. The results of this study provide a comprehensive view from both transcriptome and exoproteome levels, of how M. thermophila responds to a wide range of biomass sources using its genomic resources.',\n",
       "  'CI': ['Copyright (c) 2014 The Authors. Published by Elsevier Inc. All rights reserved.'],\n",
       "  'FAU': ['Kolbusz, Magdalena Anna',\n",
       "   'Di Falco, Marcos',\n",
       "   'Ishmael, Nadeeza',\n",
       "   'Marqueteau, Sandrine',\n",
       "   'Moisan, Marie-Claude',\n",
       "   'Baptista, Cassio da Silva',\n",
       "   'Powlowski, Justin',\n",
       "   'Tsang, Adrian'],\n",
       "  'AU': ['Kolbusz MA',\n",
       "   'Di Falco M',\n",
       "   'Ishmael N',\n",
       "   'Marqueteau S',\n",
       "   'Moisan MC',\n",
       "   'Baptista CDS',\n",
       "   'Powlowski J',\n",
       "   'Tsang A'],\n",
       "  'AD': ['Centre for Structural and Functional Genomics, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada; Department of Biology, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada; Department of Chemistry and Biochemistry, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada. Electronic address: magdalena.kolbusz@concordia.ca.',\n",
       "   'Centre for Structural and Functional Genomics, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada. Electronic address: marcos.difalco@concordia.ca.',\n",
       "   'Centre for Structural and Functional Genomics, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada. Electronic address: nadeeza.ishmael@concordia.ca.',\n",
       "   'Centre for Structural and Functional Genomics, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada. Electronic address: sandrine.marqueteau@concordia.ca.',\n",
       "   'Centre for Structural and Functional Genomics, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada. Electronic address: marie-claude.moisan@concordia.ca.',\n",
       "   'Centre for Structural and Functional Genomics, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada. Electronic address: Cassio.Baptista@nrc-cnrc.gc.ca.',\n",
       "   'Centre for Structural and Functional Genomics, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada; Department of Chemistry and Biochemistry, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada. Electronic address: justin.powlowski@concordia.ca.',\n",
       "   'Centre for Structural and Functional Genomics, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada; Department of Biology, Concordia University, 7141 Sherbrooke Street West, Montreal, Quebec H4B 1R6, Canada. Electronic address: adrian.tsang@concordia.ca.'],\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20140529',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Fungal Genet Biol',\n",
       "  'JT': 'Fungal genetics and biology : FG & B',\n",
       "  'JID': '9607601',\n",
       "  'RN': ['0 (Proteome)', 'EC 3.2.1.- (Glycoside Hydrolases)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Biomass',\n",
       "   '*Gene Expression Profiling',\n",
       "   'Glycoside Hydrolases/*genetics/*metabolism',\n",
       "   'Plants/*metabolism/microbiology',\n",
       "   'Proteome/*analysis',\n",
       "   'Sordariales/*enzymology/*genetics'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['Biomass degradation',\n",
       "   'Carbohydrate-active enzymes',\n",
       "   'Mass spectrometry',\n",
       "   'Myceliophthora thermophila',\n",
       "   'RNA-Seq'],\n",
       "  'EDAT': '2014/06/03 06:00',\n",
       "  'MHDA': '2015/07/16 06:00',\n",
       "  'CRDT': ['2014/06/03 06:00'],\n",
       "  'PHST': ['2014/03/15 00:00 [received]',\n",
       "   '2014/05/16 00:00 [revised]',\n",
       "   '2014/05/17 00:00 [accepted]',\n",
       "   '2014/06/03 06:00 [entrez]',\n",
       "   '2014/06/03 06:00 [pubmed]',\n",
       "   '2015/07/16 06:00 [medline]'],\n",
       "  'AID': ['S1087-1845(14)00084-X [pii]', '10.1016/j.fgb.2014.05.006 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Fungal Genet Biol. 2014 Nov;72:10-20. doi: 10.1016/j.fgb.2014.05.006. Epub 2014 May 29.'}]"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "records"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "code",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.8"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "c3d95d76b6129e1d94786a4741a564730b26c468655b76be3100adc56a57df93"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
